<html>

<head></head>

<body>
    <response>
        <status>200</status>
        <content>
            <response>
                <status>200</status>
                <content>
                    <div class="front">
                        <p class="jrnlID" data-journal-id-type="publisher-id">default</p>
                        <p class="jrnlTitleGroup"><span class="jrnlTitle">default</span></p>
                        <p class="jrnlAbbrTitle">default</p>
                        <p class="jrnlISSN" data-publication-format="electronic" data-pub-type="epub">1234-232</p>
                        <p class="jrnlISSN" data-publication-format="print" data-pub-type="ppub">123-12</p>
                        <p class="jrnlPublisher"><span class="jrnlPublisherName">default</span></p><span
                            class="jrnlMsNumber" pub-id-type="publisher-id">default123</span>
                        <p class="jrnlDOI" pub-id-type="doi">10.1021/default123</p>
                        <h1 class="jrnlArtTitle">Transcriptomic signatures associated with underlying rapid changes in
                            the early phase brain of bi-directional sex change in <em><span class="del"
                                    data-type="s-case" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100247000" data-cid="1718100247000_s">Trimma</span><span class="ins"
                                    data-type="s-case" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100247000" data-cid="1718100247000_s">trimma</span> okinawae</em>
                        </h1>
                        <div class="jrnlAuthorsGroup" data-group="author" data-content-type="author">
                            <p class="jrnlAuthors"><span class="jrnlAuthorGroup" data-contrib-type="author"><span
                                        class="jrnlAuthor" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                        data-ref="Corresponding Author"><span
                                            class="jrnlGivenName">Katsunori</span><span
                                            class="jrnlSurName">Tamagawa</span><sup><sup class="jrnlAffRef"
                                                data-ref-type="aff" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                                data-citation-string=" 92bf2d78-bea1-4176-b2b7-87083ce1607b ">1,*</sup></sup></span><span
                                        class="jrnlAuthor" data-rid="cd35f9b5-58f9-430b-817e-920418d48835"><span
                                            class="jrnlGivenName">Tomoki</span><span
                                            class="jrnlSurName">Sunobe</span><sup><sup class="jrnlAffRef"
                                                data-ref-type="aff" data-rid="cd35f9b5-58f9-430b-817e-920418d48835"
                                                data-citation-string=" cd35f9b5-58f9-430b-817e-920418d48835 ">2</sup></sup></span><span
                                        class="jrnlAuthor" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"><span
                                            class="jrnlGivenName">Takashi</span><span
                                            class="jrnlSurName">Makino</span><sup><sup class="jrnlAffRef"
                                                data-ref-type="aff" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                                data-citation-string=" 92bf2d78-bea1-4176-b2b7-87083ce1607b ">1</sup></sup></span><span
                                        class="jrnlAuthor" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                        data-ref="Corresponding Author"><span class="jrnlGivenName">Masakado</span><span
                                            class="jrnlSurName">Kawata</span><sup><sup class="jrnlAffRef"
                                                data-ref-type="aff" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                                data-citation-string=" 92bf2d78-bea1-4176-b2b7-87083ce1607b ">1,*</sup></sup></span>,
                                    , ,</span></p>
                            <div class="jrnlAffGroup">
                                <p class="jrnlAff" id="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                    data-ref="Corresponding Affiliation"><sup>
                                        <xref data-ref-type="aff" data-rid="92bf2d78-bea1-4176-b2b7-87083ce1607b"
                                            data-citation-string=" 92bf2d78-bea1-4176-b2b7-87083ce1607b ">1</xref>
                                    </sup><span class="jrnlInstitution" data-content-type="department">Graduate School
                                        of Life Sciences</span>, <span class="jrnlInstitution">Tohoku University</span>,
                                    Aoba-ku, Sendai 980-8578, <span class="jrnlCountry">Japan</span></p>
                                <p class="jrnlAff" id="cd35f9b5-58f9-430b-817e-920418d48835"><sup>
                                        <xref data-ref-type="aff" data-rid="cd35f9b5-58f9-430b-817e-920418d48835"
                                            data-citation-string=" cd35f9b5-58f9-430b-817e-920418d48835 ">2</xref>
                                    </sup><span class="jrnlInstitution" data-content-type="department">Laboratory of
                                        Fish Behavioral Ecology, Tateyama Station, Field Science Center</span>, <span
                                        class="jrnlInstitution">Tokyo University of Marine Science and
                                        Technology</span>, 670 Banda, Tateyama, Chiba, 294-0308, <span
                                        class="jrnlCountry">Japan</span></p>
                            </div>
                        </div>
                        <div class="author-notes">
                            <p class="jrnlCorrAff" id="857fd5e8-854f-4c2b-a4ea-04c1b217d23c"><span
                                    class="jrnlAuthor"><span class="jrnlGivenName">Katsunori</span><span
                                        class="jrnlSurName">Tamagawa</span></span></p>
                            <p class="jrnlCorrAff" id="e081070e-ae9f-4272-bb06-16ba95e9ca6b">Address: 6-3, Aoba,
                                Aramaki, Aoba, Sendai, Miyagi 980-8578, <span class="jrnlCountry">Japan</span></p>
                            <p class="jrnlCorrAff" id="475fa83c-fe79-46c3-909b-5d078397c581">E-mail: <span
                                    class="jrnlEmail">tamagawa.katsunori@gmail.com</span></p>
                            <p class="jrnlCorrAff" id="78da73c3-fe78-42fc-a350-8ff9c85ba0ef"><span
                                    class="jrnlEmail">https://orcid.org/0000-0002-8796-6486</span></p>
                            <p class="jrnlCorrAff" id="604f8c3f-c2a1-4272-9970-5c6ca3fa8be4"><span
                                    class="jrnlAuthor"><span class="jrnlGivenName">Masakado</span><span
                                        class="jrnlSurName">Kawata</span></span></p>
                            <p class="jrnlCorrAff" id="3e9f74f3-54b6-412e-aace-e71c0e6bd851">Address: 6-3, Aoba,
                                Aramaki, Aoba, Sendai, Miyagi 980-8578, <span class="jrnlCountry">Japan</span></p>
                            <p class="jrnlCorrAff" id="06f66c51-b72c-4eb5-9de0-5c77f9bdd893">E-mail: <span
                                    class="jrnlEmail">kawata@tohoku.ac.jp</span></p>
                            <p class="jrnlCorrAff" id="c9e75c8f-c9cd-4520-85fa-780bd380b6f9"><span
                                    class="jrnlEmail">https://orcid.org/0000-0001-8716-5438</span></p>
                        </div>
                        <div class="jrnlKwdGroup">
                            <p class="jrnlKeywordPara"><span class="jrnlKeyword"><span class="jrnlKeyword">sequential
                                        hermaphroditism</span><span class="jrnlKeyword">sexual plasticity</span><span
                                        class="jrnlKeyword">brain</span><span
                                        class="jrnlKeyword">transcriptomics</span><i><span class="jrnlKeyword">trimma
                                            okinawae</span></i></span></p>
                        </div>
                    </div>
                    <div class="body" data-edited="true">
                        <p class="jrnlSecPara"><b><sup>*</sup>Corresponding Authors</b></p>
                        <p class="jrnlSecPara"><b>Keywords</b></p>
                        <p class="jrnlSecPara"><b>Introduction</b> Sexual dimorphism is observed in a wide range of
                            vertebrates. Females and males exhibit clear differences in behavior, physiology, and
                            morphology <span class="jrnlBibRef" data-citation-string=" R1 " data-links="#1,#2"
                                data-track-change="(1,2)">1</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR2 " data-links="#1,#2" data-track-change="(1,2)">2</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR2 " data-links="#1,#2"
                                data-track-change="(1,2)">2</span>. In most species, the sexual fate of an individual is
                            generally determined at an early phase of development <span class="jrnlBibRef"
                                data-citation-string=" R2 " data-links="#3,#4" data-track-change="(3,4)"
                                data-renumbered="R3 to R2">2</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406967">(3,4)</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR4 " data-links="#3,#4"
                                data-track-change="(3,4)">4</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR4 " data-links="#3,#4" data-track-change="(3,4)">4</span>.
                            Interestingly, teleosts exhibit divergent sex determination systems and retain extreme
                            sexual plasticity throughout their life [<span class="jrnlBibRef"
                                data-citation-string=" R3 " data-links="#5" data-track-change="(5)"
                                data-renumbered="R5 to R3">3</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406968">(5)</span>].
                            Various sex-determination systems, including environmental and genetic sex determination as
                            well as sex change (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR"
                                data-tag-type="EGIEPR" data-offset="524" data-match="i.e.," data-pattern=""
                                data-tag-index="3_1">i.e.,</span> sequential hermaphroditism), are observed in a diverse
                            range of teleosts and their ecological and evolutionary roles have attracted significant
                            interest [<span class="jrnlBibRef" data-citation-string=" R4 " data-links="#6"
                                data-track-change="(6)" data-renumbered="R6 to R4">4</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406969">(6)</span>].</p>
                        <p class="jrnlSecPara">One of the most fascinating models of sexual plasticity is sex change
                            regulated by social status in coral reef fish <span class="jrnlBibRef"
                                data-citation-string=" R5 " data-links="#7,#8" data-track-change="(7,8)"
                                data-renumbered="R7 to R5">5</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406970">(7,8)</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR8 " data-links="#7,#8"
                                data-track-change="(7,8)">8</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR8 " data-links="#7,#8" data-track-change="(7,8)">8</span>.
                            Socially controlled sex change including protogyny (female-to-male), protandry
                            (male-to-female), and bi-directional sex change (both directions serially) can occur rapidly
                            [<span class="jrnlBibRef" data-citation-string=" R6 " data-links="#9"
                                data-track-change="(9)" data-renumbered="R9 to R6">6</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406971">(9)</span>]. In particular, behavioral sex change can occur
                            rapidly and within an hour following a social status change and before gonadal change,
                            whereas changes in the brain trigger the sex change in the fish <span class="jrnlBibRef"
                                data-citation-string=" R7 " data-links="#10,#11" data-track-change="(10,11)"
                                data-renumbered="R10 to R7">7</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406972">(10,11)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR11 " data-links="#10,#11"
                                data-track-change="(10,11)">11</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR11 " data-links="#10,#11"
                                data-track-change="(10,11)">11</span>. Although gene expression change is likely to be
                            involved in the initiation of sex change, transcriptional regulation in the brain during the
                            early phase of sex change is poorly understood.</p>
                        <p class="jrnlSecPara">The Okinawa rubble goby, <em>Trimma okinawae,</em> can undergo
                            bi-directional sex change. It is distributed between Kagoshima and southward to the Ryukyu
                            Islands in Japan. A polygynous mating system occurs in <em>T. okinawae</em>, in which a
                            dominant individual becomes male and exhibits territorial behavior to maintain the harem,
                            whereas the others are females developed from juveniles or changed their sex and exhibit
                            subordinate behavior [<span class="jrnlBibRef" data-citation-string=" R8 " data-links="#12"
                                data-track-change="(12)" data-renumbered="R12 to R8">8</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406973">(12)</span>]. The dominance hierarchy of <em>T. okinawae</em>
                            is based on body size within the harem. The largest female changes its sex to male if males
                            leave the harem; however, these males can change back into females upon encountering a
                            larger, more dominant fish <span class="jrnlBibRef" data-citation-string=" R8 "
                                data-links="#12,#13" data-track-change="(12,13)"
                                data-renumbered="R12 to R8">8</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406974">(12,13)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR13 " data-links="#12,#13"
                                data-track-change="(12,13)">13</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR13 " data-links="#12,#13"
                                data-track-change="(12,13)">13</span>. Because sex changes in both directions can be
                            manipulated artificially and are reproducible in small aquariums, <em>T. okinawae</em>
                            represents an excellent model for exploring its regulation during the initial phase at a
                            genetic and molecular level.</p>
                        <p class="jrnlSecPara">To establish a transcriptomic signature associated with rapid sex change,
                            RNA-sequencing was performed using the brains of T. okinawae and the resulting data was used
                            for <span class="jrnlPatterns" data-tag-desc="HIGH"
                                title="changed from 'de novo' to 'de novo'" data-tag-type="HIGH" data-offset="171"
                                data-match="de novo" data-replace="de novo" data-pattern="$1" data-tag-index="6_4"
                                data-function="addTrackChanges">de novo</span> assembly. We selected fish that had only
                            undergone behavioral sex change in each direction, but not gonadal sex change, by
                            artificially replacing individuals in an aquarium. Gene expression profiles were compared
                            between fish that had behaviorally changed sex with those of stable males and females who
                            did not undergo sex change <span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR" data-tag-type="VAR"
                                data-offset="509" data-match="for ≥1 month" data-replace="for" data-pattern="$1"
                                data-tag-index="6_1">for</span><span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="changed '␣≥' to '≥'"
                                data-tag-type="OPR" data-offset="512" data-match="for ≥1 month" data-replace="≥"
                                data-pattern="$2" data-tag-index="6_1" data-track-change=" ≥">≥</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="516" data-match="for ≥1 month" data-replace="1"
                                data-pattern="$1" data-tag-index="6_1">1</span><span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="UNT" data-tag-type="UNT"
                                data-offset="517" data-match="for ≥1 month" data-replace=" month" data-pattern=" $2"
                                data-tag-index="6_1"> month</span> in the laboratory. The transcriptomic changes were
                            identified in the brain associated with the rapid changes occurring in response to social
                            stimuli. Candidate genes implicated in rapid sex change were identified, which provide
                            insight into the regulation of rapid sex change and the sexual plasticity of teleosts.</p>
                        <h2 class="jrnlHead2">RNA extraction and <span class="jrnlPatterns" data-tag-desc="HIGH"
                                title="changed from '<em>de novo</em>' to '<em>de novo</em>'" data-tag-type="HIGH"
                                data-offset="19" data-match="de novo" data-replace="de novo" data-pattern="$1"
                                data-tag-index="7_1" data-function="addTrackChanges">de novo</span> assembly</h2>
                        <p class="jrnlSecPara">Total RNA was extracted and purified from whole brain using a Maxwell
                            16LEV simply RNA tissue kit (Promega) based on the manufacturer’s instructions. RNA was
                            extracted from each of five females, males, male-like females, and female-like males. In
                            total, 20 RNA libraries were prepared using the NEBNext Ultra RNA Library Prep Kit for
                            Illumina, and <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="350" data-match="150 bp" data-replace="150"
                                data-pattern="$1" data-tag-index="8_5">150</span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="353"
                                data-match="150 bp" data-replace=" bp" data-pattern=" $2" data-tag-index="8_5">
                                bp</span> paired-end reads were generated on the Illumina HiSeq <span
                                class="jrnlPatterns" data-tag-desc="THSE" title="THSE" data-tag-type="THSE"
                                data-offset="411" data-match="4000" data-replace="4000" data-pattern="4000"
                                data-tag-index="8_8">4000</span> platform. The quality of the sequencing reads were
                            assessed using FastQC v0.<span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER"
                                data-tag-type="PER" data-offset="492" data-match="11.7 (16)" data-pattern=""
                                data-tag-index="8_1">11.7</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="496" data-match="11.7 (16)" data-pattern=""
                                data-tag-index="8_1"> [<span class="jrnlBibRef" data-citation-string=" R9 "
                                    data-links="#16" data-track-change="(16)" data-renumbered="R16 to R9">9</span><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406975">(16)</span>]</span> and
                            low-quality reads and bases (<span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR" data-tag-type="VAR"
                                data-offset="535" data-match="Q &amp;lt; 20" data-replace="Q" data-pattern="$1"
                                data-tag-index="8_4">Q</span><span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="changed '␣<␣' to '<'"
                                data-tag-type="OPR" data-offset="536" data-match="Q &amp;lt; 20" data-replace="<"
                                data-pattern="$2" data-tag-index="8_4" data-track-change=" &amp;lt; ">&lt;</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="539" data-match="Q &amp;lt; 20" data-replace="20"
                                data-pattern="$3" data-tag-index="8_4">20</span>) were trimmed and discarded using
                            Trimmomatic v0.<span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER"
                                data-tag-type="PER" data-offset="591" data-match="36 (17)" data-pattern=""
                                data-tag-index="8_2">36</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="593" data-match="36 (17)" data-pattern=""
                                data-tag-index="8_2"> [<span class="jrnlBibRef" data-citation-string=" R10 "
                                    data-links="#17" data-track-change="(17)"
                                    data-renumbered="R17 to R10">10</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406976">(17)</span>]</span>. <span class="jrnlPatterns"
                                data-tag-desc="HIGH" title="changed from '<em>De novo</em>' to '<em>De novo</em>'"
                                data-tag-type="HIGH" data-offset="600" data-match="De novo" data-replace="De novo"
                                data-pattern="$1" data-tag-index="8_10" data-function="addTrackChanges"><em>De
                                    novo</em></span> assembly was done using high-quality reads of all <span
                                class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                data-offset="658" data-match="20 samples" data-replace="20" data-pattern="$1"
                                data-tag-index="8_6">20</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT"
                                title="UNT" data-tag-type="UNT" data-offset="660" data-match="20 samples"
                                data-replace=" samples" data-pattern=" $2" data-tag-index="8_6"> samples</span> by
                            Trinity v2.6.5 using default parameters [<span class="jrnlBibRef"
                                data-citation-string=" R11 " data-links="#18" data-track-change="(18)"
                                data-renumbered="R18 to R11">11</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406977">(18)</span>].
                            Subsequently, protein-coding sequences were predicted with TransDecoder v5.1.0 (<a
                                href="https://github.com/TransDecoder/TransDecoder">https://github.com/TransDecoder/TransDecoder</a>)
                            using default parameters. To remove redundancy from protein-coding sequences, CD-hit v4.6.8
                            (cd-hit-est) was used with a sequence identity threshold of -c <span class="jrnlPatterns"
                                data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="1000"
                                data-match="0.95 (19)" data-pattern="" data-tag-index="8_3">0.95</span><span
                                class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL"
                                data-offset="1004" data-match="0.95 (19)" data-pattern="" data-tag-index="8_3"> [<span
                                    class="jrnlBibRef" data-citation-string=" R12 " data-links="#19"
                                    data-track-change="(19)" data-renumbered="R19 to R12">12</span><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406978">(19)</span>]</span>. The
                            comprehensiveness of the transcriptome dataset was assessed using the Benchmarking Universal
                            Single-Copy Ortholog (v3) with the Actinopterygii dataset <span class="jrnlBibRef"
                                data-citation-string=" R13 " data-links="#20,#21" data-track-change="(20,21)"
                                data-renumbered="R20 to R13">13</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406979">(20,21)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR21 " data-links="#20,#21"
                                data-track-change="(20,21)">21</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR21 " data-links="#20,#21"
                                data-track-change="(20,21)">21</span>. To search for orthologs and perform functional
                            annotation, estimated protein-coding sequences were analyzed by eggNOG-mapper v2 [<span
                                class="jrnlBibRef" data-citation-string=" R14 " data-links="#22"
                                data-track-change="(22)" data-renumbered="R22 to R14">14</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406980">(22)</span>].</p>
                        <h2 class="jrnlHead2">Gene expression analysis and <span class="jrnlPatterns"
                                data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="29"
                                data-match="gene ontology" data-pattern="" data-tag-index="" data-abbr="GO">gene
                                ontology</span> enrichment</h2>
                        <p class="jrnlSecPara">RSEM v1.<span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="PER" data-tag-type="PER" data-offset="8" data-match="3.0 (23)" data-pattern=""
                                data-tag-index="10_1">3.0</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="11" data-match="3.0 (23)" data-pattern=""
                                data-tag-index="10_1"> [<span class="jrnlBibRef" data-citation-string=" R15 "
                                    data-links="#23" data-track-change="(23)"
                                    data-renumbered="R23 to R15">15</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406981">(23)</span>]</span> with bowtie2 v2.<span
                                class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER"
                                data-offset="33" data-match="3.5 (24)" data-pattern=""
                                data-tag-index="10_2">3.5</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="36" data-match="3.5 (24)" data-pattern=""
                                data-tag-index="10_2"> [<span class="jrnlBibRef" data-citation-string=" R16 "
                                    data-links="#24" data-track-change="(24)"
                                    data-renumbered="R24 to R16">16</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406982">(24)</span>]</span> was used to map back the reads and
                            calculate the gene-level expression abundance for each sample using high-quality reads with
                            default parameters. Pairwise comparisons of gene expression levels of Actinopterygii
                            orthologous genes estimated by eggNOG-mapper v2 for stable females vs. males, females vs.
                            male-like females, and males vs. female-like males were done using the Wald test in DEseq2
                            v1.<span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER"
                                data-offset="438" data-match="36.0 (25)" data-pattern=""
                                data-tag-index="10_3">36.0</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="442" data-match="36.0 (25)" data-pattern=""
                                data-tag-index="10_3"> [<span class="jrnlBibRef" data-citation-string=" R17 "
                                    data-links="#25" data-track-change="(25)"
                                    data-renumbered="R25 to R17">17</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406983">(25)</span>]</span>. Genes were filtered out with very low
                            expression (sum of expected <span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR" data-tag-type="VAR"
                                data-offset="515" data-match="count &amp;lt; 10" data-replace="count" data-pattern="$1"
                                data-tag-index="10_5">count</span><span class="jrnlPatterns"
                                data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="changed '␣<␣' to '<'"
                                data-tag-type="OPR" data-offset="520" data-match="count &amp;lt; 10" data-replace="<"
                                data-pattern="$2" data-tag-index="10_5" data-track-change=" &amp;lt; ">&lt;</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="523" data-match="count &amp;lt; 10" data-replace="10"
                                data-pattern="$3" data-tag-index="10_5">10</span>). Genes with a <span
                                class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN"
                                data-offset="541" data-match="false discovering rate" data-pattern="" data-tag-index=""
                                data-abbr="FDR">false discovering rate</span> (<span class="jrnlPatterns"
                                data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="565"
                                data-match=" (FDR)" data-pattern="" data-tag-index="10_7">FDR</span>) less than 0.05
                            were considered <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN"
                                data-tag-type="EXPN" data-offset="601" data-match="differentially expressed genes"
                                data-pattern="" data-tag-index="" data-abbr="DEG">differentially expressed genes</span>
                            (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR"
                                data-offset="633" data-match=" (DEGs)" data-pattern=""
                                data-tag-index="10_8">DEGs</span>). As another approach, to detect genes that were
                            associated with changes in behavior, gonadal sex, and social stimulation, a <span
                                class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN"
                                data-offset="763" data-match="likelihood ratio test" data-pattern=""
                                data-tag-index="">likelihood ratio test</span> (<span class="jrnlPatterns"
                                data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="786"
                                data-match=" (LRT)" data-pattern="" data-tag-index="10_9">LRT</span>) was performed
                            using DEseq2 [<span class="jrnlBibRef" data-citation-string=" R17 " data-links="#25"
                                data-track-change="(25)" data-renumbered="R25 to R17">17</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406984">(25)</span>]. We established the full model consisting of
                            explanatory variables, <span class="jrnlPatterns" data-tag-desc="IENR" title="IENR"
                                data-tag-type="IENR" data-offset="891"
                                data-match="LRT) was performed using DEseq2 (25). We established the full model consisting of explanatory variables, i.e.,"
                                data-pattern="" data-tag-index="10_11">i.e.,</span> behavioral sex (female/female-like
                            male or male/male-like female), gonadal sex (female/male-like female or male/female-like
                            male), and social stimulation (female/male or female-like male/male-like female) (<span
                                class="jrnlUncitedRef" data-citation-string=" F2 " data-old-class="jrnlFigRef"
                                data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406942">Fig.
                                    2</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406942">figure 2</span></span>). Then,
                            LRTs were performed between the full model and each single model reducing a variable to
                            detect the effect of each variable on the gene expression (<span class="jrnlUncitedRef"
                                data-citation-string=" F2 " data-old-class="jrnlFigRef" data-uncited="true"><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406943">Fig. 2B</span><span
                                    class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406943">figure 2b</span></span>). If LRT
                            using a reduced model focusing on behavioral sex detects significant differences of the
                            likelihood for a given gene, the gene is assumed to be a behavioral sex-associated gene.
                            Using this approach, we addressed only consistent gene expression changes associated with
                            each variable (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR"
                                data-tag-type="EGIEPR" data-offset="1564" data-match="e.g.," data-pattern=""
                                data-tag-index="10_12">e.g.,</span> concordant upregulation in male-like females and
                            female-like males by social stimulation). The genes detected from these analyses were used
                            for <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN"
                                data-offset="1714" data-match="gene ontology" data-pattern="" data-tag-index=""
                                data-abbr="GO">gene ontology</span> (<span class="jrnlPatterns" data-tag-desc="ABBR"
                                title="ABBR" data-tag-type="ABBR" data-offset="1729" data-match=" (GO)" data-pattern=""
                                data-tag-index="10_10">GO</span>) enrichment analysis based on GO annotations from
                            eggNOG-mapper v2 using the GOseq v1.<span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="PER" data-tag-type="PER" data-offset="1818" data-match="48.0 (26)"
                                data-pattern="" data-tag-index="10_4">48.0</span><span class="jrnlPatterns"
                                data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="1822"
                                data-match="48.0 (26)" data-pattern="" data-tag-index="10_4"> [<span class="jrnlBibRef"
                                    data-citation-string=" R18 " data-links="#26" data-track-change="(26)"
                                    data-renumbered="R26 to R18">18</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406985">(26)</span>]</span>. The FDR was calculated using
                            overrepresented <span class="jrnlPatterns" data-tag-desc="HIGH"
                                title="changed from '<em>p </em>values' to '<em>p </em>values'" data-tag-type="HIGH"
                                data-offset="1874" data-match="p values" data-replace="p values" data-pattern="$1"
                                data-tag-index="10_14" data-function="addTrackChanges">p values</span> by the
                            Benjamini-Hochberg method [<span class="jrnlBibRef" data-citation-string=" R19 "
                                data-links="#27" data-track-change="(27)" data-renumbered="R27 to R19">19</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125406986">(27)</span>] and GO terms with an
                            <span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR"
                                data-tag-type="VAR" data-offset="1942" data-match="FDR &amp;lt;0.05 were"
                                data-replace="FDR" data-pattern="$1" data-tag-index="10_6">FDR</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT"
                                title="changed '␣<' to '<'" data-tag-type="OPR" data-offset="1945"
                                data-match="FDR &amp;lt;0.05 were" data-replace="<" data-pattern="$2"
                                data-tag-index="10_6" data-track-change=" &amp;lt;">&lt;</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="1947" data-match="FDR &amp;lt;0.05 were"
                                data-replace="0.05" data-pattern="$3" data-tag-index="10_6">0.05</span><span
                                class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="UNT"
                                data-tag-type="UNT" data-offset="1951" data-match="FDR &amp;lt;0.05 were"
                                data-replace=" were" data-pattern=" $6" data-tag-index="10_6"> were</span> reported as
                            significantly enriched GO terms. All statistical analyses were conducted using R 4.2.1
                            software (<a href="https://www.r-project.org/">https://www.r-project.org/</a>).</p>
                        <h1 class="jrnlHead1" label="1"><span class="label">1.</span> Results</h1>
                        <p class="jrnlSecPara"><span class="jrnlPatterns" data-tag-desc="HIGH"
                                title="changed from '<em>De novo</em>' to '<em>De novo</em>'" data-tag-type="HIGH"
                                data-offset="0" data-match="De novo" data-replace="De novo" data-pattern="$1"
                                data-tag-index="12_11" data-function="addTrackChanges"><em>De novo</em></span> assembly
                            and filtering resulted in <span class="jrnlPatterns" data-tag-desc="THSE"
                                title="changed '177,631' to '1 77 631'" data-tag-type="THSE" data-offset="43"
                                data-match="177,631" data-replace="1 77 631" data-pattern="1 77 631"
                                data-tag-index="12_7" data-track-change="177,631">1 77 631</span> contigs coding for
                            putative proteins (<span class="jrnlUncitedRef" data-citation-string=" ST1 "
                                data-old-class="jrnlTblRef" data-uncited="true">Table S1</span>). An assessment of the
                            completeness of the transcriptome assembly indicated that <span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="179"
                                data-match="82.1%" data-replace="82.1" data-pattern="$1"
                                data-tag-index="12_2">82.1</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT"
                                title="UNT" data-tag-type="UNT" data-offset="183" data-match="82.1%" data-replace="%"
                                data-pattern="%" data-tag-index="12_2">%</span><span class="jrnlPatterns"
                                data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="184"
                                data-match="82.1% (56.3%" data-pattern="" data-tag-index="12_1"> (56.3%</span> complete
                            and <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                data-offset="205" data-match="25.8%" data-replace="25.8" data-pattern="$1"
                                data-tag-index="12_4">25.8</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT"
                                title="UNT" data-tag-type="UNT" data-offset="209" data-match="25.8%" data-replace="%"
                                data-pattern="%" data-tag-index="12_4">%</span> fragmented) single-copy orthologs were
                            present, whereas <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL"
                                data-tag-type="VAL" data-offset="267" data-match="17.9%" data-replace="17.9"
                                data-pattern="$1" data-tag-index="12_5">17.9</span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="271"
                                data-match="17.9%" data-replace="%" data-pattern="%" data-tag-index="12_5">%</span> were
                            missing. The putative proteins coded in the contigs were subjected to eggNOG-mapper v2 and
                            annotated by taxonomic classification (<span class="jrnlUncitedRef"
                                data-citation-string=" SF1 " data-old-class="jrnlFigRef" data-uncited="true">Fig.
                                S1</span>). Although most were classified into Actinopterygii orthologs (<span
                                class="jrnlPatterns" data-tag-desc="THSE" title="changed '77,703' to '77 703'"
                                data-tag-type="THSE" data-offset="479" data-match="77,703" data-replace="77 703"
                                data-pattern="77 703" data-tag-index="12_8" data-track-change="77,703">77 703</span> out
                            of <span class="jrnlPatterns" data-tag-desc="THSE" title="changed '129,505' to '1 29 505'"
                                data-tag-type="THSE" data-offset="493" data-match="129,505" data-replace="1 29 505"
                                data-pattern="1 29 505" data-tag-index="12_9"
                                data-track-change="129,505">1 29 505</span> proteins), the remainder were classified
                            into orthologs of a different or uncertain taxon. The genes coding for Actinopterygii
                            ortholog proteins included <span class="jrnlPatterns" data-tag-desc="THSE"
                                title="changed '29,762' to '29 762'" data-tag-type="THSE" data-offset="655"
                                data-match="29,762" data-replace="29 762" data-pattern="29 762" data-tag-index="12_10"
                                data-track-change="29,762">29 762</span> putative protein-coding genes expressed in the
                            brain of <em>T. okinawae</em>, which we used for subsequent analyses.</p>
                        <p class="jrnlSecPara">The gene expression profiles revealed an unclear separation among the
                            four groups (<span class="jrnlUncitedRef" data-citation-string=" SF2 "
                                data-old-class="jrnlFigRef" data-uncited="true">Fig. S2</span>). A pairwise comparison
                            between stable females vs. males revealed <span class="jrnlPatterns" data-tag-desc="SNUM"
                                title="SNUM" data-tag-type="SNUM" data-offset="157" data-match="6" data-pattern=""
                                data-tag-index="13_5">6</span> female-biased and <span class="jrnlPatterns"
                                data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="177" data-match="3"
                                data-pattern="" data-tag-index="13_6">3</span> male-biased DEGs (<span
                                class="jrnlUncitedRef" data-citation-string=" F3 " data-old-class="jrnlFigRef"
                                data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406944">Fig. <span
                                        class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                        data-offset="202" data-match="3A" data-replace="3" data-pattern="$1"
                                        data-tag-index="13_1">3</span></span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406944">figure 3</span></span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="203"
                                data-match="3A" data-replace="A" data-pattern="default" data-tag-index="13_1"><span
                                    class="jrnlUncitedRef" data-citation-string=" F3 " data-old-class="jrnlFigRef"
                                    data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                        data-username="Exeter" data-time="1718100333000"
                                        data-cid="1720125406945">A</span><span class="ins cts-1"
                                        data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                        data-cid="1720125406945">figure a</span></span></span>,<span
                                class="jrnlUncitedRef" data-old-class="jrnlTblRef" data-uncited="true"> Tabl</span>e
                            S2). We also identified 13 DEGs between females vs. male-like females, and <span
                                class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM"
                                data-offset="287" data-match="8" data-pattern="" data-tag-index="13_7">8</span>
                            female-biased and <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM"
                                data-tag-type="SNUM" data-offset="307" data-match="5" data-pattern=""
                                data-tag-index="13_8">5</span> male-like female-biased genes (<span
                                class="jrnlUncitedRef" data-citation-string=" F3 " data-old-class="jrnlFigRef"><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406946">Fig. <span class="jrnlPatterns"
                                        data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="345"
                                        data-match="3A" data-replace="3" data-pattern="$1"
                                        data-tag-index="13_2">3</span></span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406946">figure 3</span></span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="346"
                                data-match="3A" data-replace="A" data-pattern="default" data-tag-index="13_2"><span
                                    class="jrnlUncitedRef" data-citation-string=" F3 " data-old-class="jrnlFigRef"><span
                                        class="del cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406947">A</span><span
                                        class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406947">figure
                                        a</span></span></span>,<span class="jrnlUncitedRef" data-old-class="jrnlTblRef">
                                Tabl</span>e S3). In contrast to the other two comparisons, we identified 228 DEGs
                            between males vs. female-like males and 84 of these were male-biased and 144 were
                            female-like male-biased genes (<span class="jrnlUncitedRef" data-citation-string=" F3 "
                                data-old-class="jrnlFigRef"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406948">Fig. <span
                                        class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                        data-offset="543" data-match="3A" data-replace="3" data-pattern="$1"
                                        data-tag-index="13_3">3</span></span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406948">figure 3</span></span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="544"
                                data-match="3A" data-replace="A" data-pattern="default" data-tag-index="13_3"><span
                                    class="jrnlUncitedRef" data-citation-string=" F3 " data-old-class="jrnlFigRef"><span
                                        class="del cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406949">A</span><span
                                        class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406949">figure
                                        a</span></span></span>,<span class="jrnlUncitedRef" data-old-class="jrnlTblRef">
                                Tabl</span>e S4). These DEGs for males vs. female-like males exhibited a significant
                            enrichment in <span class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247000">“</span><span class="ins cts-1"
                                data-tag-desc="LQUOTE" title="changed from '“' to '‘'" data-tag-type="LQUOTE"
                                data-offset="639" data-match="“" data-replace="‘" data-pattern="‘"
                                data-tag-index="13_10" data-function="addTrackChanges" data-track-change="“"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247000">‘</span>pattern orientation<span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247008">”</span><span class="ins cts-1" data-tag-desc="RQUOTE"
                                title="changed from '”' to '’'" data-tag-type="RQUOTE" data-offset="661" data-match="”"
                                data-replace="’" data-pattern="’" data-tag-index="13_12" data-function="addTrackChanges"
                                data-track-change="”" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247008">’</span> and <span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247001">“</span><span class="ins cts-1" data-tag-desc="LQUOTE"
                                title="changed from '“' to '‘'" data-tag-type="LQUOTE" data-offset="669" data-match="“"
                                data-replace="‘" data-pattern="‘" data-tag-index="13_11" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247001">‘</span>regulation of histone H4-K16
                            acetylation,<span class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247009">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="713" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="13_13" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247009">’</span> whereas no significant enrichment was observed for
                            other comparisons (<span class="jrnlUncitedRef" data-citation-string=" F3 "
                                data-old-class="jrnlFigRef" data-uncited="true"><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406950">Fig. 3B</span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406950">figure 3<i>b</i></span></span>,<span class="jrnlUncitedRef"
                                data-old-class="jrnlTblRef"> Tabl</span>e 1,<span class="jrnlUncitedRef"
                                data-old-class="jrnlTblRef"> Tabl</span>e S5<span class="jrnlPatterns"
                                data-tag-desc="HYPTOMIN,VAL" title="changed '–' to '−'" data-tag-type="HYPTOMIN"
                                data-offset="813" data-match="–7" data-replace="−" data-pattern="−"
                                data-tag-index="13_4" data-track-change="–">−</span>7). Overlap of the DEGs from the
                            three comparisons indicated that most of the genes were independently detected by each
                            comparison (<span class="jrnlUncitedRef" data-citation-string=" F3 "
                                data-old-class="jrnlFigRef"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406951">Fig.
                                    3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406951">figure 3<i>b</i></span></span>).
                            The intersection of the three comparisons revealed only <span class="jrnlPatterns"
                                data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="1014" data-match="1"
                                data-pattern="" data-tag-index="13_9">1</span> gene, which is <em>KIAA0100</em>
                            ortholog. With respect to genes immediately controlled by social stimulation, 4 DEGs were
                            detected by pairwise comparisons between females vs. male-like females and males vs.
                            female-like males (<span class="jrnlUncitedRef" data-citation-string=" F3 "
                                data-old-class="jrnlFigRef"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406952">Fig.
                                    3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406952">figure 3<i>b</i></span></span>).
                        </p>
                        <p class="jrnlSecPara">To identify genes that consistently changed their expression based on
                            sexual and social status in the brain, we conducted LRT to compare the full model with the
                            reduced model (<span class="jrnlUncitedRef" data-citation-string=" F2 F4 "
                                data-old-class="jrnlFigRef" data-uncited="true"><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406953">Fig. 2, 4</span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406953">figures 2 and 4</span></span>). The LRT using reduced
                            models for each condition identified 36, 66, and 168 genes associated with behavioral sex,
                            gonadal sex, and social stimulation, respectively (<span class="jrnlUncitedRef"
                                data-citation-string=" F4 " data-old-class="jrnlFigRef" data-uncited="true"><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406954">Fig. 4</span><span
                                    class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406954">figure 4</span></span>, <span
                                class="jrnlUncitedRef" data-citation-string=" SF3 " data-old-class="jrnlFigRef"
                                data-uncited="true">Fig. S3</span>, Table S8<span class="jrnlPatterns"
                                data-tag-desc="HYPTOMIN,VAL" title="changed '–' to '−'" data-tag-type="HYPTOMIN"
                                data-offset="377" data-match="–10" data-replace="−" data-pattern="−"
                                data-tag-index="14_6" data-track-change="–">−</span>10); however, the genes detected by
                            each test showed little overlap (<span class="jrnlUncitedRef" data-citation-string=" F4 "
                                data-old-class="jrnlFigRef"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406955">Fig. <span
                                        class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                        data-offset="454" data-match="4A" data-replace="4" data-pattern="$1"
                                        data-tag-index="14_4">4</span></span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406955">figure 4</span></span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="455"
                                data-match="4A" data-replace="A" data-pattern="default" data-tag-index="14_4"><span
                                    class="jrnlUncitedRef" data-citation-string=" F4 " data-old-class="jrnlFigRef"><span
                                        class="del cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406956">A</span><span
                                        class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                        data-time="1718100333000" data-cid="1720125406956">figure
                                        a</span></span></span>). Using this approach, several genes exhibited
                            intermediate, but significant expression changes, which were not detected by pairwise
                            comparisons. For the gonadal sex-associated genes, <span class="jrnlPatterns"
                                data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="641"
                                data-match="13 (19.7%)" data-pattern="" data-tag-index="14_1">13</span><span
                                class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL"
                                data-offset="643" data-match="13 (19.7%)" data-pattern="" data-tag-index="14_1">
                                (19.7%)</span> were also detected by pairwise comparisons and the other 53 were only
                            detected by the LRT (<span class="jrnlUncitedRef" data-citation-string=" F4 "
                                data-old-class="jrnlFigRef" data-uncited="true"><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406957">Fig. 4B</span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406957">figure 4<i>b</i></span></span>). In contrast, <span
                                class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER"
                                data-offset="766" data-match="20 (58.8%)" data-pattern=""
                                data-tag-index="14_2">20</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT"
                                title="VAL" data-tag-type="VAL" data-offset="768" data-match="20 (58.8%)"
                                data-pattern="" data-tag-index="14_2"> (58.8%)</span> and <span class="jrnlPatterns"
                                data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="781"
                                data-match="105 (62.5%)" data-pattern="" data-tag-index="14_3">105</span><span
                                class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL"
                                data-offset="784" data-match="105 (62.5%)" data-pattern="" data-tag-index="14_3">
                                (62.5%)</span> genes were also identified by pairwise comparisons for the genes
                            identified as behavioral sex-associated or social stimulation-associated, respectively, and
                            most of the overlapped genes represented DEGs between males vs. female-like males (<span
                                class="jrnlUncitedRef" data-citation-string=" F4 " data-old-class="jrnlFigRef"><span
                                    class="del cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406958">Fig. <span class="jrnlPatterns"
                                        data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="1038"
                                        data-match="4C" data-replace="4" data-pattern="$1"
                                        data-tag-index="14_5">4</span></span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406958">figure 4</span></span><span class="jrnlPatterns"
                                data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="1039"
                                data-match="4C" data-replace="C" data-pattern="default" data-tag-index="14_5"><span
                                    class="jrnlUncitedRef" data-citation-string=" F4 " data-old-class="jrnlFigRef"
                                    data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                        data-username="Exeter" data-time="1718100333000"
                                        data-cid="1720125406959">C</span><span class="ins cts-1"
                                        data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                        data-cid="1720125406959">figure c</span></span></span>, D). GO enrichment
                            analyses were performed and several overrepresented GO terms were identified, except for
                            gonadal sex-associated genes (<span class="jrnlUncitedRef" data-citation-string=" T1 "
                                data-old-class="jrnlTblRef" data-uncited="true"><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406960">Table 1</span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406960">table 1</span></span>, Table S11<span class="jrnlPatterns"
                                data-tag-desc="HYPTOMIN,VAL" title="changed '–' to '−'" data-tag-type="HYPTOMIN"
                                data-offset="1197" data-match="–13" data-replace="−" data-pattern="−"
                                data-tag-index="14_7" data-track-change="–">−</span>13). The behavioral sex-associated
                            genes were significantly enriched in <span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100247001" data-cid="1718100247002">“</span><span
                                class="ins cts-1" data-tag-desc="LQUOTE" title="changed from '“' to '‘'"
                                data-tag-type="LQUOTE" data-offset="1272" data-match="“" data-replace="‘"
                                data-pattern="‘" data-tag-index="14_8" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247002">‘</span>regulation of
                            locomotion.<span class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247010">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1300" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="14_11" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247010">’</span> The social stimulation-associated genes were
                            significantly enriched in several GO terms including <span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247003">“</span><span class="ins cts-1" data-tag-desc="LQUOTE"
                                title="changed from '“' to '‘'" data-tag-type="LQUOTE" data-offset="1402" data-match="“"
                                data-replace="‘" data-pattern="‘" data-tag-index="14_9" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247003">‘</span>pattern orientation<span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247011">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1424" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="14_12" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247011">’</span> and <span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100247001" data-cid="1718100247004">“</span><span
                                class="ins cts-1" data-tag-desc="LQUOTE" title="changed from '“' to '‘'"
                                data-tag-type="LQUOTE" data-offset="1432" data-match="“" data-replace="‘"
                                data-pattern="‘" data-tag-index="14_10" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247004">‘</span>regulation of histone H4-K16
                            acetylation<span class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247012">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1475" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="14_13" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247012">’</span> (<span class="jrnlUncitedRef"
                                data-citation-string=" T1 " data-old-class="jrnlTblRef"><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406961">Table 1</span><span class="ins cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406961">table 1</span></span>).</p>
                        <h1 class="jrnlHead1" label="2"><span class="label">2.</span> Discussion</h1>
                        <p class="jrnlSecPara">The transcriptomic changes occurring in the brain during the early phase
                            of sex change are understudied. To our knowledge, previous studies using hermaphroditic
                            species have primarily focused on changes in the gonads or the brain within a few days after
                            the initiation of sex changes <span class="jrnlBibRef" data-citation-string=" R20 "
                                data-links="#28,#29,#30,#29,#30" data-track-change="(28"
                                data-renumbered="R28 to R20">20</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406987">(28</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="(28">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="(28">30</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="(28">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="(28">30</span><span class="jrnlBibRef" data-citation-string=" R20 "
                                data-links="#28,#29,#30,#29,#30" data-track-change="&amp;ndash;"
                                data-renumbered="R28 to R20">20</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406988">–</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="&amp;ndash;">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="&amp;ndash;">30</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="&amp;ndash;">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="&amp;ndash;">30</span><span class="jrnlBibRef"
                                data-citation-string=" R20 " data-links="#28,#29,#30,#29,#30" data-track-change="30"
                                data-renumbered="R28 to R20">20</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406989">30</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="30">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="30">30</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="30">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change="30">30</span><span class="jrnlBibRef" data-citation-string=" R20 "
                                data-links="#28,#29,#30,#29,#30" data-track-change=")"
                                data-renumbered="R28 to R20">20</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406990">)</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change=")">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change=")">30</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR29 " data-links="#28,#29,#30,#29,#30"
                                data-track-change=")">29</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR30 " data-links="#28,#29,#30,#29,#30"
                                data-track-change=")">30</span>, except for recent work [<span class="jrnlBibRef"
                                data-citation-string=" R21 " data-links="#31" data-track-change="(31)"
                                data-renumbered="R31 to R21">21</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406991">(31)</span>].
                            Previous studies using gonochoristic <span class="jrnlBibRef" data-citation-string=" R22 "
                                data-links="#32,#33" data-track-change="(32,33)"
                                data-renumbered="R32 to R22">22</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406992">(32,33)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR33 " data-links="#32,#33"
                                data-track-change="(32,33)">33</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR33 " data-links="#32,#33" data-track-change="(32,33)">33</span>
                            and hermaphrodite fish <span class="jrnlPatterns" data-tag-desc="VAL,RNGS,RNGC,RNGE,UNT"
                                title="RNGS" data-tag-type="RNGS" data-offset="392" data-match="(28–31,34)"
                                data-replace="(28" data-pattern="$2" data-tag-index="16_2">(28</span><span
                                class="jrnlPatterns" data-tag-desc="VAL,RNGS,RNGC,RNGE,UNT" title="changed '–' to '−'"
                                data-tag-type="HYPTOMIN" data-offset="395" data-match="(28–31,34)" data-replace="−"
                                data-pattern="−" data-tag-index="16_2" data-track-change="–">−</span><span
                                class="jrnlPatterns" data-tag-desc="VAL,RNGS,RNGC,RNGE,UNT" title="RNGE"
                                data-tag-type="RNGE" data-offset="398" data-match="(28–31,34)" data-replace="31,34)"
                                data-pattern="$4" data-tag-index="16_2">31,34)</span> revealed few DEGs in the brain
                            between the sexes. Our analysis also identified a small number of DEGs, which is consistent
                            with the previous studies. Although the asymmetric treatment inducing sex change may affect
                            the number of detected genes, especially through stress during transferring and fighting, we
                            found several genes significantly associated with behavioral changes and social stimulation.
                            In the present study, we could not exclude the effect of byproducts from artificial
                            operations on gene expression, such as a winner effect following territorial conflicts and
                            exhaustion after a fight. However, our analysis provides a unique dataset to examine the
                            regulation of rapid responses in the brain during sex changes as well as a practical model
                            system.</p>
                        <p class="jrnlSecPara">The KIAA0100 ortholog, which is a lipid transporter, known as BLTP2 or
                            BCOX1 in humans, exhibited sexually dimorphic expression. The expression levels were changed
                            oppositely depending on the direction of the sex change (<span class="jrnlPatterns"
                                data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="221"
                                data-match="i.e.," data-pattern="" data-tag-index="17_1">i.e.,</span> consistent with
                            behavioral sex) (<span class="jrnlUncitedRef" data-citation-string=" F3 "
                                data-old-class="jrnlFigRef"><span class="del cts-1" data-userid="preediting"
                                    data-username="Exeter" data-time="1718100333000" data-cid="1720125406962">Fig.
                                    3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                    data-time="1718100333000" data-cid="1720125406962">figure 3b</span></span>).
                            Although the information on other organisms is limited, the KIAA0100 ortholog in Drosophila
                            melanogaster, hobbit, regulates neuroendocrine exocytosis and its mutation results in a
                            marked reduction in body size, because it fails to promote insulin secretion <span
                                class="jrnlBibRef" data-citation-string=" R23 " data-links="#35,#36"
                                data-track-change="(35,36)" data-renumbered="R35 to R23">23</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125406993">(35,36)</span>; <span
                                class="jrnlPossBibRef" data-citation-string=" RR36 " data-links="#35,#36"
                                data-track-change="(35,36)">36</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR36 " data-links="#35,#36"
                                data-track-change="(35,36)">36</span>. Because body size is an important factor for sex
                            change of T. okinawae [<span class="jrnlBibRef" data-citation-string=" R8 " data-links="#12"
                                data-track-change="(12)" data-renumbered="R12 to R8">8</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406994">(12)</span>], the KIAA0100 ortholog may be associated with sex
                            change through body size and growth regulation. Hobbit is required for the maintenance of
                            endocrine signaling [<span class="jrnlBibRef" data-citation-string=" R23 " data-links="#35"
                                data-track-change="(35)" data-renumbered="R35 to R23">23</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406995">(35)</span>], and therefore the ortholog of KIAA0100 is
                            possibly associated with neuroendocrine regulation during the sex change in the brain of T.
                            okinawae.</p>
                        <p class="jrnlSecPara">Although only a few DEGs were identified in the females vs. males
                            comparison, several notable differences were detected in the gonadal sex-associated genes by
                            the LRT. For example, genes of the ortholog <span class="jrnlPatterns" data-tag-desc="EXPN"
                                title="EXPN" data-tag-type="EXPN" data-offset="203"
                                data-match="BPI fold containing family C" data-pattern="" data-tag-index=""
                                data-abbr="BPIFC">BPI fold containing family C</span> (<span class="jrnlPatterns"
                                data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="233"
                                data-match=" (BPIFC)" data-pattern="" data-tag-index="18_3">BPIFC</span>) were highly
                            expressed in the brains of fish with female gonads (<span class="jrnlPatterns"
                                data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="304"
                                data-match="i.e.," data-pattern="" data-tag-index="18_4">i.e.,</span> gonads of female
                            and male-like female) in T. okinawae (<span class="jrnlUncitedRef"
                                data-citation-string=" ST9 " data-old-class="jrnlTblRef" data-uncited="true">Table
                                S9</span>). A previous study using Zebrafish revealed that this gene modulates the
                            expression of kiss2, an orthologous gene regulating reproductive development through GnRH
                            neuronal activity in mammals [<span class="jrnlBibRef" data-citation-string=" R24 "
                                data-links="#37" data-track-change="(37)" data-renumbered="R37 to R24">24</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125406996">(37)</span>]. Interestingly,
                            kisspeptin may have other neuroendocrinal functions on social and sexual behavior by
                            regulating neurons that express isotocin and vasotocin, although it is not a direct
                            regulator of reproduction through GnRH neurons in several fish species <span
                                class="jrnlBibRef" data-citation-string=" R25 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="(38" data-renumbered="R38 to R25">25</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125406997">(38</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="(38">39</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="(38">40</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="(38">39</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="(38">40</span><span class="jrnlBibRef" data-citation-string=" R25 "
                                data-links="#38,#39,#40,#39,#40" data-track-change="&amp;ndash;"
                                data-renumbered="R38 to R25">25</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125406998">–</span>;
                            <span class="jrnlPossBibRef" data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="&amp;ndash;">39</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="&amp;ndash;">40</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="&amp;ndash;">39</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                data-track-change="&amp;ndash;">40</span><span class="jrnlPatterns"
                                data-tag-desc="VAL,RNGS,RNGC,RNGE,UNT" title="RNGE" data-tag-type="RNGE"
                                data-offset="833" data-match="(38–40)" data-replace="40)" data-pattern="$4"
                                data-tag-index="18_1"><span class="jrnlBibRef" data-citation-string=" R25 "
                                    data-links="#38,#39,#40,#39,#40" data-track-change="40)"
                                    data-renumbered="R38 to R25">25</span><span class="del cts-1"
                                    data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                    data-cid="1720125406999">40)</span>; <span class="jrnlPossBibRef"
                                    data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                    data-track-change="40)">39</span>; <span class="jrnlPossBibRef"
                                    data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                    data-track-change="40)">40</span>; <span class="jrnlPossBibRef"
                                    data-citation-string=" RR39 " data-links="#38,#39,#40,#39,#40"
                                    data-track-change="40)">39</span>; <span class="jrnlPossBibRef"
                                    data-citation-string=" RR40 " data-links="#38,#39,#40,#39,#40"
                                    data-track-change="40)">40</span></span>. Because these &nbsp;neurotransmitters may
                            control behavioral sex change in a social context <span class="jrnlBibRef"
                                data-citation-string=" R26 " data-links="#11,#31" data-track-change="(11,31)"
                                data-renumbered="R11 to R26">26</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125407000">(11,31)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR31 " data-links="#11,#31"
                                data-track-change="(11,31)">31</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR31 " data-links="#11,#31"
                                data-track-change="(11,31)">31</span>, regulatory changes in these factors may also play
                            an important role in T. okinawae.</p>
                        <p class="jrnlSecPara">&nbsp;&nbsp;&nbsp;&nbsp; In the pairwise comparison between males vs.
                            female-like males, genes associated with the regulation of histone H4-K16 acetylation were
                            enriched in the DEGs. Moreover, Autism Susceptibility Candidate 2 (AUTS2) and Sirtuin 1
                            (SIRT1) were identified as social stimulation-associated genes by the LRT. This suggests
                            that these genes are regulated in the same manner by social stimulation, regardless of sex
                            change direction (<span class="jrnlUncitedRef" data-citation-string=" ST10 "
                                data-old-class="jrnlTblRef" data-uncited="true">Table S10</span>). AUTS2 is a component
                            of the polycomb group multiprotein PRC1-like complex, which activates gene transcription by
                            recruiting histone acetyltransferase. It also plays various roles in neuronal migration and
                            neuritogenesis <span class="jrnlBibRef" data-citation-string=" R27 " data-links="#41,#42"
                                data-track-change="(41,42)" data-renumbered="R41 to R27">27</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125407001">(41,42)</span>; <span
                                class="jrnlPossBibRef" data-citation-string=" RR42 " data-links="#41,#42"
                                data-track-change="(41,42)">42</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR42 " data-links="#41,#42"
                                data-track-change="(41,42)">42</span>. SIRT1 is a deacetylase that acts on histones and
                            is abundantly expressed in the brain [<span class="jrnlBibRef" data-citation-string=" R28 "
                                data-links="#43" data-track-change="(43)" data-renumbered="R43 to R28">28</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125407002">(43)</span>]. Histone acetylation in
                            the brain is involved in social behavior, including pair bonding in prairie voles and
                            caste-related behavior in ants <span class="jrnlBibRef" data-citation-string=" R29 "
                                data-links="#44,#45" data-track-change="(44,45)"
                                data-renumbered="R44 to R29">29</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125407003">(44,45)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR45 " data-links="#44,#45"
                                data-track-change="(44,45)">45</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR45 " data-links="#44,#45"
                                data-track-change="(44,45)">45</span>, and is suggested to play crucial roles in the
                            gonadal sex change of hermaphrodite fish <span class="jrnlBibRef"
                                data-citation-string=" R21 " data-links="#31,#46" data-track-change="(31,46)"
                                data-renumbered="R31 to R21">21</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125407004">(31,46)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR46 " data-links="#31,#46"
                                data-track-change="(31,46)">46</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR46 " data-links="#31,#46"
                                data-track-change="(31,46)">46</span>. In mammals, SIRT1 epigenetically regulates kiss1
                            expression and controls pubertal timing depending on environmental conditions (<span
                                class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR"
                                data-offset="1137" data-match="i.e.," data-pattern="" data-tag-index="19_7">i.e.,</span>
                            obesity and nutrition) [<span class="jrnlBibRef" data-citation-string=" R28 "
                                data-links="#43" data-track-change="(43)" data-renumbered="R43 to R28">28</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125407005">(43)</span>]. SIRT1 and AUTS2 may
                            regulate the chromatin state in kisspeptin neurons or other neurons through social
                            stimulation and sex change in T. okinawae. From this analysis, three AUTS2 orthologs were
                            detected as both stimulation-associated genes and DEGs between male vs. female-like male.
                            Enrichment in GO included <span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100247001" data-cid="1718100247005">“</span><span
                                class="ins cts-1" data-tag-desc="LQUOTE" title="changed from '“' to '‘'"
                                data-tag-type="LQUOTE" data-offset="1481" data-match="“" data-replace="‘"
                                data-pattern="‘" data-tag-index="19_1" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247005">‘</span>pattern orientation,<span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247013">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1504" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="19_4" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247013">’</span> <span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100247001" data-cid="1718100247006">“</span><span
                                class="ins cts-1" data-tag-desc="LQUOTE" title="changed from '“' to '‘'"
                                data-tag-type="LQUOTE" data-offset="1508" data-match="“" data-replace="‘"
                                data-pattern="‘" data-tag-index="19_2" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247006">‘</span>innate vocalization
                            behavior,<span class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247014">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1540" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="19_5" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247014">’</span> and <span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100247001" data-cid="1718100247007">“</span><span
                                class="ins cts-1" data-tag-desc="LQUOTE" title="changed from '“' to '‘'"
                                data-tag-type="LQUOTE" data-offset="1548" data-match="“" data-replace="‘"
                                data-pattern="‘" data-tag-index="19_3" data-function="addTrackChanges"
                                data-track-change="“" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247007">‘</span>dendrite extension,<span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100247001" data-cid="1718100247015">”</span><span class="ins cts-1"
                                data-tag-desc="RQUOTE" title="changed from '”' to '’'" data-tag-type="RQUOTE"
                                data-offset="1570" data-match="”" data-replace="’" data-pattern="’"
                                data-tag-index="19_6" data-function="addTrackChanges" data-track-change="”"
                                data-userid="preediting" data-username="Exeter" data-time="1718100247001"
                                data-cid="1718100247015">’</span> largely relies on these orthologs. AUTS2 is associated
                            with various neuropathological conditions in humans and head size and neurodevelopment
                            function in zebrafish <span class="jrnlBibRef" data-citation-string=" R27 "
                                data-links="#41,#47" data-track-change="(41,47)"
                                data-renumbered="R41 to R27">27</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125407006">(41,47)</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR47 " data-links="#41,#47"
                                data-track-change="(41,47)">47</span>; <span class="jrnlPossBibRef"
                                data-citation-string=" RR47 " data-links="#41,#47"
                                data-track-change="(41,47)">47</span>. Therefore, the expression shift may contribute to
                            the transition of the neurological state associated with sex change in T. okinawae.</p>
                        <p class="jrnlSecPara">The ortholog of lysine demethylase 3B (<em>KDM3B</em>), which is lysine 9
                            of histone H3 (H3K9) demethylase, was identified as a social stimulation-associated gene by
                            the LRT method (<span class="jrnlUncitedRef" data-citation-string=" ST10 "
                                data-old-class="jrnlTblRef">Table S10</span>). The methylation of H3K9 and H3K4 are
                            hallmarks of transcriptionally suppressed and activated chromatin. <em>KDM3B</em> knock-out
                            XY mice were frequently sex-reversed [<span class="jrnlBibRef" data-citation-string=" R30 "
                                data-links="#48" data-track-change="(48)" data-renumbered="R48 to R30">30</span><span
                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                data-time="1718100333000" data-cid="1720125407007">(48)</span>]. A previous study of
                            <em>KDM3B</em> knock-out mice revealed that circulating levels of 17β-estradiol increased
                            and impaired male-specific sexual behavior [<span class="jrnlBibRef"
                                data-citation-string=" R31 " data-links="#30" data-track-change="(30)"
                                data-renumbered="R30 to R31">31</span><span class="del cts-1" data-userid="preediting"
                                data-username="Exeter" data-time="1718100333000" data-cid="1720125407008">(30)</span>].
                            Several H3K9 demethylases are highly expressed in the brain of the androdioecious killifish,
                            <em>Kryptolebias marmoratus</em>, and exhibit significant sexually biased expression [<span
                                class="jrnlBibRef" data-citation-string=" R32 " data-links="#49"
                                data-track-change="(49)" data-renumbered="R49 to R32">32</span><span class="del cts-1"
                                data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                data-cid="1720125407009">(49)</span>]. Based on these findings, <em>KDM3B</em>
                            expression in the brain may respond to social stimulation and activate the genes associated
                            with sexual development or behavior in teleosts.</p>
                        <p class="jrnlSecPara">Our analysis proposed various genes, including novel candidates,
                            associated with the initiation of sex changes in the brain. Future research requires a
                            cautious interpretation of the function of these genes since some gene expressions in the
                            brain are highly localized and therefore might be offset due to using the whole brain. The
                            transcriptome dataset generated here provides potential targets for screening and sheds
                            light on the initiation mechanism of sex changes in fish.</p>
                        <h1 class="jrnlHead1" label="3"><span class="label">3.</span> Figure and table captions</h1>
                        <p class="jrnlSecPara">Results of <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN"
                                data-tag-type="EXPN" data-offset="12" data-match="gene ontology" data-pattern=""
                                data-tag-index="" data-abbr="GO">gene ontology</span> enrichment analysis for pairwise
                            comparison and <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN"
                                data-tag-type="EXPN" data-offset="74" data-match="log-likelihood ratio test"
                                data-pattern="" data-tag-index="" data-abbr="LRT">log-likelihood ratio test</span>
                            (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR"
                                data-offset="101" data-match=" (LRT)" data-pattern="" data-tag-index="23_1">LRT</span>).
                        </p>
                    </div>
                    <div class="back" data-edited="true">
                        <div class="jrnlFootNoteGroup">
                            <h1 class="jrnlFNHead">Notes</h1>
                            <div class="jrnlFootNote" data-fn-type="other"><label>Materials and methods</label>
                                <p class="jrnlFNPara"><b>Sample collection and experimental operation</b> Fish were
                                    collected from a wild population by scuba diving in Makurazaki, Kagoshima
                                    Prefecture, Japan, in <span class="jrnlPatterns" data-tag-desc="HIGH"
                                        title="changed from 'May 2016' to 'May 2016'" data-tag-type="HIGH"
                                        data-offset="178" data-match="May 2016" data-replace="May 2016"
                                        data-pattern="$1" data-tag-index="24_18" data-function="addTrackChanges">May
                                        2016</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">, and transported to the laboratory. To form a stable polygynous
                                    harem, we inserted a polyvinyl chloride tube as a nest site in each aquarium (<span
                                        class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                        data-offset="329" data-match="60 cm" data-replace="60" data-pattern="$1"
                                        data-tag-index="24_1">60</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="331" data-match="60 cm" data-replace=" cm"
                                        data-pattern=" $2" data-tag-index="24_1"> cm</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">width, <span class="jrnlPatterns" data-tag-desc="VAL,UNT"
                                        title="VAL" data-tag-type="VAL" data-offset="342" data-match="36 cm"
                                        data-replace="36" data-pattern="$1" data-tag-index="24_2">36</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="344" data-match="36 cm" data-replace=" cm"
                                        data-pattern=" $2" data-tag-index="24_2"> cm</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">height, and <span class="jrnlPatterns" data-tag-desc="VAL,UNT"
                                        title="VAL" data-tag-type="VAL" data-offset="360" data-match="30 cm"
                                        data-replace="30" data-pattern="$1" data-tag-index="24_3">30</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="362" data-match="30 cm" data-replace=" cm"
                                        data-pattern=" $2" data-tag-index="24_3"> cm</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">depth) and maintained the harem for more than two weeks. Brains
                                    were harvested from males and females from stable harems as well as from
                                    behaviorally sex-changed males and females (five individuals per condition) <span
                                        class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                        data-offset="579" data-match="1 h" data-replace="1" data-pattern="$1"
                                        data-tag-index="24_4">1</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="580" data-match="1 h" data-replace=" h"
                                        data-pattern=" $2" data-tag-index="24_4"> h</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">after manipulation. We defined females behaving like males,
                                    <em>i.e.,</em> showing territorial and nest-cleaning behavior, as male-like females,
                                    while males behaving like females, <span class="jrnlPatterns" data-tag-desc="IENR"
                                        title="IENR" data-tag-type="IENR" data-offset="754"
                                        data-match="60 cm width, 36 cm height, and 30 cm depth) and maintained the harem for more than two weeks. Brains were harvested from males and females from stable harems as well as from behaviorally sex-changed males and females (five individuals per condition) 1 h after manipulation. We defined females behaving like males, i.e., showing territorial and nest-cleaning behavior, as male-like females, while males behaving like females, i.e.,"
                                        data-pattern="" data-tag-index="24_16"><em>i.e.,</em></span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">showing subordinate behavior and leaving the nest tube, as
                                    female-like males, <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL"
                                        data-tag-type="VAL" data-offset="838" data-match="1 h" data-replace="1"
                                        data-pattern="$1" data-tag-index="24_5">1</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="839" data-match="1 h" data-replace=" h"
                                        data-pattern=" $2" data-tag-index="24_5"> h</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">after artificial manipulation (<span class="jrnlUncitedRef"
                                        data-citation-string=" F1 " data-old-class="jrnlFigRef"
                                        data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                            data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406963">Fig. 1</span><span class="ins cts-1"
                                            data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406963">figure 1</span></span>). Male-like females were
                                    obtained artificially by removing a dominant male from the harem. This is a stimulus
                                    for female-to-male sex change as the largest female in the harem adopts typical male
                                    behavior. We selected this female one hour after the removal of the dominant male,
                                    and it was designated a male-like female (<span class="jrnlUncitedRef"
                                        data-citation-string=" F1 " data-old-class="jrnlFigRef"><span class="del cts-1"
                                            data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406964">Fig. <span class="jrnlPatterns"
                                                data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL"
                                                data-offset="1205" data-match="1A" data-replace="1" data-pattern="$1"
                                                data-tag-index="24_6">1</span></span><span class="ins cts-1"
                                            data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406964">figure 1</span></span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="1206" data-match="1A" data-replace="A"
                                        data-pattern="default" data-tag-index="24_6"><span class="jrnlUncitedRef"
                                            data-citation-string=" F1 " data-old-class="jrnlFigRef"><span
                                                class="del cts-1" data-userid="preediting" data-username="Exeter"
                                                data-time="1718100333000" data-cid="1720125406965">A</span><span
                                                class="ins cts-1" data-userid="preediting" data-username="Exeter"
                                                data-time="1718100333000" data-cid="1720125406965">figure
                                                a</span></span></span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">). Meanwhile, female-like males were obtained through artificial
                                    encounters of males from two distinct harems. The males from two different tanks
                                    were moved to an experimental tank and acclimated for <span class="jrnlPatterns"
                                        data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="1407"
                                        data-match="30 min" data-replace="30" data-pattern="$1"
                                        data-tag-index="24_7">30</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="1409" data-match="30 min" data-replace=" min"
                                        data-pattern=" $2" data-tag-index="24_7"> min</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">. During this time, each male was placed in an opaque cage (made
                                    of plastic netting and a thick PVC tube), so that they could not recognize one
                                    another. After removing the opaque cages, the males began to fight with one another
                                    over the PVC tube nest, which is a stimulus for a male-to-female sex change. The
                                    dominant male continued to behave as a male, whereas the loser began to exhibit
                                    typical female behavior. We selected the losing male one hour after the removal of
                                    the opaque cages and this male was designated a female-like male (<span
                                        class="jrnlUncitedRef" data-citation-string=" F1 " data-old-class="jrnlFigRef"
                                        data-uncited="true"><span class="del cts-1" data-userid="preediting"
                                            data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406966">Fig. 1B</span><span class="ins cts-1"
                                            data-userid="preediting" data-username="Exeter" data-time="1718100333000"
                                            data-cid="1720125406966">figure 1<i>b</i></span></span>). The gonadal sex
                                    state of each fish was confirmed by urogenital papillae structure, which depends on
                                    gonadal sexes and manifests as a long tapered posterior in males and a bulbous
                                    appearance in females (12,14,15). The animal experiments were approved by the
                                    committee of Tohoku University (Permit number: <span class="jrnlPatterns"
                                        data-tag-desc="THSE" title="THSE" data-tag-type="THSE" data-offset="2266"
                                        data-match="2017" data-replace="2017" data-pattern="2017"
                                        data-tag-index="24_15">2017</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">LsA<span class="jrnlPatterns" data-tag-desc="HYPTOMIN,VAL"
                                        title="changed '-' to '−'" data-tag-type="HYPTOMIN" data-offset="2273"
                                        data-match="-023" data-replace="−" data-pattern="−" data-tag-index="24_11"
                                        data-track-change="-">−</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">023). The fish were euthanized with an overdose of ethyl
                                    3-aminobenzoate methanesulfonate (MS<span class="jrnlPatterns"
                                        data-tag-desc="HYPTOMIN,VAL" title="changed '-' to '−'" data-tag-type="HYPTOMIN"
                                        data-offset="2368" data-match="-222" data-replace="−" data-pattern="−"
                                        data-tag-index="24_12" data-track-change="-">−</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">222; Sigma-Aldrich, St. Louis, MO; <span class="jrnlPatterns"
                                        data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="2404"
                                        data-match="100 mg/l" data-replace="100" data-pattern="$1"
                                        data-tag-index="24_8">100</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="2407" data-match="100 mg/l"
                                        data-replace=" mg/l" data-pattern=" $2" data-tag-index="24_8"> mg/l</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">), their brains were immediately excised, placed into tubes
                                    containing <em>RNA</em>later stabilization solution (Thermo Fisher Scientific)
                                    within <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL"
                                        data-tag-type="VAL" data-offset="2549" data-match="30 min" data-replace="30"
                                        data-pattern="$1" data-tag-index="24_9">30</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="2551" data-match="30 min" data-replace=" min"
                                        data-pattern=" $2" data-tag-index="24_9"> min</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">after taking the fish from the aquarium. The samples were stored
                                    at −<span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL"
                                        data-tag-type="VAL" data-offset="2627" data-match="80°C" data-replace="80"
                                        data-pattern="$1" data-tag-index="24_10">80</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT"
                                        data-tag-type="UNT" data-offset="2629" data-match="80°C" data-replace="°C"
                                        data-pattern="°C" data-tag-index="24_10">°C</span></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">until RNA extraction.</p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other"><label>Data availability</label>
                                <p class="jrnlFNPara">The <em><span class="jrnlPatterns" data-tag-desc="HIGH"
                                            title="changed from 'de novo' to 'de novo'" data-tag-type="HIGH"
                                            data-offset="2676" data-match="de novo" data-replace="de novo"
                                            data-pattern="$1" data-tag-index="24_19" data-function="addTrackChanges">de
                                            novo</span></em></p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara"> transcriptome assembly, annotation of the transcriptome, RSEM
                                    outputs, and the code are deposited in the Dryad Digital Repository (<a
                                        href="https://doi.org/10.5061/dryad.vdncjsz0r">https://doi.org/10.5061/dryad.vdncjsz0r</a>
                                </p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other">
                                <p class="jrnlFNPara">). Raw sequence reads can be found in the SRA database under
                                    BioProject ID: PRJDB15183.</p>
                            </div>
                            <div class="jrnlFootNote" data-fn-type="other"><label>Author Contributions</label>
                                <p class="jrnlFNPara">K.T., M.K. and T.S. designed the study. K.T. and T.S. conducted
                                    field sampling and animal treatments for experiment. K.T. generated and analyzed the
                                    data with assistance from T.M. and M.K. All authors wrote and reviewed the
                                    manuscript.</p>
                            </div>
                        </div>
                        <div class="jrnlRefGroup">
                            <h1 class="jrnlRefHead" data-level="1">References</h1>
                            <p class="jrnlRefText" data-editable="false" data-id="R1" id="R1"
                                data-reftype="article-journal" data-doi="10.1038/nn1325" data-pmid="15452574"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R1"><span class="RefSlNo" id="65058688-8bf6-4c61-9f72-101ec139b049">1.</span>
                                <span class="RefAuthor" id="6f342968-fc4c-4bbf-8b5e-9259f02fb640"><span
                                        class="RefSurName">Morris</span> <span class="RefGivenName">JA</span></span>,
                                <span class="RefAuthor" id="4e48e863-97e8-47f2-a204-c2a4f3517da3"><span
                                        class="RefSurName">Jordan</span> <span class="RefGivenName">CL</span></span>,
                                <span class="RefAuthor" id="e87146b9-1140-4d26-846a-df02b8c1c4ee"><span
                                        class="RefSurName">Breedlove</span> <span class="RefGivenName">SM</span></span>.
                                <span class="RefYear" id="bfd08758-1974-4e0d-9ba6-3dda58cd8c15">2004</span> <span
                                    class="RefArticleTitle" id="c7a946e4-3775-460b-b7f8-d91682ad86fc">Sexual
                                    differentiation of the vertebrate nervous system</span>. <span
                                    class="RefJournalTitle" id="840bfa92-0814-4881-8253-653567fcf4d6"><i>Nat
                                        Neurosci</i></span> <span class="RefVolume"
                                    id="b6315b2e-e025-4e3f-bcca-760c962cfeca"><b>7</b></span>, <span class="RefFPage"
                                    id="9b40306d-dc7c-4a9a-8f5f-d1cb6d6e0416">1034</span>–<span class="RefLPage"
                                    id="ad78ba46-f25c-4b4e-8694-ec976d60baa9">1039</span>. (<span class="RefDOI"
                                    id="cdb2e291-f9d9-4206-b911-f546de838eec">doi:10.1038/nn1325</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R2" id="R2"
                                data-reftype="article-journal" data-doi="10.1038/nrg2687" data-pmid="19834484"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R2"><span class="RefSlNo" id="ef9ae1cb-fcd8-4448-bcaa-9d68c58748f7">2.</span>
                                <span class="RefAuthor" id="33b42f9b-2db5-416a-8d07-1218a7ea8c06"><span
                                        class="RefSurName">Williams</span> <span class="RefGivenName">TM</span></span>,
                                <span class="RefAuthor" id="81083131-4650-48dd-8f48-70a026c6e6db"><span
                                        class="RefSurName">Carroll</span> <span class="RefGivenName">SB</span></span>.
                                <span class="RefYear" id="3ed6f664-f6f3-470c-aa76-9c2a805f537f">2009</span> <span
                                    class="RefArticleTitle" id="a1552f9e-9baa-47aa-86d5-89627646bf0f">Genetic and
                                    molecular insights into the development and evolution of sexual dimorphism</span>.
                                <span class="RefJournalTitle" id="c78aff52-7e3c-422f-af77-3edbb55812c1"><i>Nat Rev
                                        Genet</i></span> <span class="RefVolume"
                                    id="1df37b38-6cf9-4b02-baf3-920a3a267bc8"><b>10</b></span>, <span class="RefFPage"
                                    id="36a18ad6-88ac-4862-a3ba-162e1b6a37c3">797</span>–<span class="RefLPage"
                                    id="6523c31b-3f74-40eb-8534-9b338acf6f41">804</span>. (<span class="RefDOI"
                                    id="c74ac60f-a059-4555-b120-9a5a6150c886">doi:10.1038/nrg2687</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R3" id="R3"
                                data-reftype="article-journal" data-doi="10.1007/s10228-020-00754-6"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R3"><span class="RefSlNo"
                                    id="8fbb66af-331b-4265-8a53-b4b94441fa25">3.</span> KuwamuraT., SunobeT., SakaiY.,
                                KadotaT., SawadaK. <span class="RefYear"
                                    id="808a36dc-7223-4455-a41d-a65c326d8b3d">2020</span> <span class="RefArticleTitle"
                                    id="438c0cab-9cbe-4399-a0ca-e712a058ab10">Hermaphroditism in Fishes: An Annotated
                                    List of Species, Phylogeny, and Mating System</span>. <span class="RefJournalTitle"
                                    id="27ed4ba6-db0f-45cc-b2c2-e977dae0824c"><i>Ichthyol Res</i></span> <span
                                    class="RefVolume" id="77fe7c25-b4f9-4e33-8adc-1deb05cc8814"><b>67</b></span>, <span
                                    class="RefFPage" id="03ad968e-fefc-4046-964b-ff71522cc516">341</span>–<span
                                    class="RefLPage" id="89acc1c6-ed79-4ee9-adda-eba8333ee581">360</span>. (<span
                                    class="RefDOI"
                                    id="f855cdb4-d6a5-4de4-83cc-4f6633465f1a">doi:10.1007/s10228-020-00754-6</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R4" id="R4"
                                data-reftype="article-journal" data-doi="10.1159/000342009" data-pmid="22948719"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R4"><span class="RefSlNo" id="b1a5c2b3-10f6-4e36-90bc-679b7a10cbe8">4.</span>
                                <span class="RefAuthor" id="2c032102-caac-41e1-bed8-00c986b817ae"><span
                                        class="RefSurName">Kobayashi</span> <span class="RefGivenName">Y</span></span>,
                                <span class="RefAuthor" id="581fcfcc-fa7a-4324-b312-112e85f15788"><span
                                        class="RefSurName">Nagahama</span> <span class="RefGivenName">Y</span></span>,
                                <span class="RefAuthor" id="6d5e21ec-df9b-495e-9822-dff9e497f625"><span
                                        class="RefSurName">Nakamura</span> <span class="RefGivenName">M</span></span>.
                                <span class="RefYear" id="233a6565-f889-4e6c-a015-5fa887a1b3c2">2013</span> <span
                                    class="RefArticleTitle" id="2f26fff5-1727-4289-b945-c65d11452ed9">Diversity and
                                    plasticity of sex determination and differentiation in fishes</span>. <span
                                    class="RefJournalTitle" id="e957bf9f-d9c2-4e09-9193-15506311989f"><i>Sex
                                        Dev</i></span> <span class="RefVolume"
                                    id="feecc378-08b5-43bb-ab81-0f13cd2eb538"><b>7</b></span>, <span class="RefFPage"
                                    id="6072e7aa-ec20-4927-9977-a74503eabde6">115</span>–<span class="RefLPage"
                                    id="4c7b8252-5ae8-417d-9dae-05239bf21a97">125</span>. (<span class="RefDOI"
                                    id="0131eceb-a64a-4fcc-833d-2608ea2fee20">doi:10.1159/000342009</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R5" id="R5"
                                data-reftype="article-journal" data-doi="10.1016/j.semcdb.2008.12.003"
                                data-pmid="19124081" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R5"><span class="RefSlNo"
                                    id="4c8ed243-00b8-4b26-9278-dd17081fa537">5.</span> <span class="RefAuthor"
                                    id="5dcb529c-855d-453c-b998-7f8ddd002004"><span class="RefSurName">Godwin</span>
                                    <span class="RefGivenName">J</span></span>. <span class="RefYear"
                                    id="50085511-c16f-4962-a110-9ef0f4695da9">2009</span> <span class="RefArticleTitle"
                                    id="6b428e33-b4a6-4db8-8d0a-02ed56903e2d">Social determination of sex in reef
                                    fishes</span>. <span class="RefJournalTitle"
                                    id="c6a85025-b625-46e8-9da5-78aeed961c09"><i>Semin Cell Dev Biol</i></span> <span
                                    class="RefVolume" id="2ecaf269-f40f-48d6-8164-5fa9a0e27243"><b>20</b></span>, <span
                                    class="RefFPage" id="4d7b8302-5d6a-4446-9739-ce8756e5e9bd">264</span>–<span
                                    class="RefLPage" id="cad3525d-9b73-4dcf-aa2c-ec3f6595d00f">270</span>. (<span
                                    class="RefDOI"
                                    id="87bbfd46-a6ec-450d-81f2-da0a5e604aad">doi:10.1016/j.semcdb.2008.12.003</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R6" id="R6"
                                data-reftype="article-journal" data-doi="10.1007/PL00012232" data-reftype-name="Journal"
                                data-csl="true" data-validated-log="true" data-validated="true" new-id="R6"><span
                                    class="RefSlNo" id="b8d12ea5-f78e-459e-9589-3b708cdfd60a">6.</span> <span
                                    class="RefAuthor" id="d756b6d5-d0e2-43f0-9f72-d7aba85617ab"><span
                                        class="RefSurName">H</span> <span class="RefGivenName">ReavisR</span></span>,
                                <span class="RefAuthor" id="76b0ffb4-9acb-4b61-b8d7-0a8cd1fb6fa6"><span
                                        class="RefSurName">S</span> <span class="RefGivenName">GroberM</span></span>.
                                <span class="RefYear" id="67205ed5-0d81-43cd-a0bf-678ec9e4ce18">1999</span> <span
                                    class="RefArticleTitle" id="f4062aac-e052-4558-8fd5-57aef55241d0">An Integrative
                                    Approach to Sex Change: Social, Behavioural and Neurochemical Changes in Lythrypnus
                                    Dalli (Pisces)</span>. <span class="RefJournalTitle"
                                    id="009f8f6a-80b9-4981-bcbc-4a447257c7c8"><i>Acta Ethol</i></span> <span
                                    class="RefVolume" id="c7e1ffcd-fbe1-4d12-a76e-cb0e28af136c"><b>2</b></span>, <span
                                    class="RefFPage" id="bcebcd51-ab50-4890-ab38-d4ce8b10743c">51</span>–<span
                                    class="RefLPage" id="c724748c-e606-4d02-bb08-cc29faa6f8ee">60</span>. (<span
                                    class="RefDOI"
                                    id="cd513ed6-5d81-4cf2-a76c-5a45ab96f2bb">doi:10.1007/PL00012232</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R7" id="R7"
                                data-reftype="article-journal" data-doi="10.1016/j.yfrne.2010.02.002"
                                data-pmid="20176046" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R7"><span class="RefSlNo"
                                    id="8d5b5f65-7f2e-447e-a039-820778ea962d">7.</span> <span class="RefAuthor"
                                    id="f970b60a-09c8-4f9b-a65e-73883eedcac2"><span class="RefSurName">Godwin</span>
                                    <span class="RefGivenName">J</span></span>. <span class="RefYear"
                                    id="295cfbf9-35d3-418e-9615-df2acd378fec">2010</span> <span class="RefArticleTitle"
                                    id="22a7d02a-ad14-472b-8328-4ca91d50cbf3">Neuroendocrinology of sexual plasticity in
                                    teleost fishes</span>. <span class="RefJournalTitle"
                                    id="0c2045b9-7664-4f7b-b29c-f064a7ba806e"><i>Front Neuroendocrinol</i></span> <span
                                    class="RefVolume" id="4743e16f-2f98-4c04-83d1-83d5d709b0dd"><b>31</b></span>, <span
                                    class="RefFPage" id="16e704e8-788b-4463-8692-93e26d8f9c84">203</span>–<span
                                    class="RefLPage" id="ae4fb4d7-3018-4a63-b727-4ce876cb2b8e">216</span>. (<span
                                    class="RefDOI"
                                    id="b51313a8-ed84-4bdb-825d-329abc455871">doi:10.1016/j.yfrne.2010.02.002</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R8" id="R8"
                                data-reftype="article-journal" data-doi="10.1111/j.1439-0310.1993.tb00450.x"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R8"><span class="RefSlNo"
                                    id="5748883d-f524-4ea1-afed-80df7cf8b77b">8.</span> SunobeT., NakazonoA. <span
                                    class="RefYear" id="a05df62a-62ec-411f-b5a4-97df13e7377e">1993</span> <span
                                    class="RefArticleTitle" id="06def2a2-65b2-4841-9919-8fd92b88d4c4">Sex Change in Both
                                    Directions by Alteration of Social Dominance inTrimma Okinawae(Pisces:
                                    Gobiidae)</span>. <span class="RefJournalTitle"
                                    id="68de8c51-b437-46e6-9a01-e51e1fae1c65"><i>Ethology</i></span> <span
                                    class="RefVolume" id="ea2bbc15-e9d5-4004-92dd-cb9572c5e65e"><b>94</b></span>, <span
                                    class="RefFPage" id="0c0c52f7-f985-4ce9-be56-bb71613b25f2">339</span>–<span
                                    class="RefLPage" id="496d0b7e-7671-451e-aade-be28aeb3f888">345</span>. (<span
                                    class="RefDOI"
                                    id="700b6864-52dd-4f95-ade4-031e4c10aa52">doi:10.1111/j.1439-0310.1993.tb00450.x</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R9" id="R9" data-reftype="webpage"
                                data-reftype-name="Website" data-csl="true" data-validated-log="false"
                                data-validated="false" new-id="R9"><span class="RefSlNo"
                                    id="816dbef8-d38e-4aef-a375-30af54cc3611">9.</span> AndrewsS. <span
                                    class="RefArticleTitle" id="f2a02109-7879-4591-9160-05e4455071f8"><i>FastQC: A
                                        quality control tool for high throughput sequence data</i></span>. <span
                                    class="RefComments" id="e273eb63-5158-45a1-abc1-29f1beea0021">See</span> <span
                                    class="RefWebSite"
                                    id="f362b749-8fd2-4a3c-9481-522b47efc91c">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span>.
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R10" id="R10"
                                data-reftype="article-journal" data-doi="10.1093/bioinformatics/btu170"
                                data-pmid="24695404" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R10"><span class="RefSlNo"
                                    id="f55ebc76-1dbf-4d52-929f-d9d9f3f74f28">10.</span> <span class="RefAuthor"
                                    id="bdc00dd8-073d-4793-8132-2fce06d4d313"><span class="RefSurName">Bolger</span>
                                    <span class="RefGivenName">AM</span></span>, <span class="RefAuthor"
                                    id="d67ab1c3-ed9c-4871-96ca-16d2c4eb3e69"><span class="RefSurName">Lohse</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="581f9356-3fa4-4465-929f-abd10ab73580"><span class="RefSurName">Usadel</span>
                                    <span class="RefGivenName">B</span></span>. <span class="RefYear"
                                    id="b11b2cd6-5966-492d-9b8a-4a7762506320">2014</span> <span class="RefArticleTitle"
                                    id="b8f12b1a-cf73-46b3-ad3a-2c89b1848ab5">Trimmomatic: a flexible trimmer for
                                    Illumina sequence data</span>. <span class="RefJournalTitle"
                                    id="63a9ccad-a3f4-4d96-9a14-06eb2a7cca88"><i>Bioinformatics</i></span> <span
                                    class="RefVolume" id="8d7cb0a8-a7a8-4afd-8749-d39bd8020729"><b>30</b></span>, <span
                                    class="RefFPage" id="ccd75813-0ce1-4940-a47b-c023f67da33b">2114</span>–<span
                                    class="RefLPage" id="056d873f-efd7-415e-b3d3-7ca28cf2beec">2120</span>. (<span
                                    class="RefDOI"
                                    id="6906cf47-c507-4f10-9713-2640d6071621">doi:10.1093/bioinformatics/btu170</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R11" id="R11"
                                data-reftype="article-journal" data-doi="10.1038/nprot.2013.084" data-pmid="23845962"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R11"><span class="RefSlNo" id="70a45120-a824-41fb-94c5-0df4c8e4c8f0">11.</span>
                                <span class="RefAuthor" id="62d726ad-4eb1-4051-ae6e-6f1c7abfaa9f"><span
                                        class="RefSurName">Haas</span> <span class="RefGivenName">BJ</span></span> <span
                                    class="RefEtal" id="e59722eb-cec3-4eff-81de-7ac1cb35f5b7"><i>et al</i></span>. <span
                                    class="RefYear" id="2b085e6b-3a2d-42fc-b059-2a7f371bca8a">2013</span> <span
                                    class="RefArticleTitle" id="b8af5817-8f1b-4727-809e-6183fff23568">De novo transcript
                                    sequence reconstruction from RNA-seq using the Trinity platform for reference
                                    generation and analysis</span>. <span class="RefJournalTitle"
                                    id="d6ba4b44-a756-49f6-a80e-3afbb31b8068"><i>Nat Protoc</i></span> <span
                                    class="RefVolume" id="707f759b-433a-4d35-88be-4a4f6edba757"><b>8</b></span>, <span
                                    class="RefFPage" id="7349f39f-db55-4820-a789-3762e403bdd8">1494</span>–<span
                                    class="RefLPage" id="15b3d667-3634-4df4-afe2-d37d49a32fc2">1512</span>. (<span
                                    class="RefDOI"
                                    id="5950e6d1-62c5-4be5-8787-ac95dd806af9">doi:10.1038/nprot.2013.084</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R12" id="R12"
                                data-reftype="article-journal" data-doi="10.1093/bioinformatics/bts565"
                                data-pmid="23060610" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R12"><span class="RefSlNo"
                                    id="dd55a66a-542e-4eb2-802f-87bca326c497">12.</span> <span class="RefAuthor"
                                    id="1b60dc79-883a-4444-8415-8c76cf12bb71"><span class="RefSurName">Fu</span> <span
                                        class="RefGivenName">L</span></span>, <span class="RefAuthor"
                                    id="1a6f3ca4-6187-469f-90de-fe67b65441ec"><span class="RefSurName">Niu</span> <span
                                        class="RefGivenName">B</span></span>, <span class="RefAuthor"
                                    id="9ea49482-9c50-4167-8e04-42eb647ca6b7"><span class="RefSurName">Zhu</span> <span
                                        class="RefGivenName">Z</span></span>, <span class="RefAuthor"
                                    id="73e94313-103a-46c0-b9f9-3003d7ade1f9"><span class="RefSurName">Wu</span> <span
                                        class="RefGivenName">S</span></span>, <span class="RefAuthor"
                                    id="7ad0161b-a875-4289-8559-4f772e2403b0"><span class="RefSurName">Li</span> <span
                                        class="RefGivenName">W</span></span>. <span class="RefYear"
                                    id="132610c1-fb34-4abf-95b0-8d8dce7c6883">2012</span> <span class="RefArticleTitle"
                                    id="8ec3d3ea-0f93-4938-9741-abc633d5e150">CD-HIT: accelerated for clustering the
                                    next-generation sequencing data</span>. <span class="RefJournalTitle"
                                    id="93f3e826-b957-49e8-88e3-23a4f7b95119"><i>Bioinformatics</i></span> <span
                                    class="RefVolume" id="faca394a-d25b-4c07-a2b8-f5b7c6e84990"><b>28</b></span>, <span
                                    class="RefFPage" id="88bc3869-d801-47e7-8676-4218325ab6a8">3150</span>–<span
                                    class="RefLPage" id="dc73739b-2c68-4655-9ee4-7e4cb7114b9b">3152</span>. (<span
                                    class="RefDOI"
                                    id="96dfb716-3ddb-40c8-b1f0-d2b612cda4e6">doi:10.1093/bioinformatics/bts565</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R13" id="R13"
                                data-reftype="article-journal" data-doi="10.1093/molbev/msx319" data-pmid="29220515"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R13"><span class="RefSlNo" id="cf3f0048-4b85-4a85-8aa8-b578fb532ffb">13.</span>
                                <span class="RefAuthor" id="3fa94901-3363-463f-88ea-c1185c8fffa9"><span
                                        class="RefSurName">Waterhouse</span> <span
                                        class="RefGivenName">RM</span></span>, <span class="RefAuthor"
                                    id="219a7f2f-12df-4a01-9b19-b2d9f91a89ad"><span class="RefSurName">Seppey</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="767d6ba4-35b8-4e35-ba5e-bd43cb396875"><span class="RefSurName">Simão</span>
                                    <span class="RefGivenName">FA</span></span>, <span class="RefAuthor"
                                    id="fd80ce2b-660b-4793-83ce-3bde75b0c9b4"><span class="RefSurName">Manni</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="77b250d4-aad0-4798-845b-03b8428db20d"><span class="RefSurName">Ioannidis</span>
                                    <span class="RefGivenName">P</span></span>, <span class="RefAuthor"
                                    id="6615102a-2715-4477-a15b-cc3d35a2f6a9"><span
                                        class="RefSurName">Klioutchnikov</span> <span
                                        class="RefGivenName">G</span></span>, <span class="RefAuthor"
                                    id="9a6e00a6-066c-485a-af7c-05a058c0fcd8"><span
                                        class="RefSurName">Kriventseva</span> <span
                                        class="RefGivenName">EV</span></span>, <span class="RefAuthor"
                                    id="bc11f0bf-3891-4475-8aa0-b2b051ce6519"><span class="RefSurName">Zdobnov</span>
                                    <span class="RefGivenName">EM</span></span>. <span class="RefYear"
                                    id="2a9ca48a-7b12-4218-9ab0-4784c562dd3a">2018</span> <span class="RefArticleTitle"
                                    id="ccbd9a70-cd1d-44dd-8ce8-06e043e96eef">BUSCO Applications from Quality
                                    Assessments to Gene Prediction and Phylogenomics</span>. <span
                                    class="RefJournalTitle" id="150e3357-e65e-4c0a-8ad3-9653493576a4"><i>Mol Biol
                                        Evol</i></span> <span class="RefVolume"
                                    id="6aa97a5c-1443-4efb-8120-f54ce140cea3"><b>35</b></span>, <span class="RefFPage"
                                    id="a7e932a0-34dd-40e0-b5db-18f3e722d567">543</span>–<span class="RefLPage"
                                    id="b93a1b62-ca91-45e5-b0bd-ab3340abd1e7">548</span>. (<span class="RefDOI"
                                    id="fe9ccb3a-b9e9-4c1a-96ee-05a5dddccddc">doi:10.1093/molbev/msx319</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R14" id="R14"
                                data-reftype="article-journal" data-doi="10.1093/molbev/msab293" data-pmid="34597405"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R14"><span class="RefSlNo" id="7a00c99a-4a9c-4c67-8cfa-3535d736ed17">14.</span>
                                <span class="RefAuthor" id="e2cc6744-53e3-485b-b1bf-693e89ad464a"><span
                                        class="RefSurName">Cantalapiedra</span> <span
                                        class="RefGivenName">CP</span></span>, <span class="RefAuthor"
                                    id="b4ea3b70-1d13-43a2-8cbb-198597976a30"><span
                                        class="RefSurName">Hernández-Plaza</span> <span
                                        class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="c5733f50-656c-4ec3-8c6e-26d31b5996e8"><span class="RefSurName">Letunic</span>
                                    <span class="RefGivenName">I</span></span>, <span class="RefAuthor"
                                    id="174d644a-688c-475a-96f6-e14d38284978"><span class="RefSurName">Bork</span> <span
                                        class="RefGivenName">P</span></span>, <span class="RefAuthor"
                                    id="86d127b9-bd43-47dd-8b78-f2844d94ff7b"><span
                                        class="RefSurName">Huerta-Cepas</span> <span
                                        class="RefGivenName">J</span></span>. <span class="RefYear"
                                    id="c36034f8-1f04-43a2-a74f-e4d12a7d9f32">2021</span> <span class="RefArticleTitle"
                                    id="dcd1bf78-d02b-449c-95a2-9008c1095a65">eggNOG-mapper v2: Functional Annotation,
                                    Orthology Assignments, and Domain Prediction at the Metagenomic Scale</span>. <span
                                    class="RefJournalTitle" id="d7334b3e-8510-4238-b1d5-2c2505766216"><i>Mol Biol
                                        Evol</i></span> <span class="RefVolume"
                                    id="9bf4dc25-4b1e-411c-bf79-9d46a6e1db7b"><b>38</b></span>, <span class="RefFPage"
                                    id="6b154c7a-99b2-4159-a2ee-63d1d01596e8">5825</span>–<span class="RefLPage"
                                    id="87322f1c-860c-4f96-a5cd-7d6eb8b5109d">5829</span>. (<span class="RefDOI"
                                    id="649bc440-0569-4130-830b-e23ebce792ef">doi:10.1093/molbev/msab293</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R15" id="R15"
                                data-reftype="article-journal" data-doi="10.1186/1471-2105-12-323" data-pmid="21816040"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R15"><span class="RefSlNo" id="b2eb943f-82e6-415c-90f5-d638b5561fb9">15.</span>
                                <span class="RefAuthor" id="deafdbb3-ec09-4704-9c0a-5ef251399312"><span
                                        class="RefSurName">Li</span> <span class="RefGivenName">B</span></span>, <span
                                    class="RefAuthor" id="7c638a76-4ef3-4a6a-a272-3d65e595c861"><span
                                        class="RefSurName">Dewey</span> <span class="RefGivenName">CN</span></span>.
                                <span class="RefYear" id="38471885-8df4-41ae-8bf6-7803ce3776bf">2011</span> <span
                                    class="RefArticleTitle" id="6036f0db-37c1-4c6c-b4e6-c6e3cc5f824e">RSEM: accurate
                                    transcript quantification from RNA-Seq data with or without a reference
                                    genome</span>. <span class="RefJournalTitle"
                                    id="7a412a35-1ddc-4ac2-92be-1cb5b37242e9"><i>BMC Bioinformatics</i></span> <span
                                    class="RefVolume" id="278f7528-9143-4fb7-ac31-7c52cff7f8b3"><b>12</b></span>, <span
                                    class="RefELocation" id="e93e4585-2377-4150-aa90-d3f06c016d44">323</span>. (<span
                                    class="RefDOI"
                                    id="75fc9b44-3efe-4b24-8671-9597e2e8bd04">doi:10.1186/1471-2105-12-323</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R16" id="R16"
                                data-reftype="article-journal" data-doi="10.1038/nmeth.1923" data-pmid="22388286"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R16"><span class="RefSlNo" id="a14636e8-0096-4260-8fc2-9fee0e642a98">16.</span>
                                <span class="RefAuthor" id="0f8eb95d-5267-4d67-8573-570f864dbd09"><span
                                        class="RefSurName">Langmead</span> <span class="RefGivenName">B</span></span>,
                                <span class="RefAuthor" id="097e0a94-5b08-47f0-bd53-b485dc359157"><span
                                        class="RefSurName">Salzberg</span> <span class="RefGivenName">SL</span></span>.
                                <span class="RefYear" id="59c740e9-a154-4a8c-a019-8c453df11a62">2012</span> <span
                                    class="RefArticleTitle" id="2335afac-1c35-4d9b-bab8-63fd4d9f4698">Fast gapped-read
                                    alignment with Bowtie 2</span>. <span class="RefJournalTitle"
                                    id="fbeb4363-9d7f-4572-83c1-c667c4a03541"><i>Nat Methods</i></span> <span
                                    class="RefVolume" id="47dbd686-f183-45f0-b9c6-43eb4e481a0a"><b>9</b></span>, <span
                                    class="RefFPage" id="25a5268b-7503-43ba-a987-4ea2eb89c5be">357</span>–<span
                                    class="RefLPage" id="5b45d81f-c610-48e1-9681-3a888718c6dd">359</span>. (<span
                                    class="RefDOI"
                                    id="9e978f5a-e7dd-4ca2-bb52-3b276c153237">doi:10.1038/nmeth.1923</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R17" id="R17"
                                data-reftype="article-journal" data-doi="10.1186/s13059-014-0550-8" data-pmid="25516281"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R17"><span class="RefSlNo" id="2ddbca93-5b26-4140-827e-2cf3bd974ca8">17.</span>
                                <span class="RefAuthor" id="4b72eb62-2499-4b1f-919b-6c7efe96ce56"><span
                                        class="RefSurName">Love</span> <span class="RefGivenName">MI</span></span>,
                                <span class="RefAuthor" id="464f8bad-6482-41fe-abd5-66f1abeb66e7"><span
                                        class="RefSurName">Huber</span> <span class="RefGivenName">W</span></span>,
                                <span class="RefAuthor" id="c7a3881d-5734-47e3-b089-7c8c08d48605"><span
                                        class="RefSurName">Anders</span> <span class="RefGivenName">S</span></span>.
                                <span class="RefYear" id="b8dce65b-65e5-4686-8863-22e28afe6185">2014</span> <span
                                    class="RefArticleTitle" id="afa1e106-2e94-44ae-8caf-f074b225c056">Moderated
                                    estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <span
                                    class="RefJournalTitle" id="4195bad4-3181-40a1-b7c9-da316df55a4f"><i>Genome
                                        Biol</i></span> <span class="RefVolume"
                                    id="f472eb76-71f4-4a04-aef6-e5f4f03311ba"><b>15</b></span>, <span class="RefFPage"
                                    id="2c4703bd-e49a-4442-a952-9a88398943ba">550</span>. (<span class="RefDOI"
                                    id="48e07431-90ee-4afe-92a5-e082321b786d">doi:10.1186/s13059-014-0550-8</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R18" id="R18"
                                data-reftype="article-journal" data-doi="10.1186/gb-2010-11-2-r14" data-pmid="20132535"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R18"><span class="RefSlNo" id="c4b1a1db-df8f-4e41-9bf7-63a62d5276d5">18.</span>
                                <span class="RefAuthor" id="4c13ca38-e6bf-4421-8581-d67eb107c408"><span
                                        class="RefSurName">Young</span> <span class="RefGivenName">MD</span></span>,
                                <span class="RefAuthor" id="a7e8fdf8-a0d1-4c8e-ac9d-11b4e0c11e12"><span
                                        class="RefSurName">Wakefield</span> <span class="RefGivenName">MJ</span></span>,
                                <span class="RefAuthor" id="6adeb704-39d0-494f-ad5c-9d17153333f1"><span
                                        class="RefSurName">Smyth</span> <span class="RefGivenName">GK</span></span>,
                                <span class="RefAuthor" id="bd17b678-2384-404d-b041-b0ae627ae5e3"><span
                                        class="RefSurName">Oshlack</span> <span class="RefGivenName">A</span></span>.
                                <span class="RefYear" id="d946a828-8bb9-4089-b5af-1bcc2c77cb94">2010</span> <span
                                    class="RefArticleTitle" id="24e49d8a-da28-4810-b80c-df2d47ac3c92">Gene ontology
                                    analysis for RNA-seq: accounting for selection bias</span>. <span
                                    class="RefJournalTitle" id="5c438346-5200-46e7-a7c8-4a70de72fe54"><i>Genome
                                        Biol</i></span> <span class="RefVolume"
                                    id="4d4f8281-e434-4559-a87a-e32bedfd6341"><b>11</b></span>, <span
                                    class="RefELocation" id="c4700790-5dc1-4728-85c8-435834f1d834">R14</span>. (<span
                                    class="RefDOI"
                                    id="168d4876-ed90-49f7-96ff-846720126165">doi:10.1186/gb-2010-11-2-r14</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R19" id="R19"
                                data-reftype="article-journal" data-doi="10.1111/j.2517-6161.1995.tb02031.x"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R19"><span class="RefSlNo"
                                    id="a9461a59-81c9-4547-8e94-fb118c618b03">19.</span> BenjaminiY., HochbergY. <span
                                    class="RefYear" id="338ed92a-5b4b-48ae-b5ac-920ae0c07131">1995</span> <span
                                    class="RefArticleTitle" id="99e4f61d-f999-46be-96fb-d48f2d575094">Controlling the
                                    False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>.
                                <span class="RefJournalTitle" id="6a912981-345d-4e42-8674-07a459a74369"><i>Journal of
                                        the Royal Statistical Society Series B</i></span> <span class="RefVolume"
                                    id="8b6d64c7-e003-457f-ba52-8992e4dddbfb"><b>57</b></span>, <span class="RefFPage"
                                    id="63ce834c-eafa-4937-930b-32e993e4bb30">289</span>–<span class="RefLPage"
                                    id="ad7948d0-8871-48dd-b987-e894a2bba7dd">300</span>. (<span class="RefDOI"
                                    id="21c8babf-3ce1-49da-b83b-7ca127df83c1">doi:10.1111/j.2517-6161.1995.tb02031.x</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R20" id="R20"
                                data-reftype="article-journal" data-doi="10.1038/srep35461" data-pmid="27748421"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R20"><span class="RefSlNo" id="31215f96-12d7-4144-bae9-e1fb32923611">20.</span>
                                <span class="RefAuthor" id="693780d0-22bb-4272-ae42-f4868b03416b"><span
                                        class="RefSurName">Casas</span> <span class="RefGivenName">L</span></span>,
                                <span class="RefAuthor" id="82d32814-ae87-47a7-9ec6-1586f841c9d9"><span
                                        class="RefSurName">Saborido-Rey</span> <span
                                        class="RefGivenName">F</span></span>, <span class="RefAuthor"
                                    id="6d31b0e2-6bd2-4aab-b6f6-4a2dda3cdb8c"><span class="RefSurName">Ryu</span> <span
                                        class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="bbe95744-668f-4390-87a6-0fb4bae31d76"><span class="RefSurName">Michell</span>
                                    <span class="RefGivenName">C</span></span>, <span class="RefAuthor"
                                    id="e7736082-92b3-4110-8e3d-6b0ece3546a1"><span class="RefSurName">Ravasi</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="8770aa13-c830-4bfe-bd37-00920b2fc8c9"><span class="RefSurName">Irigoien</span>
                                    <span class="RefGivenName">X</span></span>. <span class="RefYear"
                                    id="35bb7ea7-561f-4a17-8f2d-5277283251e5">2016</span> <span class="RefArticleTitle"
                                    id="97cf619d-714d-4896-bf42-59f2258ca024">Sex Change in Clownfish: Molecular
                                    Insights from Transcriptome Analysis</span>. <span class="RefJournalTitle"
                                    id="688b9acd-8214-49b8-9abc-ead3d72686db"><i>Sci Rep</i></span> <span
                                    class="RefVolume" id="7a7b95ee-63bb-4f24-b2c7-77379a4fecc5"><b>6</b></span>, <span
                                    class="RefFPage" id="335ea4b0-33e2-4c73-8c09-c4b6cf83c495">35461</span>. (<span
                                    class="RefDOI"
                                    id="9bb56d1a-63b8-4090-8f24-9232c1694c06">doi:10.1038/srep35461</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R21" id="R21"
                                data-reftype="article-journal" data-doi="10.1126/sciadv.aaw7006" data-pmid="31309157"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R21"><span class="RefSlNo" id="a8c868f6-c65b-4a7b-8276-fafdf50e1d0f">21.</span>
                                <span class="RefAuthor" id="20f17a59-579d-4386-9c8a-02ac17106a42"><span
                                        class="RefSurName">Todd</span> <span class="RefGivenName">EV</span></span> <span
                                    class="RefEtal" id="e21dac24-6fbe-4f13-86d8-750a732c5476"><i>et al</i></span>. <span
                                    class="RefYear" id="38604749-495e-4f58-9d1a-d841f2f9bc15">2019</span> <span
                                    class="RefArticleTitle" id="1c516689-1065-4bf5-8fe5-c4c0106a19e1">Stress, novel sex
                                    genes, and epigenetic reprogramming orchestrate socially controlled sex
                                    change</span>. <span class="RefJournalTitle"
                                    id="b2b98c0b-e12c-4762-90e2-aa57eb3dab96"><i>Sci Adv</i></span> <span
                                    class="RefVolume" id="e97c5cab-ba3f-4f8d-ba77-7b0ea8f94bd2"><b>5</b></span>, <span
                                    class="RefELocation" id="b73e1c87-876c-47b1-9b78-726cac8d414a">eaaw7006</span>.
                                (<span class="RefDOI"
                                    id="69ab27fb-68d4-42aa-b9a6-79e2fb7589f1">doi:10.1126/sciadv.aaw7006</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R22" id="R22"
                                data-reftype="article-journal" data-doi="10.1186/s12864-017-3652-3" data-pmid="28388875"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R22"><span class="RefSlNo" id="dfb44b30-4d78-47dc-941f-22ea757c18a6">22.</span>
                                <span class="RefAuthor" id="7ae80f1a-292d-4264-a266-7fa2738ab852"><span
                                        class="RefSurName">Cribbin</span> <span class="RefGivenName">KM</span></span>,
                                <span class="RefAuthor" id="7bccc818-1427-4f22-b42d-9b201d566e16"><span
                                        class="RefSurName">Quackenbush</span> <span
                                        class="RefGivenName">CR</span></span>, <span class="RefAuthor"
                                    id="28f9b642-9ccf-4437-8b86-2f236acc1385"><span class="RefSurName">Taylor</span>
                                    <span class="RefGivenName">K</span></span>, <span class="RefAuthor"
                                    id="75e82e4d-0775-46d1-b4ac-562e9e0611b8"><span
                                        class="RefSurName">Arias-Rodriguez</span> <span
                                        class="RefGivenName">L</span></span>, <span class="RefAuthor"
                                    id="e4858f09-9ee0-4905-8a3a-c24c177ac8a9"><span class="RefSurName">Kelley</span>
                                    <span class="RefGivenName">JL</span></span>. <span class="RefYear"
                                    id="dcc01fef-826e-47ab-9037-732df9cd0a01">2017</span> <span class="RefArticleTitle"
                                    id="460eff84-9e7b-46d3-b735-8f3061c807d3">Sex-specific differences in transcriptome
                                    profiles of brain and muscle tissue of the tropical gar</span>. <span
                                    class="RefJournalTitle" id="8b5ac5d2-d9a3-4127-a18c-7cb7259cdf41"><i>BMC
                                        Genomics</i></span> <span class="RefVolume"
                                    id="1972d712-dedd-43f3-a34e-8ecf2166e859"><b>18</b></span>, <span class="RefFPage"
                                    id="26d14ee9-cf0e-421a-bb9f-0f6144bccba5">283</span>. (<span class="RefDOI"
                                    id="f61f429f-2c0e-40c9-bdc4-5c41db65e9e4">doi:10.1186/s12864-017-3652-3</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R23" id="R23"
                                data-reftype="article-journal" data-doi="10.1242/dev.161356" data-pmid="29891564"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R23"><span class="RefSlNo" id="bb08eef8-4f3c-4e59-ba02-daede8c18d84">23.</span>
                                <span class="RefAuthor" id="4e047d6b-e4b5-4258-9bf2-92311595cb80"><span
                                        class="RefSurName">Neuman</span> <span class="RefGivenName">SD</span></span>,
                                <span class="RefAuthor" id="ead52318-7d6b-4065-8cf8-cb450c1934b4"><span
                                        class="RefSurName">Bashirullah</span> <span
                                        class="RefGivenName">A</span></span>. <span class="RefYear"
                                    id="14ab9c4a-7719-473a-bf79-7791568d7f2e">2018</span> <span class="RefArticleTitle"
                                    id="10bdb678-adca-4fcd-ae5d-90cfe511860d">Hobbit regulates intracellular trafficking
                                    to drive insulin-dependent growth during <i>Drosophila</i> development</span>. <span
                                    class="RefJournalTitle"
                                    id="66220808-d349-44f8-ba9c-abd8c4d8779a"><i>Development</i></span> <span
                                    class="RefVolume" id="3d11e652-b400-455e-aeeb-3f40d0ceeb45"><b>145</b></span>, <span
                                    class="RefELocation" id="0495f4ce-7ef6-4df3-a45f-6cf969b7737b">dev161356</span>.
                                (<span class="RefDOI"
                                    id="15edf142-5a41-4f22-a815-563cc2e2cd65">doi:10.1242/dev.161356</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R24" id="R24"
                                data-reftype="article-journal" data-doi="10.1038/s41598-017-08248-8"
                                data-pmid="28801581" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R24"><span class="RefSlNo"
                                    id="fa365706-7926-4f40-bc71-13b2cd023e96">24.</span> <span class="RefAuthor"
                                    id="53f9f6f8-680d-4414-8c7c-b50e90848c88"><span class="RefSurName">Moriya</span>
                                    <span class="RefGivenName">S</span></span>, <span class="RefAuthor"
                                    id="27561667-a4e3-41a5-b2b0-ba1b289f9369"><span class="RefSurName">Tahsin</span>
                                    <span class="RefGivenName">N</span></span>, <span class="RefAuthor"
                                    id="986b248c-2e42-4f1d-a531-5f42314d9ac5"><span class="RefSurName">Parhar</span>
                                    <span class="RefGivenName">IS</span></span>. <span class="RefYear"
                                    id="af56bdcd-3db9-4a80-8f14-d93c89c58e1d">2017</span> <span class="RefArticleTitle"
                                    id="86edb5b7-559a-4876-9998-a7434f75106a">Bpifcl modulates kiss2 expression under
                                    the influence of 11-ketotestosterone in female zebrafish</span>. <span
                                    class="RefJournalTitle" id="2d0ce6d4-1912-4d03-be6b-1988f18e1aa4"><i>Sci
                                        Rep</i></span> <span class="RefVolume"
                                    id="03fb2ffc-986b-49a1-bf94-58c73c8d56fa"><b>7</b></span>, <span class="RefFPage"
                                    id="f7fe2f89-a53a-4bed-aa9b-ac48274b5586">7926</span>. (<span class="RefDOI"
                                    id="094e162d-3c5e-405d-a80e-4fe5d36ed6dd">doi:10.1038/s41598-017-08248-8</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R25" id="R25"
                                data-reftype="article-journal" data-doi="10.1371/journal.pone.0062776"
                                data-pmid="23638144" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R25"><span class="RefSlNo"
                                    id="d5701078-41be-4bb6-af2c-6dd02ba472de">25.</span> <span class="RefAuthor"
                                    id="feed1096-5999-4596-9bf6-3090e30dbd0f"><span class="RefSurName">Kanda</span>
                                    <span class="RefGivenName">S</span></span>, <span class="RefAuthor"
                                    id="e3ad25d7-96c2-439a-bf0c-eda110be7d65"><span class="RefSurName">Akazome</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="6ad2e65f-9b81-4d65-ab37-ea6b83c9f24a"><span class="RefSurName">Mitani</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="a86bb3a8-266b-4748-ba25-f756a5ba5aa6"><span class="RefSurName">Okubo</span>
                                    <span class="RefGivenName">K</span></span>, <span class="RefAuthor"
                                    id="f26f7f4c-7907-4f60-a028-011b39ed3772"><span class="RefSurName">Oka</span> <span
                                        class="RefGivenName">Y</span></span>. <span class="RefYear"
                                    id="cc5af983-fb2d-4b81-be78-eae365aa663f">2013</span> <span class="RefArticleTitle"
                                    id="10b7b473-3488-49a0-a29a-7d0a27bfecc3">Neuroanatomical evidence that kisspeptin
                                    directly regulates isotocin and vasotocin neurons</span>. <span
                                    class="RefJournalTitle" id="9b7b9be8-b916-43b6-8e8e-d6f45787824c"><i>PLoS
                                        One</i></span> <span class="RefVolume"
                                    id="cceec36f-ddd7-41aa-a505-e1b88af82ac1"><b>8</b></span>, <span
                                    class="RefELocation" id="e53bbd82-0224-4336-b1a0-44dcd2fabf5d">e62776</span>. (<span
                                    class="RefDOI"
                                    id="5ddac018-7b96-4b8b-8f00-9d62c135250b">doi:10.1371/journal.pone.0062776</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R26" id="R26"
                                data-reftype="article-journal" data-doi="10.1093/icb/icv041" data-pmid="25980565"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R26"><span class="RefSlNo" id="20e3263a-9f5b-410e-ab29-f43d5f40f7a8">26.</span>
                                <span class="RefAuthor" id="e5d0741f-ee32-4f39-b6f5-e26875a2f043"><span
                                        class="RefSurName">Lamm</span> <span class="RefGivenName">MS</span></span>,
                                <span class="RefAuthor" id="be43d9e8-754f-4ffb-8fe9-04b2af152036"><span
                                        class="RefSurName">Liu</span> <span class="RefGivenName">H</span></span>, <span
                                    class="RefAuthor" id="949ee115-728c-4401-99e0-41a8704b8d67"><span
                                        class="RefSurName">Gemmell</span> <span class="RefGivenName">NJ</span></span>,
                                <span class="RefAuthor" id="84c17019-afde-4ced-9291-66f7ad3f1050"><span
                                        class="RefSurName">Godwin</span> <span class="RefGivenName">JR</span></span>.
                                <span class="RefYear" id="db3f0ae3-bb97-4bc3-b735-4d0fff7917aa">2015</span> <span
                                    class="RefArticleTitle" id="ff11f9da-df48-4b1c-a044-deae9fd304e2">The Need for
                                    Speed: Neuroendocrine Regulation of Socially-controlled Sex Change</span>. <span
                                    class="RefJournalTitle" id="8e5de37f-2d87-4fe2-9038-a9b4c717b4ac"><i>Integr Comp
                                        Biol</i></span> <span class="RefVolume"
                                    id="4ddfb3f5-087a-4e20-8cd6-8367d6acc0d1"><b>55</b></span>, <span class="RefFPage"
                                    id="3226c7d7-84ec-42f2-b910-c2cfd0e6f2d6">307</span>–<span class="RefLPage"
                                    id="429f6a22-2a45-484a-87e3-0abf820b14ad">322</span>. (<span class="RefDOI"
                                    id="bc1d4449-a9a1-490f-a380-a7ac201b033a">doi:10.1093/icb/icv041</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R27" id="R27"
                                data-reftype="article-journal" data-doi="10.1038/nature13921" data-pmid="25519132"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R27"><span class="RefSlNo" id="30e7e14d-7bca-4faf-82c1-450f7218334c">27.</span>
                                <span class="RefAuthor" id="1f93bcd8-80b3-4d0b-a5cf-fbfcb485c923"><span
                                        class="RefSurName">Gao</span> <span class="RefGivenName">Z</span></span>, <span
                                    class="RefAuthor" id="7f1957b5-f232-47b2-9493-ab17af72444e"><span
                                        class="RefSurName">Lee</span> <span class="RefGivenName">P</span></span>, <span
                                    class="RefAuthor" id="d93c0540-b915-4746-8a7c-ede904a61c42"><span
                                        class="RefSurName">Stafford</span> <span class="RefGivenName">JM</span></span>,
                                <span class="RefAuthor" id="a5ed4d66-e223-42d4-8e67-d167c599384f"><span
                                        class="RefSurName">von Schimmelmann</span> <span
                                        class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="851bbf50-5ab4-444c-b821-2033ff8931f1"><span class="RefSurName">Schaefer</span>
                                    <span class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="628a880e-0c22-4336-92a0-54e324780d9a"><span class="RefSurName">Reinberg</span>
                                    <span class="RefGivenName">D</span></span>. <span class="RefYear"
                                    id="cf24d28e-e259-47b2-9ccb-5bc2fc4cc75f">2014</span> <span class="RefArticleTitle"
                                    id="f9c6e33f-e986-4fb2-b49d-ffeb48e6f620">An AUTS2-Polycomb complex activates gene
                                    expression in the CNS</span>. <span class="RefJournalTitle"
                                    id="ca14622b-6103-422f-9798-fcabf98c75eb"><i>Nature</i></span> <span
                                    class="RefVolume" id="4d8bb5f1-8f0c-474a-99ea-f5d187697885"><b>516</b></span>, <span
                                    class="RefFPage" id="d54fd25a-bed1-4a32-af05-8fc0a78e0306">349</span>–<span
                                    class="RefLPage" id="98b3c616-c52a-4ef6-9778-acdd4998c51f">354</span>. (<span
                                    class="RefDOI"
                                    id="a6df1554-1cd0-4894-8470-10973e921172">doi:10.1038/nature13921</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R28" id="R28"
                                data-reftype="article-journal" data-doi="10.1038/s41467-018-06459-9"
                                data-pmid="30305620" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R28"><span class="RefSlNo"
                                    id="1fa574f8-e19e-4b1f-aa1b-d9894d5a193a">28.</span> <span class="RefAuthor"
                                    id="3f2385e7-672f-43d4-a61a-6c8498ae5593"><span class="RefSurName">Vazquez</span>
                                    <span class="RefGivenName">MJ</span></span> <span class="RefEtal"
                                    id="ee91624a-db22-44f0-90ae-5124a6b97bf9"><i>et al</i></span>. <span class="RefYear"
                                    id="70e6e077-42ae-47f2-ace4-22bb66c6b77f">2018</span> <span class="RefArticleTitle"
                                    id="4fcaa566-7e6c-4f6d-9518-55bd5ac10d3f">SIRT1 mediates obesity- and
                                    nutrient-dependent perturbation of pubertal timing by epigenetically controlling
                                    Kiss1 expression</span>. <span class="RefJournalTitle"
                                    id="8e0aeeea-3753-4c28-a5d4-b76549ea5bab"><i>Nat Commun</i></span> <span
                                    class="RefVolume" id="6bc44ab8-524c-4486-975e-aec7e21e297a"><b>9</b></span>, <span
                                    class="RefFPage" id="10549c0b-839e-4161-8670-c961fac95bb1">4194</span>. (<span
                                    class="RefDOI"
                                    id="4fdc31cf-0fa0-4c91-9816-73a15ecf27c9">doi:10.1038/s41467-018-06459-9</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R29" id="R29"
                                data-reftype="article-journal" data-doi="10.1038/nn.3420" data-pmid="23727821"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R29"><span class="RefSlNo" id="7df7600f-c206-429a-a3ff-dafcca346e7a">29.</span>
                                <span class="RefAuthor" id="b57b921a-25a5-47f6-a3ed-b9535d90f52e"><span
                                        class="RefSurName">Wang</span> <span class="RefGivenName">H</span></span>, <span
                                    class="RefAuthor" id="82330d2b-a2c3-44d0-9be6-84f7c9a7f128"><span
                                        class="RefSurName">Duclot</span> <span class="RefGivenName">F</span></span>,
                                <span class="RefAuthor" id="321c4cdc-9b93-46ad-9fa4-073ac5ad0efd"><span
                                        class="RefSurName">Liu</span> <span class="RefGivenName">Y</span></span>, <span
                                    class="RefAuthor" id="4f738729-b784-4c4e-a7b1-5084c8deee83"><span
                                        class="RefSurName">Wang</span> <span class="RefGivenName">Z</span></span>, <span
                                    class="RefAuthor" id="4586e2a5-7458-4475-94bb-7c3db13dd4d4"><span
                                        class="RefSurName">Kabbaj</span> <span class="RefGivenName">M</span></span>.
                                <span class="RefYear" id="1bc53e34-5490-4ca7-ad90-0a2b7b6b6507">2013</span> <span
                                    class="RefArticleTitle" id="f47fec49-07c6-4aae-a1db-f81d52fa9a6f">Histone
                                    deacetylase inhibitors facilitate partner preference formation in female prairie
                                    voles</span>. <span class="RefJournalTitle"
                                    id="dff65cc0-ce38-4798-a46a-9add301b0281"><i>Nat Neurosci</i></span> <span
                                    class="RefVolume" id="cafd2a37-6576-4ef9-a023-1a04a85981b7"><b>16</b></span>, <span
                                    class="RefFPage" id="ec4823a6-2c38-4f00-aca7-043a0c37e6d9">919</span>–<span
                                    class="RefLPage" id="766ae6a2-0ba0-4603-9994-36d8cd9d4f35">924</span>. (<span
                                    class="RefDOI" id="2eae4bb2-5546-4d33-a845-b948d2d7a7bf">doi:10.1038/nn.3420</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R30" id="R30"
                                data-reftype="article-journal" data-doi="10.1126/science.1239864" data-pmid="24009392"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R30"><span class="RefSlNo" id="8199f2c7-cca3-4f8f-ae9b-e586a368ab8b">30.</span>
                                <span class="RefAuthor" id="682e5be4-c141-4341-81b0-66f999f0fdf7"><span
                                        class="RefSurName">Kuroki</span> <span class="RefGivenName">S</span></span>
                                <span class="RefEtal" id="07c5d184-4353-4fb5-ad91-dae21f1d6043"><i>et al</i></span>.
                                <span class="RefYear" id="7e2a6212-41cc-4403-a2dc-3c76406a03ab">2013</span> <span
                                    class="RefArticleTitle" id="2a203f59-4326-4f06-82b3-bd5ce50ae7eb">Epigenetic
                                    regulation of mouse sex determination by the histone demethylase Jmjd1a</span>.
                                <span class="RefJournalTitle"
                                    id="d8714173-741d-4769-ab1d-969d87b132c8"><i>Science</i></span> <span
                                    class="RefVolume" id="b60615c9-2a97-4215-ab7c-4f691c684bc7"><b>341</b></span>, <span
                                    class="RefFPage" id="b4b7bcb5-7750-48b6-af9e-f9b1f249c219">1106</span>–<span
                                    class="RefLPage" id="bba57eef-74fd-4cd9-8128-c83afb5ab887">1109</span>. (<span
                                    class="RefDOI"
                                    id="6f6b81ae-ae40-450d-816e-662a81f1fe54">doi:10.1126/science.1239864</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R31" id="R31"
                                data-reftype="article-journal" data-doi="10.1186/s13293-015-0044-8" data-pmid="26613014"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R31"><span class="RefSlNo" id="46bb4ba4-91f3-45d3-a775-76060c3815bb">31.</span>
                                <span class="RefAuthor" id="3fd34b45-79cd-4150-89c9-33fcef6c4b74"><span
                                        class="RefSurName">Liu</span> <span class="RefGivenName">H</span></span>, <span
                                    class="RefAuthor" id="9bd9473b-1c5e-447e-a4ed-d01273dcb8b0"><span
                                        class="RefSurName">Lamm</span> <span class="RefGivenName">MS</span></span>,
                                <span class="RefAuthor" id="631067e3-69fe-4da5-b0ce-00e10d95f23e"><span
                                        class="RefSurName">Rutherford</span> <span class="RefGivenName">K</span></span>,
                                <span class="RefAuthor" id="42b0c5ff-857f-46bd-84c1-4c14089b8fb2"><span
                                        class="RefSurName">Black</span> <span class="RefGivenName">MA</span></span>,
                                <span class="RefAuthor" id="3e652b68-5e51-42f3-a801-8812dcf811f7"><span
                                        class="RefSurName">Godwin</span> <span class="RefGivenName">JR</span></span>,
                                <span class="RefAuthor" id="fa1b0a30-0e5f-4c07-acaf-46d73904d1dc"><span
                                        class="RefSurName">Gemmell</span> <span class="RefGivenName">NJ</span></span>.
                                <span class="RefYear" id="4b7e6b87-e3bf-494c-89fd-62ab0b3d86d2">2015</span> <span
                                    class="RefArticleTitle" id="e389d2bb-fdff-426a-972c-b62d122e86d5">Large-scale
                                    transcriptome sequencing reveals novel expression patterns for key sex-related genes
                                    in a sex-changing fish</span>. <span class="RefJournalTitle"
                                    id="65d0fbc8-8905-4755-96fe-636ebc8d4e21"><i>Biol Sex Differ</i></span> <span
                                    class="RefVolume" id="59d68975-6949-4fb9-b389-25a2d7dc8a52"><b>6</b></span>, <span
                                    class="RefFPage" id="92bb20e1-36fe-4704-871e-d63c2cd0bfe7">26</span>. (<span
                                    class="RefDOI"
                                    id="d91f9d74-bced-4fb6-9cd6-709bdd7ca77f">doi:10.1186/s13293-015-0044-8</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R32" id="R32"
                                data-reftype="article-journal" data-doi="10.1016/j.gene.2018.12.057"
                                data-pmid="30615915" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R32"><span class="RefSlNo"
                                    id="e3cd9d2e-15fd-4cf1-af5f-d79a9d62145f">32.</span> <span class="RefAuthor"
                                    id="2df74fab-9893-4899-8398-9550f2f4f0ee"><span class="RefSurName">Fellous</span>
                                    <span class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="a16b00d8-c01b-49e5-ae9d-3ffdc1b23a59"><span class="RefSurName">Earley</span>
                                    <span class="RefGivenName">RL</span></span>, <span class="RefAuthor"
                                    id="3e1e006a-5035-4617-9e31-80814b027a26"><span class="RefSurName">Silvestre</span>
                                    <span class="RefGivenName">F</span></span>. <span class="RefYear"
                                    id="f511bda0-dc11-416f-a483-bafe8f00c64b">2019</span> <span class="RefArticleTitle"
                                    id="5d015a3f-c0a2-48aa-a237-27ef65c19fe6">Identification and expression of mangrove
                                    rivulus (Kryptolebias marmoratus) histone deacetylase (HDAC) and lysine
                                    acetyltransferase (KAT) genes</span>. <span class="RefJournalTitle"
                                    id="38af593d-976c-46b1-9797-1959194609f2"><i>Gene</i></span> <span class="RefVolume"
                                    id="1f6f6950-da43-4724-80d4-936cfb80d960"><b>691</b></span>, <span class="RefFPage"
                                    id="a7f38484-a6ef-4979-abe7-f077de64252f">56</span>–<span class="RefLPage"
                                    id="6c955bdb-c7bd-4cdb-be67-3cb1e3079e9b">69</span>. (<span class="RefDOI"
                                    id="51c75a40-cfbe-440d-b2f9-e3208d21dad3">doi:10.1016/j.gene.2018.12.057</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R33" id="R33"
                                data-reftype="article-journal" data-doi="10.1111/j.1439-0310.2005.01147.x"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R33" data-uncited="true"><span class="RefSlNo"
                                    id="ee042ff5-c015-4a39-b566-b26bbe1e6a74">33.</span> <span class="RefAuthor"
                                    id="f1d87dae-bccb-4e52-a5ce-097d17c59e34"><span class="RefSurName">U</span> <span
                                        class="RefGivenName">BlanckenhornW</span></span>. <span class="RefYear"
                                    id="58d43ddd-d265-41af-bf17-f85755de01fc">2005</span> <span class="RefArticleTitle"
                                    id="a30f4c09-4f1c-444e-8992-1b4011896b80">Behavioral Causes and Consequences of
                                    Sexual Size Dimorphism</span>. <span class="RefJournalTitle"
                                    id="8fb5b6d7-f190-42cf-8df8-25443dc67aed"><i>Ethology</i></span> <span
                                    class="RefVolume" id="e6b4bb92-2dda-4048-9b59-3b294be5f19a"><b>111</b></span>, <span
                                    class="RefFPage" id="610cece9-9783-4df9-9250-52d91ae2a939">977</span>–<span
                                    class="RefLPage" id="d2f30b66-5886-4352-b76e-fb79d85d2b3c">1016</span>. (<span
                                    class="RefDOI"
                                    id="04bb7dc8-af95-469e-9731-ca6f2a9acb5f">doi:10.1111/j.1439-0310.2005.01147.x</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R34" id="R34"
                                data-reftype="article-journal" data-doi="10.1038/nrg.2017.60" data-pmid="28804140"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R34" data-uncited="true"><span class="RefSlNo"
                                    id="b7dd193e-3aad-4347-80f6-d340777447a5">34.</span> <span class="RefAuthor"
                                    id="c85ab207-4a5f-49f8-a84e-51e5f4c753f6"><span class="RefSurName">Capel</span>
                                    <span class="RefGivenName">B</span></span>. <span class="RefYear"
                                    id="fddd5e93-e6c4-4233-b900-7f8b83e1671e">2017</span> <span class="RefArticleTitle"
                                    id="a1c69505-052d-4e5a-af7c-3774660a9461">Vertebrate sex determination: evolutionary
                                    plasticity of a fundamental switch</span>. <span class="RefJournalTitle"
                                    id="4d92a27c-589b-427c-b762-bbba94084cfe"><i>Nat Rev Genet</i></span> <span
                                    class="RefVolume" id="c2ab7723-24cf-40e9-9fae-4071b7f843fc"><b>18</b></span>, <span
                                    class="RefFPage" id="d2e06f04-70b6-4b33-b1f2-f7050e28ac15">675</span>–<span
                                    class="RefLPage" id="41ec33a0-8fd3-48de-bf37-7565e7fe1e5e">689</span>. (<span
                                    class="RefDOI"
                                    id="431f0772-7934-4dd6-a546-04f711c30950">doi:10.1038/nrg.2017.60</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R35" id="R35"
                                data-reftype="article-journal" data-doi="10.1023/A:1007389519702"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R35" data-uncited="true"><span class="RefSlNo"
                                    id="d00c04df-5053-481b-8ee1-b02ef8fbf6b9">35.</span> KuwamuraT., NakashimaY. <span
                                    class="RefYear" id="bff3a828-676a-47bd-b9bb-c0a17a515c54">1998</span> <span
                                    class="RefArticleTitle" id="ff31b712-560a-4061-8de8-c1db88b7cf34">New Aspects of Sex
                                    Change among Reef Fishes: Recent Studies in Japan</span>. <span
                                    class="RefJournalTitle" id="e5286941-f7bc-4839-a265-4f8545e73dc3"><i>Environ. Biol.
                                        Fishes</i></span> <span class="RefVolume"
                                    id="c5384fb8-49c6-4c65-8bce-9f1a521118dc"><b>52</b></span>, <span class="RefFPage"
                                    id="4aa7f4eb-2d68-4af8-b85a-54dc46bf223d">125</span>–<span class="RefLPage"
                                    id="71ee17f0-710b-4417-85f8-e0902e74c0c8">135</span>. (<span class="RefDOI"
                                    id="63d0796d-3139-4e94-aa91-84d5b266baba">doi:10.1023/A:1007389519702</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R36" id="R36"
                                data-reftype="article-journal" data-doi="10.1111/j.1095-8649.2007.01338.x"
                                data-reftype-name="Journal" data-csl="true" data-validated-log="true"
                                data-validated="true" new-id="R36" data-uncited="true"><span class="RefSlNo"
                                    id="1becc788-2b14-4d40-8a44-4ab4ea620dc8">36.</span> ManabeH., IshimuraM.,
                                ShinomiyaA., SunobeT. <span class="RefYear"
                                    id="ca118f71-1fec-40f0-bded-351b1e5cc833">2007</span> <span class="RefArticleTitle"
                                    id="d0fada7f-3411-45a7-a744-0cef987cdc50">Field Evidence for Bi‐directional Sex
                                    Change in the Polygynous Gobiid FishTrimma Okinawae</span>. <span
                                    class="RefJournalTitle" id="7d0eb448-b76d-4fc4-919a-db60c1ca1f40"><i>J. Fish
                                        Biol</i>.</span> <span class="RefVolume"
                                    id="7c1a2a00-a578-42d6-943e-20b72c697abd"><b>70</b></span>, <span class="RefFPage"
                                    id="e0418fcc-b543-461d-9b90-70b6c36736b9">600</span>–<span class="RefLPage"
                                    id="eecb42cb-d98c-4d79-ac45-acc6752d07ac">609</span>. (<span class="RefDOI"
                                    id="7dd785b8-5e7f-46ac-815b-69349fab1aaf">doi:10.1111/j.1095-8649.2007.01338.x</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R37" id="R37"
                                data-reftype="article-journal" data-doi="10.1016/j.cbpb.2005.03.012"
                                data-pmid="15916910" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R37" data-uncited="true"><span class="RefSlNo"
                                    id="24854b67-7c76-4fe2-9c73-6b70bc7221cf">37.</span> <span class="RefAuthor"
                                    id="29bcc5d5-2615-4685-b9a0-10469ab0a3c4"><span class="RefSurName">Sunobe</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="333959cc-cea6-4ee1-b83a-daada0ca5422"><span class="RefSurName">Nakamura</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="0d0a8af4-bed9-47c0-8195-c69541c5bed8"><span class="RefSurName">Kobayashi</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="5463dfd1-e432-4937-9fc7-c91d6c185129"><span class="RefSurName">Kobayashi</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="944d10c2-60e3-4d71-87a9-a9291c78f360"><span class="RefSurName">Nagahama</span>
                                    <span class="RefGivenName">Y</span></span>. <span class="RefYear"
                                    id="873adef3-02e5-46a5-9b15-b6fcee63f388">2005</span> <span class="RefArticleTitle"
                                    id="f1380d80-cadc-48ad-872f-59cd26feb593">Aromatase immunoreactivity and the role of
                                    enzymes in steroid pathways for inducing sex change in the hermaphrodite gobiid fish
                                    Trimma okinawae</span>. <span class="RefJournalTitle"
                                    id="7b64a3ee-fcd6-475a-9888-aca52edb3a96"><i>Comp Biochem Physiol A Mol Integr
                                        Physiol</i></span> <span class="RefVolume"
                                    id="cb34a1f9-558d-4fa0-be50-8be851b149da"><b>141</b></span>, <span class="RefFPage"
                                    id="6fa7848f-0cb3-4a5f-9b3e-8bda6fcb7623">54</span>–<span class="RefLPage"
                                    id="a78e04fb-aebc-4076-b49d-79580d291d25">59</span>. (<span class="RefDOI"
                                    id="b691721b-2266-49aa-8f48-3354fa1155bd">doi:10.1016/j.cbpb.2005.03.012</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R38" id="R38"
                                data-reftype="article-journal" data-doi="10.2108/zsj.29.121" data-pmid="22303852"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R38" data-uncited="true"><span class="RefSlNo"
                                    id="37a6bf53-c188-4a96-86fc-ed81bd07f667">38.</span> <span class="RefAuthor"
                                    id="a6aea092-fe0c-43cb-956f-fd3ff2459e7d"><span class="RefSurName">Kobayashi</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="c4f28c5d-7b11-4f19-9afa-f4bd5eade1f8"><span class="RefSurName">Usami</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="bee6cbf6-39e3-4d1b-a7f5-40abbd9298a5"><span class="RefSurName">Sunobe</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="479f9b9b-6919-4d83-8d4c-8bbcedce55a3"><span class="RefSurName">Manabe</span>
                                    <span class="RefGivenName">H</span></span>, <span class="RefAuthor"
                                    id="a5928fd0-aa00-4d7e-bfef-923922951cf1"><span class="RefSurName">Nagahama</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="31798cf9-c61d-4ec2-b0e4-ede35c07f029"><span class="RefSurName">Nakamura</span>
                                    <span class="RefGivenName">M</span></span>. <span class="RefYear"
                                    id="98fd6cf9-820f-493e-991f-d4fdf890fbf0">2012</span> <span class="RefArticleTitle"
                                    id="6d0131a0-c9bb-4838-8859-c554a733add1">Histological observation of the urogenital
                                    papillae in the bi-directional sex-changing gobiid fish, Trimma okinawae</span>.
                                <span class="RefJournalTitle" id="3f45fd3a-9a83-4073-91a7-104d46a853bc"><i>Zoolog
                                        Sci</i></span> <span class="RefVolume"
                                    id="cb5e7168-edff-4494-a8f4-a038ecae6bd2"><b>29</b></span>, <span class="RefFPage"
                                    id="f9cef901-7ddf-4541-80ef-daca23c4ca7f">121</span>–<span class="RefLPage"
                                    id="56e5da9d-0e29-4c52-99ec-3cde1d85dc19">126</span>. (<span class="RefDOI"
                                    id="7889ab1e-52ad-40cf-b1d3-0154cdd4d715">doi:10.2108/zsj.29.121</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R39" id="R39"
                                data-reftype="article-journal" data-doi="10.1093/bioinformatics/btv351"
                                data-pmid="26059717" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R39" data-uncited="true"><span class="RefSlNo"
                                    id="3eb5a031-78e6-4008-8230-2564c2729571">39.</span> <span class="RefAuthor"
                                    id="cb3dedab-4bb0-4623-86e0-e2115da289d0"><span class="RefSurName">Simão</span>
                                    <span class="RefGivenName">FA</span></span>, <span class="RefAuthor"
                                    id="d72bf218-3e9c-4575-9ed9-98921282adf1"><span class="RefSurName">Waterhouse</span>
                                    <span class="RefGivenName">RM</span></span>, <span class="RefAuthor"
                                    id="6d1c3326-9be2-4b86-adcf-d65b3d06ae14"><span class="RefSurName">Ioannidis</span>
                                    <span class="RefGivenName">P</span></span>, <span class="RefAuthor"
                                    id="efacb2aa-3291-433b-ad8f-b4b7a4581ec1"><span
                                        class="RefSurName">Kriventseva</span> <span
                                        class="RefGivenName">EV</span></span>, <span class="RefAuthor"
                                    id="3dc0865b-4674-4380-9bc4-eb463916fbf4"><span class="RefSurName">Zdobnov</span>
                                    <span class="RefGivenName">EM</span></span>. <span class="RefYear"
                                    id="59fc7d5d-9706-4f8b-a45b-6557dd7704b4">2015</span> <span class="RefArticleTitle"
                                    id="5879fbcc-87b3-4d9d-a7f7-73576781eb0a">BUSCO: assessing genome assembly and
                                    annotation completeness with single-copy orthologs</span>. <span
                                    class="RefJournalTitle"
                                    id="dbc8af30-036f-4bee-a304-5d2f4d73eb78"><i>Bioinformatics</i></span> <span
                                    class="RefVolume" id="01efa7f6-678e-451b-89db-97f13cef1f1e"><b>31</b></span>, <span
                                    class="RefFPage" id="1cf591c9-0d4b-41c4-8842-5724cb26a13f">3210</span>–<span
                                    class="RefLPage" id="1769d85e-1805-47c4-9f5b-99873813f646">3212</span>. (<span
                                    class="RefDOI"
                                    id="352d1905-3bc6-47bd-a1b2-9a885afc504a">doi:10.1093/bioinformatics/btv351</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R40" id="R40"
                                data-reftype="article-journal" data-doi="10.1093/molbev/msx293" data-pmid="29136184"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R40" data-uncited="true"><span class="RefSlNo"
                                    id="69e852f7-a225-418a-8bad-190849b313e1">40.</span> <span class="RefAuthor"
                                    id="b9870d32-bbf3-4671-832b-6461895fb010"><span class="RefSurName">Todd</span> <span
                                        class="RefGivenName">EV</span></span>, <span class="RefAuthor"
                                    id="c680375d-0290-4f62-b488-2873952f5546"><span class="RefSurName">Liu</span> <span
                                        class="RefGivenName">H</span></span>, <span class="RefAuthor"
                                    id="4aaf38cd-553b-4586-ae58-6c44d6a18ace"><span class="RefSurName">Lamm</span> <span
                                        class="RefGivenName">MS</span></span>, <span class="RefAuthor"
                                    id="e2515dad-9fc6-49df-bbff-49f008f4da85"><span class="RefSurName">Thomas</span>
                                    <span class="RefGivenName">JT</span></span>, <span class="RefAuthor"
                                    id="9388d0d1-9551-4587-9aea-b2724c7f128d"><span class="RefSurName">Rutherford</span>
                                    <span class="RefGivenName">K</span></span>, <span class="RefAuthor"
                                    id="932f38ba-dba4-488e-94fd-8a49dac7bff0"><span class="RefSurName">Thompson</span>
                                    <span class="RefGivenName">KC</span></span>, <span class="RefAuthor"
                                    id="b6bf0da6-2aaa-409f-af12-ca6516c0ee6d"><span class="RefSurName">Godwin</span>
                                    <span class="RefGivenName">JR</span></span>, <span class="RefAuthor"
                                    id="4b63389c-8610-48d2-9790-31d82d0442e1"><span class="RefSurName">Gemmell</span>
                                    <span class="RefGivenName">NJ</span></span>. <span class="RefYear"
                                    id="f122620b-fb2c-4d82-b48c-80574c6f862c">2018</span> <span class="RefArticleTitle"
                                    id="b9f5a5a3-0c84-4900-b38b-058469e12a94">Female Mimicry by Sneaker Males Has a
                                    Transcriptomic Signature in Both the Brain and the Gonad in a Sex-Changing
                                    Fish</span>. <span class="RefJournalTitle"
                                    id="5fd52d96-361a-494d-8f8d-21244aa2778b"><i>Mol Biol Evol</i></span> <span
                                    class="RefVolume" id="9f847f22-0f43-416e-8daa-aca6cb78b171"><b>35</b></span>, <span
                                    class="RefFPage" id="9f965b89-547b-4ea6-87e7-d4f586a53587">225</span>–<span
                                    class="RefLPage" id="b5fd7006-a123-4d97-8b44-91fa65bc30e8">241</span>. (<span
                                    class="RefDOI"
                                    id="8bb6b7bc-b9b1-4341-8c16-506c4fe399bf">doi:10.1093/molbev/msx293</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R41" id="R41"
                                data-reftype="article-journal" data-doi="10.1186/1471-2164-15-905" data-pmid="25326170"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R41" data-uncited="true"><span class="RefSlNo"
                                    id="847dfe3b-a1ea-4c74-8030-3f9f729fffa6">41.</span> <span class="RefAuthor"
                                    id="c037351d-e444-47e5-80e6-34b9c22fc429"><span class="RefSurName">Wong</span> <span
                                        class="RefGivenName">RY</span></span>, <span class="RefAuthor"
                                    id="7d6e8d9c-4d4a-4731-b443-7bde96c53a44"><span class="RefSurName">McLeod</span>
                                    <span class="RefGivenName">MM</span></span>, <span class="RefAuthor"
                                    id="1f8fa750-279c-42ba-b5c5-665edeb3c8a2"><span class="RefSurName">Godwin</span>
                                    <span class="RefGivenName">J</span></span>. <span class="RefYear"
                                    id="9cb44eeb-f998-4332-b86d-d1a3d77f7b6b">2014</span> <span class="RefArticleTitle"
                                    id="cd37c699-3f11-45c5-87f7-21505846a5f2">Limited sex-biased neural gene expression
                                    patterns across strains in Zebrafish (Danio rerio)</span>. <span
                                    class="RefJournalTitle" id="c3294e99-2ec9-4d56-8c8a-b3abce391968"><i>BMC
                                        Genomics</i></span> <span class="RefVolume"
                                    id="ed8a29d2-73c0-411d-892e-bc31a148004c"><b>15</b></span>, <span class="RefFPage"
                                    id="129e5e5d-9d0d-4779-b343-6ada1a24b5c3">905</span>. (<span class="RefDOI"
                                    id="e0613b6c-b26a-47f4-a361-c58bd77797a0">doi:10.1186/1471-2164-15-905</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R42" id="R42"
                                data-reftype="article-journal" data-doi="10.1186/1471-2164-15-655" data-pmid="25099474"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R42" data-uncited="true"><span class="RefSlNo"
                                    id="296b1a5d-0c14-4ad3-adc9-af177c272475">42.</span> <span class="RefAuthor"
                                    id="407fcffe-b6db-4efb-a3a5-ba56c593af90"><span class="RefSurName">Manousaki</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="6eb4d553-4b1e-407e-b327-a85c8f68bf46"><span
                                        class="RefSurName">Tsakogiannis</span> <span
                                        class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="499ca847-e7a8-4eb0-a11c-ea16a27fce74"><span class="RefSurName">Lagnel</span>
                                    <span class="RefGivenName">J</span></span>, <span class="RefAuthor"
                                    id="a490cc83-941e-4150-9b94-524421dcda3b"><span
                                        class="RefSurName">Sarropoulou</span> <span
                                        class="RefGivenName">E</span></span>, <span class="RefAuthor"
                                    id="b6b8d82a-ec88-4e35-a1a0-b309a96048c4"><span class="RefSurName">Xiang</span>
                                    <span class="RefGivenName">JZ</span></span>, <span class="RefAuthor"
                                    id="6603d007-8339-46d5-9989-64fa9687fd91"><span
                                        class="RefSurName">Papandroulakis</span> <span
                                        class="RefGivenName">N</span></span>, <span class="RefAuthor"
                                    id="b97e3247-e46f-4f43-a78f-546b565aedae"><span class="RefSurName">Mylonas</span>
                                    <span class="RefGivenName">CC</span></span>, <span class="RefAuthor"
                                    id="5deb9242-fcc9-4cd9-a16b-bea8e44df998"><span
                                        class="RefSurName">Tsigenopoulos</span> <span
                                        class="RefGivenName">CS</span></span>. <span class="RefYear"
                                    id="f0298edf-ad01-4dc0-9cc5-94d5a2813a88">2014</span> <span class="RefArticleTitle"
                                    id="89f0d91e-5942-44fd-aebe-5c0e1e334647">The sex-specific transcriptome of the
                                    hermaphrodite sparid sharpsnout seabream (Diplodus puntazzo)</span>. <span
                                    class="RefJournalTitle" id="030dd0e4-c3c1-4ff7-be80-1a610c67d105"><i>BMC
                                        Genomics</i></span> <span class="RefVolume"
                                    id="5cd140fe-94bc-4c9d-9db9-5e181e60fb8c"><b>15</b></span>, <span class="RefFPage"
                                    id="1d5f0ae2-2efa-4470-b4bb-af1febdfe8ea">655</span>. (<span class="RefDOI"
                                    id="f8e29a2b-d96c-464f-8a84-2aa0fd3a63a7">doi:10.1186/1471-2164-15-655</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R43" id="R43"
                                data-reftype="article-journal" data-doi="10.1186/s40246-022-00439-3"
                                data-pmid="36461115" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R43" data-uncited="true"><span class="RefSlNo"
                                    id="2a8a1802-d002-4bda-9d5a-948753086435">43.</span> <span class="RefAuthor"
                                    id="4350d71b-64f4-416b-8930-662e209453cc"><span class="RefSurName">Braschi</span>
                                    <span class="RefGivenName">B</span></span>, <span class="RefAuthor"
                                    id="5e8b11cd-34de-4460-91cd-e305684d3a02"><span class="RefSurName">Bruford</span>
                                    <span class="RefGivenName">EA</span></span>, <span class="RefAuthor"
                                    id="e6e25d79-c30b-4f6a-9720-d03f5c89ec7c"><span class="RefSurName">Cavanagh</span>
                                    <span class="RefGivenName">AT</span></span>, <span class="RefAuthor"
                                    id="3d170b92-739a-4038-aed9-c8cea7529b96"><span class="RefSurName">Neuman</span>
                                    <span class="RefGivenName">SD</span></span>, <span class="RefAuthor"
                                    id="246979f7-311c-4f6c-921e-24d6c93bfd66"><span
                                        class="RefSurName">Bashirullah</span> <span
                                        class="RefGivenName">A</span></span>. <span class="RefYear"
                                    id="b1d1ca18-12a4-48fb-a85c-ceabd2a9f240">2022</span> <span class="RefArticleTitle"
                                    id="5771d291-4764-47ce-b236-301124bf4ad4">The bridge-like lipid transfer protein
                                    (BLTP) gene group: introducing new nomenclature based on structural homology
                                    indicating shared function</span>. <span class="RefJournalTitle"
                                    id="afceb196-04e4-4b7f-aa02-92138f21a3b3"><i>Hum Genomics</i></span> <span
                                    class="RefVolume" id="01fafb13-adeb-45f4-863d-7d967029cea6"><b>16</b></span>, <span
                                    class="RefFPage" id="ff09db7c-0022-48db-817c-4ed331376a03">66</span>. (<span
                                    class="RefDOI"
                                    id="ff976940-f904-4289-8733-1e071ebda372">doi:10.1186/s40246-022-00439-3</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R44" id="R44"
                                data-reftype="article-journal" data-doi="10.1210/en.2017-00808" data-pmid="29053844"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R44" data-uncited="true"><span class="RefSlNo"
                                    id="f7f1d2e1-81c5-4721-ad3b-df16e5774a4e">44.</span> <span class="RefAuthor"
                                    id="c75c9997-cbaa-4a2b-a8a0-fd74b3609ccc"><span class="RefSurName">Nakajo</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="cc038a72-b961-4d98-8df7-aa1b80f38800"><span class="RefSurName">Kanda</span>
                                    <span class="RefGivenName">S</span></span>, <span class="RefAuthor"
                                    id="3b2b138f-4f08-4aad-9266-cda6cbcbb8fe"><span class="RefSurName">Karigo</span>
                                    <span class="RefGivenName">T</span></span>, <span class="RefAuthor"
                                    id="a3d464e0-9e8a-4109-bf5d-1c2085d5fd5f"><span class="RefSurName">Takahashi</span>
                                    <span class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="88a28d96-51df-4a02-a26c-214275f11a01"><span class="RefSurName">Akazome</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="20e80789-949e-4922-b329-ee7325b014a8"><span class="RefSurName">Uenoyama</span>
                                    <span class="RefGivenName">Y</span></span>, <span class="RefAuthor"
                                    id="c9a0a9ac-f7e7-4ff6-b2aa-1c3c4704b4ed"><span class="RefSurName">Kobayashi</span>
                                    <span class="RefGivenName">M</span></span>, <span class="RefAuthor"
                                    id="f75dfafe-f49c-425a-bf54-c7a62f6ddeb9"><span class="RefSurName">Oka</span> <span
                                        class="RefGivenName">Y</span></span>. <span class="RefYear"
                                    id="54aa3b84-c54d-4140-be56-1431ce27ae1b">2018</span> <span class="RefArticleTitle"
                                    id="18e294c0-4ad7-4c22-9707-d2c4efe9a951">Evolutionally Conserved Function of
                                    Kisspeptin Neuronal System Is Nonreproductive Regulation as Revealed by Nonmammalian
                                    Study</span>. <span class="RefJournalTitle"
                                    id="1f469fed-610e-4059-b4f9-d5e69c2ee2f2"><i>Endocrinology</i></span> <span
                                    class="RefVolume" id="69683656-6b0d-4ab5-a862-186bf8ef254e"><b>159</b></span>, <span
                                    class="RefFPage" id="cae97c83-9f61-4a71-a92e-5e31eebf9e06">163</span>–<span
                                    class="RefLPage" id="b6b8ebd1-0121-4357-8611-9790eb069a6e">183</span>. (<span
                                    class="RefDOI"
                                    id="1a6d34f3-2525-470b-85b6-e1b625522593">doi:10.1210/en.2017-00808</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R45" id="R45"
                                data-reftype="article-journal" data-doi="10.1016/j.ygcen.2020.113568"
                                data-pmid="32710898" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R45" data-uncited="true"><span class="RefSlNo"
                                    id="45d80225-ee92-4559-94bb-e3ac08e6a63b">45.</span> <span class="RefAuthor"
                                    id="2952bf22-03ce-48c3-93b1-371f06515590"><span class="RefSurName">Somoza</span>
                                    <span class="RefGivenName">GM</span></span>, <span class="RefAuthor"
                                    id="0bd0a1be-e74c-4ccf-a7c9-6e3d5b77f732"><span class="RefSurName">Mechaly</span>
                                    <span class="RefGivenName">AS</span></span>, <span class="RefAuthor"
                                    id="d952f623-45ce-42ad-b949-e3ef87f5d250"><span class="RefSurName">Trudeau</span>
                                    <span class="RefGivenName">VL</span></span>. <span class="RefYear"
                                    id="4e6f6fc6-07f3-4d18-a904-2065820dfc69">2020</span> <span class="RefArticleTitle"
                                    id="6f5fa135-b33d-4d1b-a743-24e2c22f7129">Kisspeptin and GnRH interactions in the
                                    reproductive brain of teleosts</span>. <span class="RefJournalTitle"
                                    id="7a2e7289-db02-4cda-912e-0afa73e9dd03"><i>Gen Comp Endocrinol</i></span> <span
                                    class="RefVolume" id="6e1fbd76-5711-4427-b3c9-22be3eaa87ea"><b>298</b></span>, <span
                                    class="RefFPage" id="a2daf6d9-7ca6-426e-93e5-3b8f0050536f">113568</span>. (<span
                                    class="RefDOI"
                                    id="3a7387df-50d5-420c-894c-8dc1f7a1427d">doi:10.1016/j.ygcen.2020.113568</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R46" id="R46"
                                data-reftype="article-journal" data-doi="10.1016/j.isci.2020.101183"
                                data-pmid="32498016" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R46" data-uncited="true"><span class="RefSlNo"
                                    id="7c82a6ae-80c1-4be9-9bc3-6735aa83395b">46.</span> <span class="RefAuthor"
                                    id="2bff86de-d3e6-4b1d-bc2a-b7b8a89bbf2c"><span class="RefSurName">Hori</span> <span
                                        class="RefGivenName">K</span></span> <span class="RefEtal"
                                    id="7720558f-67e1-45fc-9e66-8975c045f20a"><i>et al</i></span>. <span class="RefYear"
                                    id="b4aaf3e2-7ac7-44a4-8548-287ae4be089b">2020</span> <span class="RefArticleTitle"
                                    id="9d669030-a1b2-422f-a289-d3507423e4a3">AUTS2 Regulation of Synapses for Proper
                                    Synaptic Inputs and Social Communication</span>. <span class="RefJournalTitle"
                                    id="2d3f2c6a-d7f5-48f3-b482-22c8b6f85aec"><i>iScience</i></span> <span
                                    class="RefVolume" id="884ea2ad-a4d3-4c0b-a585-d787ca61d9b9"><b>23</b></span>, <span
                                    class="RefFPage" id="8613a02c-daa8-45e7-b821-3eec3bdd7cae">101183</span>. (<span
                                    class="RefDOI"
                                    id="2e0f4d4d-f665-4663-9178-c0dc8d64078f">doi:10.1016/j.isci.2020.101183</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R47" id="R47"
                                data-reftype="article-journal" data-doi="10.1126/science.aac6633" data-pmid="26722000"
                                data-reftype-name="Journal" data-csl="true" data-validated-online="true"
                                title="Validated from PubMed" data-validated-log="true" data-validated="true"
                                new-id="R47" data-uncited="true"><span class="RefSlNo"
                                    id="64da6c12-7786-4aea-8810-7daf7967098d">47.</span> <span class="RefAuthor"
                                    id="7bc84fb4-254a-43dd-92c3-471956eaf537"><span class="RefSurName">Simola</span>
                                    <span class="RefGivenName">DF</span></span> <span class="RefEtal"
                                    id="38a68061-5d50-42e3-933f-187f035753c4"><i>et al</i></span>. <span class="RefYear"
                                    id="74a13d47-5c7d-4dc7-99e4-9b4bdd23a059">2016</span> <span class="RefArticleTitle"
                                    id="74cf7de6-4805-4710-a9b3-86d2a049005d">Epigenetic (re)programming of
                                    caste-specific behavior in the ant Camponotus floridanus</span>. <span
                                    class="RefJournalTitle"
                                    id="322e274a-bcb5-4b48-8903-40e0f8297b66"><i>Science</i></span> <span
                                    class="RefVolume" id="83bfc6cd-47f6-49a1-809a-2eb38a24511b"><b>351</b></span>, <span
                                    class="RefELocation" id="cb1e81f9-07fd-4c1e-add4-aac50457380d">aac6633</span>.
                                (<span class="RefDOI"
                                    id="dd404d00-83e7-41b9-91fa-867050abd9e5">doi:10.1126/science.aac6633</span>)</p>
                            <p class="jrnlRefText" data-editable="false" data-id="R48" id="R48"
                                data-reftype="article-journal" data-doi="10.1146/annurev-animal-021419-083634"
                                data-pmid="31525067" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R48" data-uncited="true"><span class="RefSlNo"
                                    id="284ce37c-d80c-4cda-9a9b-9226d88a420d">48.</span> <span class="RefAuthor"
                                    id="217d8a74-3060-4f8e-a7b0-c7800331697b"><span
                                        class="RefSurName">Ortega-Recalde</span> <span
                                        class="RefGivenName">O</span></span>, <span class="RefAuthor"
                                    id="388a3798-5903-4ed8-ab5d-62fbf7eb8fad"><span class="RefSurName">Goikoetxea</span>
                                    <span class="RefGivenName">A</span></span>, <span class="RefAuthor"
                                    id="35d00a03-89b5-480d-98e9-7350b1c41573"><span class="RefSurName">Hore</span> <span
                                        class="RefGivenName">TA</span></span>, <span class="RefAuthor"
                                    id="1df11412-04fa-4fbd-96c0-dbfd509cc725"><span class="RefSurName">Todd</span> <span
                                        class="RefGivenName">EV</span></span>, <span class="RefAuthor"
                                    id="1831ea47-ec53-4c29-9996-c6a5a4326787"><span class="RefSurName">Gemmell</span>
                                    <span class="RefGivenName">NJ</span></span>. <span class="RefYear"
                                    id="d726bb39-d2d8-477d-887c-18d8dfa9ac0f">2020</span> <span class="RefArticleTitle"
                                    id="19be5346-b533-4c6e-b899-805abdc7dedf">The Genetics and Epigenetics of Sex Change
                                    in Fish</span>. <span class="RefJournalTitle"
                                    id="45859c64-ae18-437b-97eb-4db89ed44fec"><i>Annu Rev Anim Biosci</i></span> <span
                                    class="RefVolume" id="abfa53d1-905d-4fc7-bcff-03cc4f45c401"><b>8</b></span>, <span
                                    class="RefFPage" id="a3bd31d9-3791-4afc-8b8a-d29527d438ca">47</span>–<span
                                    class="RefLPage" id="9fddff27-6238-42ce-be1f-6b222756f3b2">69</span>. (<span
                                    class="RefDOI"
                                    id="c79907d9-3369-4fd8-baa9-be4f264e1cfb">doi:10.1146/annurev-animal-021419-083634</span>)
                            </p>
                            <p class="jrnlRefText" data-editable="false" data-id="R49" id="R49"
                                data-reftype="article-journal" data-doi="10.1016/j.ajhg.2012.12.011"
                                data-pmid="23332918" data-reftype-name="Journal" data-csl="true"
                                data-validated-online="true" title="Validated from PubMed" data-validated-log="true"
                                data-validated="true" new-id="R49" data-uncited="true"><span class="RefSlNo"
                                    id="063a61b9-eba4-4753-a9a9-fd0030371c14">49.</span> <span class="RefAuthor"
                                    id="4fb3c818-f4c6-4c9f-9a68-949a51bec83f"><span class="RefSurName">Beunders</span>
                                    <span class="RefGivenName">G</span></span> <span class="RefEtal"
                                    id="d9bb9f96-81fc-4170-8067-a9abd7f52a7a"><i>et al</i></span>. <span class="RefYear"
                                    id="1d9bf704-10b2-482d-8a42-4b5d36dafa1a">2013</span> <span class="RefArticleTitle"
                                    id="dba766c8-3940-4764-b928-8986255dbf40">Exonic deletions in AUTS2 cause a
                                    syndromic form of intellectual disability and suggest a critical role for the C
                                    terminus</span>. <span class="RefJournalTitle"
                                    id="b5002f71-89e5-4989-a373-d617d6265c6d"><i>Am J Hum Genet</i></span> <span
                                    class="RefVolume" id="5acb146b-e3e2-4924-9072-a6499fb35440"><b>92</b></span>, <span
                                    class="RefFPage" id="2203724e-e501-44ad-b2bb-1d2953995ec7">210</span>–<span
                                    class="RefLPage" id="3117e734-be7f-4844-8f1c-577d64779275">220</span>. (<span
                                    class="RefDOI"
                                    id="c8d7a497-bcf7-4083-b19a-c9977b6e463f">doi:10.1016/j.ajhg.2012.12.011</span>)</p>
                        </div>
                        <div class="jrnlFN" data-fn-type="acknowledgements" data-label="Acknowledgement"
                            id="idm140027623033488"><span class="label">Acknowledgments</span>
                            <div class="jrnlFNText">
                                <p class="jrnlFNPara">We thank Takuro Nakayama and Sinichiro Maruyama for advising data
                                    analysis, and Gen Kume and Midori Matsuoka for scuba sampling. Computations were
                                    partially performed on the NIG supercomputer at ROIS National Institute of Genetics.
                                </p>
                            </div>
                        </div>
                    </div>
                </content>
            </response>
        </content>
    </response>
</body>

</html>