<div class="WordSection1 wysihtml-sandbox wysihtml-editor" id="contentDivNode" data-event="click" data-message="{'click':{'funcToCall': 'contentEvt','channel':'components','topic':'general'}}" tabindex="0" contenteditable="true"><div class="front" data-edited="true" data-edited-version="2"><h1 class="jrnlArtTitle" headweightage="47">Transcriptomic signatures associated with underlying rapid changes in the early phase brain of bi-directional sex change in <i>Trimma okinawae</i></h1><p class="jrnlAuthors" data-authors="true" data-rid="ag1"><span class="jrnlAuthorGroup" data-contrib-type="author"><span class="jrnlAuthor"><span class="jrnlGivenName">Katsunori</span> <span class="jrnlSurName">Tamagawa</span></span>, </span><span class="jrnlAuthorGroup" data-contrib-type="author"><span class="jrnlAuthor"><span class="jrnlGivenName">Tomoki</span> <span class="jrnlSurName">Sunobe</span></span>, <sup class="jrnlAffRef" rid="aff2" data-rid="aff2">2</sup></span><span class="jrnlAuthorGroup" data-contrib-type="author"><span class="jrnlAuthor"><span class="jrnlGivenName">Takashi</span> <span class="jrnlSurName">Makino</span></span>, <sup class="jrnlAffRef" rid="aff1" data-rid="aff1">1</sup></span><span class="jrnlAuthorGroup" data-contrib-type="author"><span class="jrnlAuthor"><span class="jrnlGivenName">Masakado</span> <span class="jrnlSurName">Kawata</span></span></span> <span class="jrnlAuthorGroup" data-contrib-type="author"><span class="jrnlAuthor"><span class="jrnlGivenName">Katsunori</span> <span class="jrnlSurName">Tamagawa</span></span></span></p><p class="jrnlAff" id="aff1" data-id="aff1"><sup>1</sup><span class="jrnlDepartment">Graduate School of Life Sciences</span>, <span class="jrnlInstitution">Tohoku University, Aoba-ku</span>, <span class="jrnlCity">Sendai 980-8578</span>, <span class="jrnlCountry">Japan</span></p><p class="jrnlAff" id="aff2" data-id="aff2"><sup>2</sup><span class="jrnlDepartment">Laboratory of Fish Behavioral Ecology</span>, <span class="jrnlInstitution">Tateyama Station, Field Science Center, Tokyo University of Marine Science and Technology, 670 Banda, Tateyama</span>, <span class="jrnlCity">Chiba, 294-0308</span>, <span class="jrnlCountry">Japan</span></p><p class="jrnlAff" id="aff3" data-id="aff3" data-aff-reordered="true"><sup><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876731000" data-cid="1702876731000">*</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876731000" data-cid="1702876731001">3</span></sup><span class="jrnlInstitution">Corresponding Authors</span></p><p class="jrnlCorrAff" id="cor1" data-id="cor1">Address: 6-3, Aoba, Aramaki, Aoba, Sendai, Miyagi 980-8578, Japan E-mail: tamagawa.katsunori@gmail.com https://orcid.org/0000-0002-8796-6486 Masakado Kawata Address: 6-3, Aoba, Aramaki, Aoba, Sendai, Miyagi 980-8578, Japan E-mail: kawata@tohoku.ac.jp</p><div class="jrnlAbsGroup" data-type="unstructured"><span data-rid="1702876731006" class="jrnlQueryRef" data-type="start" data-query-ref="true" data-query-for="author" data-query-from="preeditor" data-channel="query" data-topic="action" data-event="click" data-message="{'funcToCall': 'highlightQuery'}" data-query-type="action"> </span><p class="jrnlAbsPara" data-head-type="bold"><b>Abstract</b> Teleost fish exhibit remarkable sexual plasticity and divergent developmental systems, including sequential hermaphroditism. One of the more fascinating models of sexual plasticity is socially controlled sex change, which is often observed in coral reef fish. The Okinawa rubble goby, Trimma okinawae, is a bi-directional sex-changing fish. It can rapidly change sex in either direction based on social circumstances. Although behavioral and neuroendocrine sex change occurs immediately and is believed to trigger gonadal changes, the underlying mechanisms remain poorly understood. In this study, we conducted a de novo transcriptome analysis of the T. okinawae brain and identified genes that are differentially expressed between the sexes and genes that were immediately controlled by social stimulation implicating sex change. Several genes showed concordant expression shifts regardless of the sex change direction and were associated with histone modification in nerve cells. These genes are known to function in the neuroendocrine control of reproduction in nerve cells. Overall, we identified genes associated with the initiation of sex change, which provides insight into the regulation of sex change and sexual plasticity.</p></div><p class="jrnlKeywordPara"><span class="jrnlKeyword">keywords</span>, <span class="jrnlKeyword">sequential hermaphroditism</span>, <span class="jrnlKeyword">sexual plasticity</span>, <span class="jrnlKeyword">brain</span>, <span class="jrnlKeyword">transcriptomics</span>, <span class="jrnlKeyword"><i>trimma okinawae</i></span></p><div class="jrnlFNGroup" data-fn-type="contributors"><p class="jrnlFNHead" headweightage="45">Author Contributions</p><p class="jrnlFNPara">K.T., M.K. and T.S. designed the study. K.T. and T.S. conducted field sampling and animal treatments for experiment. K.T. generated and analyzed the data with assistance from T.M. and M.K. All authors wrote and reviewed the manuscript.</p></div></div><div class="body" data-edited="true" data-edited-version="2"><p class="jrnlDeleted" data-untag="0">https://orcid.org/0000-0001-8716-5438</p><p class="jrnlSecPara" data-head-type="bold"><b>Introduction</b> Sexual dimorphism is observed in a wide range of vertebrates. Females and males exhibit clear differences in behavior, physiology, and morphology [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="160" data-match="1,2" data-pattern="" data-tag-index="1_1" data-citation-string=" R1 R2 " data-track-change="1,2">1,2</span>]. In most species, the sexual fate of an individual is generally determined at an early phase of development [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="274" data-match="3,4" data-pattern="" data-tag-index="1_2" data-citation-string=" R3 R4 " data-track-change="3,4">3,4</span>]. Interestingly, teleosts exhibit divergent sex determination systems and retain extreme sexual plasticity throughout their life [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="408" data-match="5" data-pattern="" data-tag-index="1_3" data-citation-string=" R5 " data-track-change="5">5</span>]. Various sex-determination systems, including environmental and genetic sex determination as well as sex change (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="524" data-match="i.e.," data-pattern="" data-tag-index="1_5">i.e.,</span> sequential hermaphroditism), are observed in a diverse range of teleosts and their ecological and evolutionary roles have attracted significant interest [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="684" data-match="6" data-pattern="" data-tag-index="1_4" data-citation-string=" R6 " data-track-change="6">6</span>].</p><p class="jrnlSecPara">One of the most fascinating models of sexual plasticity is sex change regulated by social status in coral reef fish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="117" data-match="7,8" data-pattern="" data-tag-index="2_1" data-citation-string=" R7 R8 " data-track-change="7,8">7,8</span>]. Socially controlled sex change including protogyny (female-to-male), protandry (male-to-female), and bi-directional sex change (both directions serially) can occur rapidly [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="296" data-match="9" data-pattern="" data-tag-index="2_2" data-citation-string=" R9 " data-track-change="9">9</span>]. In particular, behavioral sex change can occur rapidly and within an hour following a social status change and before gonadal change, whereas changes in the brain trigger the sex change in the fish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="499" data-match="10,11" data-pattern="" data-tag-index="2_3" data-citation-string=" R10 R11 " data-track-change="10,11">10,11</span>]. Although gene expression change is likely to be involved in the initiation of sex change, transcriptional regulation in the brain during the early phase of sex change is poorly understood.</p><p class="jrnlSecPara">The Okinawa rubble goby, <i>Trimma okinawae,</i> can undergo bi-directional sex change. It is distributed between Kagoshima and southward to the Ryukyu Islands in Japan. A polygynous mating system occurs in <i>T. okinawae</i>, in which a dominant individual becomes male and exhibits territorial behavior to maintain the harem, whereas the others are females developed from juveniles or changed their sex and exhibit subordinate behavior [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="425" data-match="12" data-pattern="" data-tag-index="3_1" data-citation-string=" R12 " data-track-change="12">12</span>]. The dominance hierarchy of <i>T. okinawae</i> is based on body size within the harem. The largest female changes its sex to male if males leave the harem; however, these males can change back into females upon encountering a larger, more dominant fish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="676" data-match="12,13" data-pattern="" data-tag-index="3_2" data-citation-string=" R12 R13 " data-track-change="12,13">12,13</span>]. Because sex changes in both directions can be manipulated artificially and are reproducible in small aquariums, <i>T. okinawae</i> represents an excellent model for exploring its regulation during the initial phase at a genetic and molecular level.</p><p class="jrnlSecPara">To establish a transcriptomic signature associated with rapid sex change, RNA-sequencing was performed using the brains of T. okinawae and the resulting data was used for <span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>de novo</i>' to '<i>de novo</i>'" data-tag-type="HIGH" data-offset="171" data-match="de novo" data-replace="de novo" data-pattern="$1" data-tag-index="4_4" data-function="addTrackChanges">de novo</span> assembly. We selected fish that had only undergone behavioral sex change in each direction, but not gonadal sex change, by artificially replacing individuals in an aquarium. Gene expression profiles were compared between fish that had behaviorally changed sex with those of stable males and females who did not undergo sex change for<span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="OPR" data-tag-type="OPR" data-offset="512" data-match="for ≥1 month" data-replace=" ≥" data-pattern="default" data-tag-index="4_1"> ≥</span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL" data-tag-type="VAL" data-offset="516" data-match="for ≥1 month" data-replace="1" data-pattern="$1" data-tag-index="4_1">1</span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="UNT" data-tag-type="UNT" data-offset="517" data-match="for ≥1 month" data-replace=" month" data-pattern=" $2" data-tag-index="4_1"> month</span> in the laboratory. The transcriptomic changes were identified in the brain associated with the rapid changes occurring in response to social stimuli. Candidate genes implicated in rapid sex change were identified, which provide insight into the regulation of rapid sex change and the sexual plasticity of teleosts.</p><h1 class="jrnlHead1" headweightage="45">Materials and methods</h1><p class="jrnlSecPara"><b>Sample collection and experimental operation</b> Fish were collected from a wild population by scuba diving in <span class="jrnlUncitedRef" data-tag-desc="AUTHORYEARREF" title="Makurazaki 2016" data-tag-type="AUTHORYEARREF" data-offset="108" data-match="Makurazaki, Kagoshima Prefecture, Japan, in May 2016, and " data-pattern="" data-tag-index="6_5" data-uncited="true">Makurazaki, Kagoshima Prefecture, Japan, in May 2016, and</span> transported to the laboratory. To form a stable polygynous harem, we inserted a polyvinyl chloride tube as a nest site in each aquarium (<span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="303" data-match="60 cm" data-replace="60" data-pattern="$1" data-tag-index="6_6">60</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="305" data-match="60 cm" data-replace=" cm" data-pattern=" $2" data-tag-index="6_6"> cm</span> width, <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="316" data-match="36 cm" data-replace="36" data-pattern="$1" data-tag-index="6_7">36</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="318" data-match="36 cm" data-replace=" cm" data-pattern=" $2" data-tag-index="6_7"> cm</span> height, and <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="334" data-match="30 cm" data-replace="30" data-pattern="$1" data-tag-index="6_8">30</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="336" data-match="30 cm" data-replace=" cm" data-pattern=" $2" data-tag-index="6_8"> cm</span> depth) and maintained the harem for more than two weeks. Brains were harvested from males and females from stable harems as well as from behaviorally sex-changed males and females (five individuals per condition) <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="553" data-match="1 h" data-replace="1" data-pattern="$1" data-tag-index="6_9">1</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="554" data-match="1 h" data-replace=" h" data-pattern=" $2" data-tag-index="6_9"> h</span> after manipulation. We defined females behaving like males, i.e., showing territorial and nest-cleaning behavior, as male-like females, while males behaving like females, i.e., showing subordinate behavior and leaving the nest tube, as female-like males, <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="812" data-match="1 h" data-replace="1" data-pattern="$1" data-tag-index="6_10">1</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="813" data-match="1 h" data-replace=" h" data-pattern=" $2" data-tag-index="6_10"> h</span> after artificial manipulation (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F1 " data-tag-type="FIGTBLREF" data-offset="847" data-match="Fig. 1" data-pattern="" data-tag-index="6_1" data-citation-string=" F1 " data-anchor="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343756">Fig. 1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343756">figure 1</span></span>). Male-like females were obtained artificially by removing a dominant male from the harem. This is a stimulus for female-to-male sex change as the largest female in the harem adopts typical male behavior. We selected this female one hour after the removal of the dominant male, and it was designated a male-like female (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F1 " data-tag-type="FIGTBLREF" data-offset="1174" data-match="Fig. 1A" data-pattern="" data-tag-index="6_2" data-citation-string=" F1 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343757">Fig. 1A</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343757">figure 1A</span></span>). Meanwhile, female-like males were obtained through artificial encounters of males from two distinct harems. The males from two different tanks were moved to an experimental tank and acclimated for <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="1381" data-match="30 min" data-replace="30" data-pattern="$1" data-tag-index="6_12">30</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="1383" data-match="30 min" data-replace=" min" data-pattern=" $2" data-tag-index="6_12"> min</span>. During this time, each male was placed in an opaque cage (made of plastic netting and a thick PVC tube), so that they could not recognize one another. After removing the opaque cages, the males began to fight with one another over the PVC tube nest, which is a stimulus for a male-to-female sex change. The dominant male continued to behave as a male, whereas the loser began to exhibit typical female behavior. We selected the losing male one hour after the removal of the opaque cages and this male was designated a female-like male (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F1 " data-tag-type="FIGTBLREF" data-offset="1925" data-match="Fig. 1B" data-pattern="" data-tag-index="6_3" data-citation-string=" F1 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343758">Fig. 1B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343758">figure 1B</span></span>). The gonadal sex state of each fish was confirmed by urogenital ﻿papillae structure, which depends on gonadal sexes and manifests as a long tapered posterior in males and a bulbous appearance in females [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="2140" data-match="12,14,15" data-pattern="" data-tag-index="6_4" data-citation-string=" R12 R14 R15 " data-track-change="12,14,15">12,14,15</span>]. The animal experiments were approved by the committee of Tohoku University (Permit number: <span class="jrnlPatterns" data-tag-desc="THSE" title="THSE" data-tag-type="THSE" data-offset="2242" data-match="2017" data-replace="2017" data-pattern="2017" data-tag-index="6_17">2017</span>LsA-023). The fish were euthanized with ﻿an overdose of ethyl 3-aminobenzoate methanesulfonate (MS-222; Sigma-Aldrich, St. Louis, MO; <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="2382" data-match="100 mg/l" data-replace="100" data-pattern="$1" data-tag-index="6_13">100</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="2385" data-match="100 mg/l" data-replace=" mg/l" data-pattern=" $2" data-tag-index="6_13"> mg/l</span>), their brains were immediately excised, placed into tubes containing RNAlater stabilization solution (Thermo Fisher Scientific) within <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="2527" data-match="30 min" data-replace="30" data-pattern="$1" data-tag-index="6_14">30</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="2529" data-match="30 min" data-replace=" min" data-pattern=" $2" data-tag-index="6_14"> min</span> after taking the fish from the aquarium. The samples were stored at −<span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="2605" data-match="80°C" data-replace="80" data-pattern="$1" data-tag-index="6_15">80</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="2607" data-match="80°C" data-replace="°C" data-pattern="°C" data-tag-index="6_15">°C</span> until RNA extraction.</p><div class="jrnlFigBlock" id="BLK_F1" data-id="BLK_F1" data-stream-name="a_F1"><p class="jrnlFigCaption" id="F1" data-id="F1"><span class="label" title=" F1 " data-offset="0" data-match="Figure 1">Figure 1.</span>The sampling method of male-like females and female-like males. (<span class="partLabel" title="PLBL" data-offset="75" data-match="A" data-pattern="" data-tag-index="">A</span>)<span class="nonBreakingSpace"> </span>The connected line consistently represents the largest female in the panels. The largest female behaving like a male, <span class="jrnlPatterns" data-tag-desc="EGNR" title="EGNR" data-tag-type="EGNR" data-offset="196" data-match="Figure 1. The sampling method of male-like females and female-like males. (A) The connected line consistently represents the largest female in the panels. The largest female behaving like a male, e.g.," data-pattern="" data-tag-index="30_2">e.g.,</span> clean-up in the nest, territorial behavior, and courtship behavior, was sampled as a male-like female. (<span class="partLabel" title="PLBL" data-offset="306" data-match="B" data-pattern="" data-tag-index="">B</span>)<span class="nonBreakingSpace"> </span>The connected line consistently represents a smaller male in the panels. The losing male exhibiting subordinate behavior, <span class="jrnlPatterns" data-tag-desc="EGNR" title="EGNR" data-tag-type="EGNR" data-offset="431" data-match=" clean-up in the nest, territorial behavior, and courtship behavior, was sampled as a male-like female. (B) The connected line consistently represents a smaller male in the panels. The losing male exhibiting subordinate behavior, e.g.," data-pattern="" data-tag-index="30_3">e.g.,</span> accepting courtship behavior from the male, being expelled from the territory, and keeping a distance from the nest tube, was considered a female-like male.</p><img src="/resources/_cache/FPO.tif.JPEG"></div><h2 class="jrnlHead2" headweightage="41">RNA extraction and <span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>de novo</i>' to '<i>de novo</i>'" data-tag-type="HIGH" data-offset="19" data-match="de novo" data-replace="de novo" data-pattern="$1" data-tag-index="7_1" data-function="addTrackChanges">de novo</span> assembly</h2><p class="jrnlSecPara">Total RNA was extracted and purified from whole brain using a Maxwell 16LEV simply RNA tissue kit (Promega) based on the manufacturer’s instructions. RNA was extracted from each of five females, males, male-like females, and female-like males. In total, 20 RNA libraries were prepared using the NEBNext Ultra RNA Library Prep Kit for Illumina, and <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="350" data-match="150 bp" data-replace="150" data-pattern="$1" data-tag-index="8_11">150</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="353" data-match="150 bp" data-replace=" bp" data-pattern=" $2" data-tag-index="8_11"> bp</span> paired-end reads were generated on the Illumina HiSeq <span class="jrnlPatterns" data-tag-desc="THSE" title="THSE" data-tag-type="THSE" data-offset="411" data-match="4000" data-replace="4000" data-pattern="4000" data-tag-index="8_14">4000</span> platform. The quality of the sequencing reads were assessed using FastQC v0.11.7 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="498" data-match="16" data-pattern="" data-tag-index="8_1" data-citation-string=" R16 " data-track-change="16">16</span>] and low-quality reads and bases (<span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR" data-tag-type="VAR" data-offset="535" data-match="Q &amp;lt; 20" data-replace="Q" data-pattern="$1" data-tag-index="8_10">Q</span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="OPR" data-tag-type="OPR" data-offset="536" data-match="Q &amp;lt; 20" data-replace=" < " data-pattern="default" data-tag-index="8_10"> &lt; </span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL" data-tag-type="VAL" data-offset="539" data-match="Q &amp;lt; 20" data-replace="20" data-pattern="$3" data-tag-index="8_10">20</span>) were trimmed and discarded using Trimmomatic v0.36 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="595" data-match="17" data-pattern="" data-tag-index="8_2" data-citation-string=" R17 " data-track-change="17">17</span>]. <span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>De novo</i>' to '<i>De novo</i>'" data-tag-type="HIGH" data-offset="600" data-match="De novo" data-replace="De novo" data-pattern="$1" data-tag-index="8_37" data-function="addTrackChanges"><i>De novo</i></span> assembly was done using high-quality reads of all <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="658" data-match="20 samples" data-replace="20" data-pattern="$1" data-tag-index="8_12">20</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="660" data-match="20 samples" data-replace=" samples" data-pattern=" $2" data-tag-index="8_12"> samples</span> by Trinity v2.6.5 using default parameters [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="713" data-match="18" data-pattern="" data-tag-index="8_3" data-citation-string=" R18 " data-track-change="18">18</span>]. Subsequently, protein-coding sequences were predicted with TransDecoder v5.1.0 ﻿(<a href="https://github.com/TransDecoder/TransDecoder">https://github.com/TransDecoder/TransDecoder</a>) using default parameters. To remove redundancy from protein-coding sequences, CD-hit v4.6.8 (cd-hit-est) was used with a sequence identity threshold of -c 0.95 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1008" data-match="19" data-pattern="" data-tag-index="8_4" data-citation-string=" R19 " data-track-change="19">19</span>]. The comprehensiveness of the transcriptome dataset was assessed using the Benchmarking Universal Single-Copy Ortholog (v3) with the Actinopterygii dataset [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1169" data-match="20,21" data-pattern="" data-tag-index="8_5" data-citation-string=" R20 R21 " data-track-change="20,21">20,21</span>]. To search for orthologs and perform functional annotation, estimated protein-coding sequences were analyzed by eggNOG-mapper v2 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1306" data-match="22" data-pattern="" data-tag-index="8_6" data-citation-string=" R22 " data-track-change="22">22</span>].</p><h2 class="jrnlHead2" headweightage="41">Gene expression analysis and <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="29" data-match="gene ontology" data-pattern="" data-tag-index="" data-abbr="GO">gene ontology</span> enrichment</h2><p class="jrnlSecPara">RSEM v1.3.0 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="13" data-match="23" data-pattern="" data-tag-index="10_3" data-citation-string=" R23 " data-track-change="23">23</span>] with bowtie2 v2.3.5 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="38" data-match="24" data-pattern="" data-tag-index="10_4" data-citation-string=" R24 " data-track-change="24">24</span>] was used to map back the reads and calculate the gene-level expression abundance for each sample using high-quality reads with default parameters. Pairwise comparisons of gene expression levels of Actinopterygii orthologous genes estimated by eggNOG-mapper v2 for stable females vs. males, females vs. male-like females, and males vs. female-like males were done using the Wald test in DEseq2 v1.36.0 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="444" data-match="25" data-pattern="" data-tag-index="10_5" data-citation-string=" R25 " data-track-change="25">25</span>]. Genes were filtered out with very low expression (sum of expected count<span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="OPR" data-tag-type="OPR" data-offset="520" data-match="count &amp;lt; 10" data-replace=" < " data-pattern="default" data-tag-index="10_13"> &lt; </span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL" data-tag-type="VAL" data-offset="523" data-match="count &amp;lt; 10" data-replace="10" data-pattern="$3" data-tag-index="10_13">10</span>). Genes with a <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="541" data-match="false discovering rate" data-pattern="" data-tag-index="" data-abbr="FDR">false discovering rate</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="565" data-match=" (FDR)" data-pattern="" data-tag-index="10_15">FDR</span>) less than 0.05 were considered <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="601" data-match="differentially expressed genes" data-pattern="" data-tag-index="" data-abbr="DEG">differentially expressed genes</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="633" data-match=" (DEGs)" data-pattern="" data-tag-index="10_16">DEGs</span>). As another approach, to detect genes that were associated with changes in behavior, gonadal sex, and social stimulation, a <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="763" data-match="likelihood ratio test" data-pattern="" data-tag-index="">likelihood ratio test</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="786" data-match=" (LRT)" data-pattern="" data-tag-index="10_17">LRT</span>) was performed using DEseq2 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="819" data-match="25" data-pattern="" data-tag-index="10_6" data-citation-string=" R25 " data-track-change="25">25</span>]. We established the full model consisting of explanatory variables, <span class="jrnlPatterns" data-tag-desc="IENR" title="IENR" data-tag-type="IENR" data-offset="891" data-match="LRT) was performed using DEseq2 (25). We established the full model consisting of explanatory variables, i.e.," data-pattern="" data-tag-index="10_19">i.e.,</span> behavioral sex (female/female-like male or male/male-like female), gonadal sex (female/male-like female or male/female-like male), and social stimulation (female/male or female-like male/male-like female) (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F2 " data-tag-type="FIGTBLREF" data-offset="1103" data-match="Fig. 2" data-pattern="" data-tag-index="10_1" data-citation-string=" F2 " data-anchor="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343759">Fig. 2</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343759">figure 2</span></span>). Then, LRTs were performed between the full model and each single model reducing a variable to detect the effect of each variable on the gene expression (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F2 " data-tag-type="FIGTBLREF" data-offset="1265" data-match="Fig. 2B" data-pattern="" data-tag-index="10_2" data-citation-string=" F2 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343760">Fig. 2B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343760">figure 2B</span></span>). If LRT using a reduced model focusing on behavioral sex detects significant differences of the likelihood for a given gene, the gene is assumed to be a behavioral sex-associated gene. Using this approach, we addressed only consistent gene expression changes associated with each variable (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="1564" data-match="e.g.," data-pattern="" data-tag-index="10_20">e.g.,</span> concordant upregulation in male-like females and female-like males by social stimulation). The genes detected from these analyses were used for <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="1714" data-match="gene ontology" data-pattern="" data-tag-index="" data-abbr="GO">gene ontology</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="1729" data-match=" (GO)" data-pattern="" data-tag-index="10_18">GO</span>) enrichment analysis based on GO annotations from eggNOG-mapper v2 using the GOseq v1.48.0 [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1824" data-match="26" data-pattern="" data-tag-index="10_7" data-citation-string=" R26 " data-track-change="26">26</span>]. The FDR was calculated using overrepresented <span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>p</i> values' to '<i>p</i> values'" data-tag-type="HIGH" data-offset="1874" data-match="p values" data-replace="p values" data-pattern="$1" data-tag-index="10_66" data-function="addTrackChanges">p values</span> by the Benjamini-Hochberg method [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1917" data-match="27" data-pattern="" data-tag-index="10_8" data-citation-string=" R27 " data-track-change="27">27</span>] and GO terms with an <span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAR" data-tag-type="VAR" data-offset="1942" data-match="FDR &amp;lt;0.05 were" data-replace="FDR" data-pattern="$1" data-tag-index="10_14">FDR</span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="OPR" data-tag-type="OPR" data-offset="1945" data-match="FDR &amp;lt;0.05 were" data-replace=" <" data-pattern="default" data-tag-index="10_14"> &lt;</span><span class="jrnlPatterns" data-tag-desc="VAR,OPR,VAL,OPRS,VALS,UNT" title="VAL" data-tag-type="VAL" data-offset="1947" data-match="FDR &amp;lt;0.05 were" data-replace="0.05" data-pattern="$3" data-tag-index="10_14">0.05</span> were reported as significantly enriched GO terms. All statistical analyses were conducted using R 4.2.1 software (<a href="https://www.r-project.org/">https://www.r-project.org/</a>).</p><div class="jrnlFigBlock" id="BLK_F2" data-id="BLK_F2" data-stream-name="a_F2"><p class="jrnlFigCaption" id="F2" data-id="F2"><span class="label" title=" F2 " data-offset="0" data-match="Figure 2">Figure 2.</span>Overview of the LRT approach by each test using DEseq2. (<span class="partLabel" title="PLBL" data-offset="67" data-match="A" data-pattern="" data-tag-index="">A</span>)<span class="nonBreakingSpace"> </span>Definition of explanation variables of the LRT in the present analysis. (<span class="partLabel" title="PLBL" data-offset="143" data-match="B" data-pattern="" data-tag-index="">B</span>)<span class="nonBreakingSpace"> </span>The full model and reduced models used for each LRT.</p><img src="/resources/_cache/FPO.tif.JPEG"></div><h1 class="jrnlHead1" headweightage="45">Results</h1><p class="jrnlSecPara"><span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>De novo</i>' to '<i>De novo</i>'" data-tag-type="HIGH" data-offset="0" data-match="De novo" data-replace="De novo" data-pattern="$1" data-tag-index="12_15" data-function="addTrackChanges"><i>De novo</i></span> assembly and filtering resulted in <span class="jrnlPatterns" data-tag-desc="THSE" title="changed '177,631' to '1 77 631'" data-tag-type="THSE" data-offset="43" data-match="177,631" data-replace="1 77 631" data-pattern="1 77 631" data-tag-index="12_11" data-track-change="177,631">1 77 631</span> contigs coding for putative proteins (<span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST1 " data-tag-type="SUPPLFTREF" data-offset="89" data-match="Table S1" data-pattern="" data-tag-index="12_1" data-citation-string=" ST1 " data-old-class="jrnlSupplRef" data-uncited="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343761">Table S1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343761">table S1</span></span>). An assessment of the completeness of the transcriptome assembly indicated that <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="179" data-match="82.1%" data-replace="82.1" data-pattern="$1" data-tag-index="12_6">82.1</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="183" data-match="82.1%" data-replace="%" data-pattern="%" data-tag-index="12_6">%</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="184" data-match="82.1% (56.3%" data-pattern="" data-tag-index="12_5"> (56.3%</span> complete and <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="205" data-match="25.8%" data-replace="25.8" data-pattern="$1" data-tag-index="12_8">25.8</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="209" data-match="25.8%" data-replace="%" data-pattern="%" data-tag-index="12_8">%</span> fragmented) single-copy orthologs were present, whereas <span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="267" data-match="17.9%" data-replace="17.9" data-pattern="$1" data-tag-index="12_9">17.9</span><span class="jrnlPatterns" data-tag-desc="VAL,UNT" title="UNT" data-tag-type="UNT" data-offset="271" data-match="17.9%" data-replace="%" data-pattern="%" data-tag-index="12_9">%</span> were missing. The putative proteins coded in the contigs were subjected to eggNOG-mapper v2 and annotated by taxonomic classification (<span class="jrnlSupplRef" data-tag-desc="SUPPLFTREF" title=" SF1 " data-tag-type="SUPPLFTREF" data-offset="408" data-match="Fig. S1" data-pattern="" data-tag-index="12_2" data-citation-string=" SF1 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343762">Fig. S1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343762">figure S1</span></span>). Although most were classified into Actinopterygii orthologs (<span class="jrnlPatterns" data-tag-desc="THSE" title="changed '77,703' to '77 703'" data-tag-type="THSE" data-offset="479" data-match="77,703" data-replace="77 703" data-pattern="77 703" data-tag-index="12_12" data-track-change="77,703">77 703</span> out of <span class="jrnlPatterns" data-tag-desc="THSE" title="changed '129,505' to '1 29 505'" data-tag-type="THSE" data-offset="493" data-match="129,505" data-replace="1 29 505" data-pattern="1 29 505" data-tag-index="12_13" data-track-change="129,505">1 29 505</span> proteins), the remainder were classified into orthologs of a different or uncertain taxon. The genes coding for Actinopterygii ortholog proteins included <span class="jrnlPatterns" data-tag-desc="THSE" title="changed '29,762' to '29 762'" data-tag-type="THSE" data-offset="655" data-match="29,762" data-replace="29 762" data-pattern="29 762" data-tag-index="12_14" data-track-change="29,762">29 762</span> putative protein-coding genes expressed in the brain of <i>T. okinawae</i>, which we used for subsequent analyses.</p><p class="jrnlSecPara">The gene expression profiles revealed an unclear separation among the four groups (<span class="jrnlSupplRef" data-tag-desc="SUPPLFTREF" title=" SF2 " data-tag-type="SUPPLFTREF" data-offset="83" data-match="Fig. S2" data-pattern="" data-tag-index="13_8" data-citation-string=" SF2 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343763">Fig. S2</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343763">figure S2</span></span>). A pairwise comparison between stable females vs. males revealed <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="157" data-match="6" data-pattern="" data-tag-index="13_21">6</span> female-biased and <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="177" data-match="3" data-pattern="" data-tag-index="13_22">3</span> male-biased DEGs (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="197" data-match="Fig. 3A" data-pattern="" data-tag-index="13_1" data-citation-string=" F3 " data-anchor="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343764">Fig. 3A</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343764">figure 3A</span></span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST2 " data-tag-type="SUPPLFTREF" data-offset="206" data-match="Table S2" data-pattern="" data-tag-index="13_9" data-citation-string=" ST2 " data-old-class="jrnlSupplRef" data-uncited="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343765">Table S2</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343765">table S2</span></span>). We also identified 13 DEGs between females vs. male-like females, and <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="287" data-match="8" data-pattern="" data-tag-index="13_23">8</span> female-biased and <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="307" data-match="5" data-pattern="" data-tag-index="13_24">5</span> male-like female-biased genes (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="340" data-match="Fig. 3A" data-pattern="" data-tag-index="13_2" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343766">Fig. 3A</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343766">figure 3A</span></span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST3 " data-tag-type="SUPPLFTREF" data-offset="349" data-match="Table S3" data-pattern="" data-tag-index="13_10" data-citation-string=" ST3 " data-old-class="jrnlSupplRef" data-uncited="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343767">Table S3</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343767">table S3</span></span>). In contrast to the other two comparisons, we identified 228 DEGs between males vs. female-like males and 84 of these were male-biased and 144 were female-like male-biased genes (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="538" data-match="Fig. 3A" data-pattern="" data-tag-index="13_3" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343768">Fig. 3A</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343768">figure 3A</span></span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST4 " data-tag-type="SUPPLFTREF" data-offset="547" data-match="Table S4" data-pattern="" data-tag-index="13_11" data-citation-string=" ST4 " data-old-class="jrnlSupplRef" data-uncited="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343769">Table S4</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343769">table S4</span></span>). These DEGs for males vs. female-like males exhibited a significant enrichment in <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="639" data-match="“" data-pattern="" data-tag-index="13_26">“</span>pattern orientation<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="661" data-match="”" data-pattern="" data-tag-index="13_28">”</span> and <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="669" data-match="“" data-pattern="" data-tag-index="13_27">“</span>regulation of histone H4-K16 acetylation,<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="713" data-match="”" data-pattern="" data-tag-index="13_29">”</span> whereas no significant enrichment was observed for other comparisons (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="787" data-match="Fig. 3B" data-pattern="" data-tag-index="13_4" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343770">Fig. 3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343770">figure 3B</span></span>, <span class="jrnlTblRef" data-tag-desc="FIGTBLREF" title=" T1 " data-tag-type="FIGTBLREF" data-offset="796" data-match="Table 1" data-pattern="" data-tag-index="13_5" data-citation-string=" T1 " data-anchor="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343771">Table 1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343771">table 1</span></span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST5 ST6 ST7 " data-tag-type="SUPPLFTREF" data-offset="805" data-match="Table S5–7" data-pattern="" data-tag-index="13_12" data-citation-string=" ST5 ST6 ST7 " data-old-class="jrnlSupplRef" data-uncited="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343772">Table S5–7</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343772">tables S5–7</span></span>). Overlap of the DEGs from the three comparisons indicated that most of the genes were independently detected by each comparison (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="948" data-match="Fig. 3B" data-pattern="" data-tag-index="13_6" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343773">Fig. 3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343773">figure 3B</span></span>). The intersection of the three comparisons revealed only <span class="jrnlPatterns" data-tag-desc="SNUM" title="SNUM" data-tag-type="SNUM" data-offset="1014" data-match="1" data-pattern="" data-tag-index="13_25">1</span> gene, which is <i>KIAA0100</i> ortholog. With respect to genes immediately controlled by social stimulation, 4 DEGs were detected by pairwise comparisons between females vs. male-like females and males vs. female-like males (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="1234" data-match="Fig. 3B" data-pattern="" data-tag-index="13_7" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343774">Fig. 3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343774">figure 3B</span></span>).</p><div class="jrnlFigBlock" id="BLK_F3" data-id="BLK_F3" data-stream-name="a_F3"><p class="jrnlFigCaption" id="F3" data-id="F3"><span class="label" title=" F3 " data-offset="0" data-match="Figure 3">Figure 3.</span><span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="10" data-match="Differentially expressed genes" data-pattern="" data-tag-index="" data-abbr="DEG">Differentially expressed genes</span> detected by pairwise comparisons. (<span class="partLabel" title="PLBL" data-offset="76" data-match="A" data-pattern="" data-tag-index="">A</span>)<span class="nonBreakingSpace"> </span>Schematic illustration and the number of <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="120" data-match="differentially expressed genes" data-pattern="" data-tag-index="" data-abbr="DEG">differentially expressed genes</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="152" data-match=" (DEGs)" data-pattern="" data-tag-index="32_2">DEGs</span>) between female vs. male, female vs. male-like female, and male vs. female-like male, respectively, are shown. The solid lines represent sexual differences and behavioral sex change focused on in this study, while dotted lines represent a gradual change after that, <span class="jrnlPatterns" data-tag-desc="IENR" title="IENR" data-tag-type="IENR" data-offset="423" data-match="A) Schematic illustration and the number of differentially expressed genes (DEGs) between female vs. male, female vs. male-like female, and male vs. female-like male, respectively, are shown. The solid lines represent sexual differences and behavioral sex change focused on in this study, while dotted lines represent a gradual change after that, i.e.," data-pattern="" data-tag-index="32_3">i.e.,</span> gonadal sex change. (<span class="partLabel" title="PLBL" data-offset="450" data-match="B" data-pattern="" data-tag-index="">B</span>)<span class="nonBreakingSpace"> </span>Venn diagram of a differentially expressed gene detected by three independent comparisons. The gene names next to the diagram in gray rectangles were obtained from eggNOG2-mapper annotation. The gene name colors, red and blue, represents upregulated and downregulated in female, respectively. The small arrows next to the gene name show upregulation in females.</p><img src="/resources/_cache/FPO.tif.JPEG"></div><p class="jrnlSecPara">To identify genes that consistently changed their expression based on sexual and social status in the brain, we conducted LRT to compare the full model with the reduced model (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F2 F4 " data-tag-type="FIGTBLREF" data-offset="176" data-match="Fig. 2, 4" data-pattern="" data-tag-index="14_1" data-citation-string=" F2 F4 " data-anchor="true"><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343775">Fig. 2, 4</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343775">figures 2 and 4</span></span>). The LRT using reduced models for each condition identified 36, 66, and 168 genes associated with behavioral sex, gonadal sex, and social stimulation, respectively (<span class="jrnlUncitedRef" data-tag-desc="SUPPLFIGREF" title=" SF4-S3 " data-tag-type="SUPPLFIGREF" data-offset="352" data-match="Fig. 4, Fig. S3" data-pattern="" data-tag-index="14_8" data-citation-string=" SF4-S3 " data-old-class="jrnlSupplRef" data-uncited="true">Fig. 4, Fig. S3</span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST8 ST9 ST10 " data-tag-type="SUPPLFTREF" data-offset="369" data-match="Table S8–10" data-pattern="" data-tag-index="14_10" data-citation-string=" ST8 ST9 ST10 " data-old-class="jrnlSupplRef" data-uncited="true">Table S8–10</span>); however, the genes detected by each test showed little overlap (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F4 " data-tag-type="FIGTBLREF" data-offset="449" data-match="Fig. 4A" data-pattern="" data-tag-index="14_3" data-citation-string=" F4 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343776">Fig. 4A</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343776">figure 4A</span></span>). Using this approach, several genes exhibited intermediate, but significant expression changes, which were not detected by pairwise comparisons. For the gonadal sex-associated genes, <span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="641" data-match="13 (19.7%)" data-pattern="" data-tag-index="14_15">13</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="643" data-match="13 (19.7%)" data-pattern="" data-tag-index="14_15"> (19.7%)</span> were also detected by pairwise comparisons and the other 53 were only detected by the LRT (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F4 " data-tag-type="FIGTBLREF" data-offset="743" data-match="Fig. 4B" data-pattern="" data-tag-index="14_4" data-citation-string=" F4 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343777">Fig. 4B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343777">figure 4B</span></span>). In contrast, <span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="766" data-match="20 (58.8%)" data-pattern="" data-tag-index="14_16">20</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="768" data-match="20 (58.8%)" data-pattern="" data-tag-index="14_16"> (58.8%)</span> and <span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="PER" data-tag-type="PER" data-offset="781" data-match="105 (62.5%)" data-pattern="" data-tag-index="14_17">105</span><span class="jrnlPatterns" data-tag-desc="PER,VAL,UNT" title="VAL" data-tag-type="VAL" data-offset="784" data-match="105 (62.5%)" data-pattern="" data-tag-index="14_17"> (62.5%)</span> genes were also identified by pairwise comparisons for the genes identified as behavioral sex-associated or social stimulation-associated, respectively, and most of the overlapped genes represented DEGs between males vs. female-like males (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F4 " data-tag-type="FIGTBLREF" data-offset="1033" data-match="Fig. 4C, D" data-pattern="" data-tag-index="14_5" data-citation-string=" F4 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343778">Fig. 4C, D</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343778">figure 4C and D</span></span>). GO enrichment analyses were performed and several overrepresented GO terms were identified, except for gonadal sex-associated genes (<span class="jrnlTblRef" data-tag-desc="FIGTBLREF" title=" T1 " data-tag-type="FIGTBLREF" data-offset="1179" data-match="Table 1" data-pattern="" data-tag-index="14_6" data-citation-string=" T1 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343779">Table 1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343779">table 1</span></span>, <span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST11 ST12 ST13 " data-tag-type="SUPPLFTREF" data-offset="1188" data-match="Table S11–13" data-pattern="" data-tag-index="14_11" data-citation-string=" ST11 ST12 ST13 " data-old-class="jrnlSupplRef" data-uncited="true">Table S11–13</span>). The behavioral sex-associated genes were significantly enriched in <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1272" data-match="“" data-pattern="" data-tag-index="14_20">“</span>regulation of locomotion.<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1300" data-match="”" data-pattern="" data-tag-index="14_23">”</span> The social stimulation-associated genes were significantly enriched in several GO terms including <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1402" data-match="“" data-pattern="" data-tag-index="14_21">“</span>pattern orientation<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1424" data-match="”" data-pattern="" data-tag-index="14_24">”</span> and <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1432" data-match="“" data-pattern="" data-tag-index="14_22">“</span>regulation of histone H4-K16 acetylation<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1475" data-match="”" data-pattern="" data-tag-index="14_25">”</span> (<span class="jrnlTblRef" data-tag-desc="FIGTBLREF" title=" T1 " data-tag-type="FIGTBLREF" data-offset="1480" data-match="Table 1" data-pattern="" data-tag-index="14_7" data-citation-string=" T1 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343780">Table 1</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343780">table 1</span></span>).</p><div class="jrnlFigBlock" id="BLK_F4" data-id="BLK_F4" data-stream-name="a_F4"><p class="jrnlFigCaption" id="F4" data-id="F4"><span class="label" title=" F4 " data-offset="0" data-match="Figure 4">Figure 4.</span>Venn diagram of the genes detected by the LRT approach. (<span class="partLabel" title="PLBL" data-offset="67" data-match="A" data-pattern="" data-tag-index="">A</span>)<span class="nonBreakingSpace"> </span>Overlaps of the genes detected using each reduced model are shown. (<span class="partLabel" title="PLBL" data-offset="138" data-match="B-D" data-pattern="" data-tag-index="">B-D</span>)<span class="nonBreakingSpace"> </span>Overlaps between the DEGs in the pairwise comparisons and the genes detected using reduced models focusing on gonadal sex, behavioral sex, and stimulated, respectively, are shown.</p><img src="/resources/_cache/FPO.tif.JPEG"></div><h1 class="jrnlHead1" headweightage="45">Discussion</h1><p class="jrnlSecPara">The transcriptomic changes occurring in the brain during the early phase of sex change are understudied. To our knowledge, previous studies using hermaphroditic species have primarily focused on changes in the gonads or the brain within a few days after the initiation of sex changes [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="285" data-match="28–30" data-pattern="" data-tag-index="16_1" data-citation-string=" R28 R29 R30 " data-track-change="28&amp;ndash;30">28–30</span>], except for recent work [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="319" data-match="31" data-pattern="" data-tag-index="16_2" data-citation-string=" R31 " data-track-change="31">31</span>]. Previous studies using gonochoristic [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="362" data-match="32,33" data-pattern="" data-tag-index="16_3" data-citation-string=" R32 R33 " data-track-change="32,33">32,33</span>] and hermaphrodite fish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="393" data-match="28–31,34" data-pattern="" data-tag-index="16_4" data-citation-string=" R28 R29 R30 R31 R34 " data-track-change="28&amp;ndash;31,34">28–31,34</span>] revealed few DEGs in the brain between the sexes. Our analysis also identified a small number of DEGs, which is consistent with the previous studies. Although the asymmetric treatment inducing sex change may affect the number of detected genes, especially through stress during transferring and fighting, we found several genes significantly associated with behavioral changes and social stimulation. In the present study, we could not exclude the effect of byproducts from artificial operations on gene expression, such as a winner effect following territorial conflicts and exhaustion after a fight. However, our analysis provides a unique dataset to examine the regulation of rapid responses in the brain during sex changes as well as a practical model system.</p><p class="jrnlSecPara">The KIAA0100 ortholog, which is a lipid transporter, known as BLTP2 or BCOX1 in humans, exhibited sexually dimorphic expression. The expression levels were changed oppositely depending on the direction of the sex change (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="221" data-match="i.e.," data-pattern="" data-tag-index="17_5">i.e.,</span> consistent with behavioral sex) (<span class="jrnlFigRef" data-tag-desc="FIGTBLREF" title=" F3 " data-tag-type="FIGTBLREF" data-offset="260" data-match="Fig. 3B" data-pattern="" data-tag-index="17_1" data-citation-string=" F3 "><span class="del cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343781">Fig. 3B</span><span class="ins cts-1" data-userid="preediting" data-username="Exeter" data-time="1702876742000" data-cid="1704613343781">figure 3B</span></span>). Although the information on other organisms is limited, the KIAA0100 ortholog in Drosophila melanogaster, hobbit, regulates neuroendocrine exocytosis and its mutation results in a marked reduction in body size, because it fails to promote insulin secretion [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="528" data-match="35,36" data-pattern="" data-tag-index="17_2" data-citation-string=" R35 R36 " data-track-change="35,36">35,36</span>]. Because body size is an important factor for sex change of T. okinawae [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="608" data-match="12" data-pattern="" data-tag-index="17_3" data-citation-string=" R12 " data-track-change="12">12</span>], the KIAA0100 ortholog may be associated with sex change through body size and growth regulation. Hobbit is required for the maintenance of endocrine signaling [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="773" data-match="35" data-pattern="" data-tag-index="17_4" data-citation-string=" R35 " data-track-change="35">35</span>], and therefore the ortholog of KIAA0100 is possibly associated with neuroendocrine regulation during the sex change in the brain of T. okinawae.</p><p class="jrnlSecPara">Although only a few DEGs were identified in the females vs. males comparison, several notable differences were detected in the gonadal sex-associated genes by the LRT. For example, genes of the ortholog <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="203" data-match="BPI fold containing family C" data-pattern="" data-tag-index="" data-abbr="BPIFC">BPI fold containing family C</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="233" data-match=" (BPIFC)" data-pattern="" data-tag-index="18_7">BPIFC</span>) were highly expressed in the brains of fish with female gonads (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="304" data-match="i.e.," data-pattern="" data-tag-index="18_8">i.e.,</span> gonads of female and male-like female) in T. okinawae (<span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST9 " data-tag-type="SUPPLFTREF" data-offset="365" data-match="Table S9" data-pattern="" data-tag-index="18_1" data-citation-string=" ST9 " data-old-class="jrnlSupplRef" data-uncited="true">Table S9</span>). A previous study using Zebrafish revealed that this gene modulates the expression of kiss2, an orthologous gene regulating reproductive development through GnRH neuronal activity in mammals [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="567" data-match="37" data-pattern="" data-tag-index="18_3" data-citation-string=" R37 " data-track-change="37">37</span>]. Interestingly, kisspeptin may have other neuroendocrinal functions on social and sexual behavior by regulating neurons that express isotocin and vasotocin, although it is not a direct regulator of reproduction through GnRH neurons in several fish species [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="828" data-match="38–40" data-pattern="" data-tag-index="18_4" data-citation-string=" R38 R39 R40 " data-track-change="38&amp;ndash;40">38–40</span>]. Because these neurotransmitters may control behavioral sex change in a social context [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="925" data-match="11,31" data-pattern="" data-tag-index="18_5" data-citation-string=" R11 R31 " data-track-change="11,31">11,31</span>], regulatory changes in these factors may also play an important role in T. okinawae.</p><p class="jrnlSecPara">In the pairwise comparison between males vs. female-like males, genes associated with the regulation of histone H4-K16 acetylation were enriched in the DEGs. Moreover, Autism Susceptibility Candidate 2 (AUTS2) and Sirtuin 1 (SIRT1) were identified as social stimulation-associated genes by the LRT. This suggests that these genes are regulated in the same manner by social stimulation, regardless of sex change direction (<span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST10 " data-tag-type="SUPPLFTREF" data-offset="422" data-match="Table S10" data-pattern="" data-tag-index="19_1" data-citation-string=" ST10 " data-old-class="jrnlSupplRef" data-uncited="true">Table S10</span>). AUTS2 is a component of the polycomb group multiprotein PRC1-like complex, which activates gene transcription by recruiting histone acetyltransferase. It also plays various roles in neuronal migration and neuritogenesis [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="655" data-match="41,42" data-pattern="" data-tag-index="19_3" data-citation-string=" R41 R42 " data-track-change="41,42">41,42</span>]. SIRT1 is a deacetylase that acts on histones and is abundantly expressed in the brain [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="750" data-match="43" data-pattern="" data-tag-index="19_4" data-citation-string=" R43 " data-track-change="43">43</span>]. Histone acetylation in the brain is involved in social behavior, including pair bonding in prairie voles and caste-related behavior in ants [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="896" data-match="44,45" data-pattern="" data-tag-index="19_5" data-citation-string=" R44 R45 " data-track-change="44,45">44,45</span>], and is suggested to play crucial roles in the gonadal sex change of hermaphrodite fish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="992" data-match="31,46" data-pattern="" data-tag-index="19_6" data-citation-string=" R31 R46 " data-track-change="31,46">31,46</span>]. In mammals, SIRT1 epigenetically regulates kiss1 expression and controls pubertal timing depending on environmental conditions (<span class="jrnlPatterns" data-tag-desc="EGIEPR" title="EGIEPR" data-tag-type="EGIEPR" data-offset="1128" data-match="i.e.," data-pattern="" data-tag-index="19_15">i.e.,</span> obesity and nutrition) [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1158" data-match="43" data-pattern="" data-tag-index="19_7" data-citation-string=" R43 " data-track-change="43">43</span>]. SIRT1 and AUTS2 may regulate the chromatin state in kisspeptin neurons or other neurons through social stimulation and sex change in T. okinawae. From this analysis, three AUTS2 orthologs were detected as both stimulation-associated genes and DEGs between male vs. female-like male. Enrichment in GO included <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1472" data-match="“" data-pattern="" data-tag-index="19_9">“</span>pattern orientation,<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1495" data-match="”" data-pattern="" data-tag-index="19_12">”</span> <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1499" data-match="“" data-pattern="" data-tag-index="19_10">“</span>innate vocalization behavior,<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1531" data-match="”" data-pattern="" data-tag-index="19_13">”</span> and <span class="jrnlPatterns" data-tag-desc="LQUOTE" title="LQUOTE" data-tag-type="LQUOTE" data-offset="1539" data-match="“" data-pattern="" data-tag-index="19_11">“</span>dendrite extension,<span class="jrnlPatterns" data-tag-desc="RQUOTE" title="RQUOTE" data-tag-type="RQUOTE" data-offset="1561" data-match="”" data-pattern="" data-tag-index="19_14">”</span> largely relies on these orthologs. AUTS2 is associated with various neuropathological conditions in humans and head size and neurodevelopment function in zebrafish [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="1730" data-match="41,47" data-pattern="" data-tag-index="19_8" data-citation-string=" R41 R47 " data-track-change="41,47">41,47</span>]. Therefore, the expression shift may contribute to the transition of the neurological state associated with sex change in T. okinawae.</p><p class="jrnlSecPara">The ortholog of lysine demethylase 3B (<i>KDM3B</i>), which is lysine 9 of histone H3 (H3K9) demethylase, was identified as a social stimulation-associated gene by the LRT method (<span class="jrnlUncitedRef" data-tag-desc="SUPPLFTREF" title=" ST10 " data-tag-type="SUPPLFTREF" data-offset="173" data-match="Table S10" data-pattern="" data-tag-index="20_1" data-citation-string=" ST10 " data-old-class="jrnlSupplRef">Table S10</span>). The methylation of H3K9 and H3K4 are hallmarks of transcriptionally suppressed and activated chromatin. <i>KDM3B</i> knock-out XY mice were frequently sex-reversed [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="343" data-match="48" data-pattern="" data-tag-index="20_3" data-citation-string=" R48 " data-track-change="48">48</span>]. A previous study of <i>KDM3B</i> knock-out mice revealed that circulating levels of 17β-estradiol increased and impaired male-specific sexual behavior [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="494" data-match="30" data-pattern="" data-tag-index="20_4" data-citation-string=" R30 " data-track-change="30">30</span>]. Several H3K9 demethylases are highly expressed in the brain of the androdioecious killifish, <i>Kryptolebias marmoratus</i>, and exhibit significant sexually biased expression [<span class="jrnlBibRef" data-tag-desc="BIBREF" data-tag-type="BIBREF" data-offset="669" data-match="49" data-pattern="" data-tag-index="20_5" data-citation-string=" R49 " data-track-change="49">49</span>]. Based on these findings, <i>KDM3B</i> expression in the brain may respond to social stimulation and activate the genes associated with sexual development or behavior in teleosts.</p><p class="jrnlSecPara">Our analysis proposed various genes, including novel candidates, associated with the initiation of sex changes in the brain. Future research requires a cautious interpretation of the function of these genes since some gene expressions in the brain are highly localized and therefore might be offset due to using the whole brain. The transcriptome dataset generated here provides potential targets for screening and sheds light on the initiation mechanism of sex changes in fish.</p><p class="jrnlDeleted" data-untag="1">The <span class="jrnlPatterns" data-tag-desc="HIGH" title="changed from '<i>de novo</i>' to '<i>de novo</i>'" data-tag-type="HIGH" data-offset="4" data-match="de novo" data-replace="de novo" data-pattern="$1" data-tag-index="25_3" data-function="addTrackChanges">de novo</span> transcriptome assembly, annotation of the transcriptome, RSEM outputs, and the code are deposited in the Dryad Digital Repository (<span class="jrnlPatterns" data-tag-desc="URL" title="URL" data-tag-type="URL" data-offset="143" data-match="https://doi.org/10.5061/dryad.vdncjsz0r" data-pattern="" data-tag-index="25_2">https://doi.org/10.5061/dryad.vdncjsz0r</span>). Raw sequence reads can be found in the SRA database under BioProject ID: PRJDB15183.</p></div><div class="back" data-edited="true" data-edited-version="2"><div class="jrnlRefGroup"><h1 class="jrnlRefHead" headweightage="45">References</h1><p id="R1" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="15452574" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/nn1325" data-validated-log="true" data-validated="true" seqid="R1" data-id="R1" data-valid-id="R1" new-id="R1"><span class="RefSlNo" id="1f529eb7-dd4e-4da0-b289-2f42630fc95c">1.</span> <span class="RefAuthor" id="79bfc570-33af-45ee-9b9f-560ccb8c11b0"><span class="RefSurName">Morris</span> <span class="RefGivenName">JA</span></span>, <span class="RefAuthor" id="472ae8ea-9162-4cca-a054-275bbd0f2269"><span class="RefSurName">Jordan</span> <span class="RefGivenName">CL</span></span>, <span class="RefAuthor" id="255827df-8912-4827-b54b-8e047068ee4d"><span class="RefSurName">Breedlove</span> <span class="RefGivenName">SM</span></span>. <span class="RefYear" id="6346ad8d-217b-485d-8bac-eb99f8afca88">2004</span> <span class="RefArticleTitle" id="4360923e-4904-4377-981b-8add6e044559">Sexual differentiation of the vertebrate nervous system</span>. <span class="RefJournalTitle" id="638e949c-6d11-4c8c-ab44-e47e134f7970"><i>Nat Neurosci</i></span>. <span class="RefVolume" id="474b7a23-2342-467f-9c4b-648ca5929aca"><b>7</b></span>, <span class="RefFPage" id="6b78da73-4cda-4148-a44f-833972049b9f">1034</span>–<span class="RefLPage" id="3bfb73a1-85ce-450d-b84e-c14cccec5f2a">1039</span>. (<span class="RefDOI" id="7f3a6ca7-4c39-43e9-a737-cd449a4859db">doi: 10.1038/nn1325</span>)</p><p id="R2" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1111/j.1439-0310.2005.01147.x" data-validated-log="true" data-validated="true" seqid="R2" data-id="R2" data-valid-id="R2" new-id="R2"><span class="RefSlNo" id="83137cb6-2732-4b92-baab-2e43a651ce7a">2.</span> <span class="RefAuthor" id="75d1f23a-dd05-4c83-acb0-1281eb8fd486"><span class="RefSurName">Blanckenhorn</span> <span class="RefGivenName">WU</span></span>. <span class="RefYear" id="5bcb02c5-546d-465b-b388-b01946bf4896">2005</span> <span class="RefArticleTitle" id="4e319248-d9cd-47f8-a039-71c42d4f7d8b">Behavioral Causes and Consequences of Sexual Size Dimorphism</span>. <span class="RefJournalTitle" id="a9a9a05c-347e-44f3-80e8-617a69b8136c"><i>Ethology</i></span>. <span class="RefVolume" id="4978b7e7-58f2-4304-bef4-8c2d85d5e264"><b>111</b></span>, <span class="RefFPage" id="4e2a3e9e-76ff-486d-b6e2-2b9bb29540ee">977</span>–<span class="RefLPage" id="0fa378c9-27e7-4a08-b25a-12b3b4d674ce">1016</span>. (<span class="RefDOI" id="3ca0be72-8665-4915-ae52-93c9da1267fc">doi: 10.1111/j.1439-0310.2005.01147.x</span>)<span class="RefWebSite" id="d7ceac05-adc1-416d-b588-e128ec932108">https://onlinelibrary.wiley.com/toc/14390310/111/11</span></p><p id="R3" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="19834484" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/nrg2687" data-validated-log="true" data-validated="true" seqid="R3" data-id="R3" data-valid-id="R3" new-id="R3"><span class="RefSlNo" id="e072a754-a039-4111-bcc0-12487e930b0a">3.</span> <span class="RefAuthor" id="dfafd6db-85c0-4733-a0d4-0815d436142b"><span class="RefSurName">Williams</span> <span class="RefGivenName">TM</span></span>, <span class="RefAuthor" id="8c186dd9-c845-4d4c-aa1f-815c94b303e7"><span class="RefSurName">Carroll</span> <span class="RefGivenName">SB</span></span>. <span class="RefYear" id="178da8d0-fe9f-4152-a25c-e918b254c5f3">2009</span> <span class="RefArticleTitle" id="92e4f32b-eb45-408d-9fe1-40db61efded0">Genetic and molecular insights into the development and evolution of sexual dimorphism</span>. <span class="RefJournalTitle" id="2321ce72-5def-4698-a2b6-a20de97c169c"><i>Nat Rev Genet</i></span>. <span class="RefVolume" id="c1d2da65-c508-4929-b3d0-01168aa49c21"><b>10</b></span>, <span class="RefFPage" id="5ecb9cee-f731-4a97-8c77-7988ea6b4419">797</span>–<span class="RefLPage" id="8b958f34-fecf-4871-b0ba-37cccce85f6f">804</span>. (<span class="RefDOI" id="1aebfe67-0a21-47fc-8ee1-40a4952a7dd7">doi: 10.1038/nrg2687</span>)<span class="RefWebSite" id="e4df87c0-c7c8-4214-b8a0-d220786c0499">http://dx.doi.org/10.1038/nrg2687</span></p><p class="jrnlRefText" seqid="R4" data-id="R4" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-doi="10.1038/nrg.2017.60" data-class-name="RefText" id="R4" data-valid-id="R4" new-id="R4"><span class="RefSlNo">4.</span> <span class="RefAuthor" id="c41320d4-82cc-4b36-b3a4-b7b53c036925"><span class="RefSurName" id="61fc144d-19cb-4f28-8817-12b4bef48c26">Capel</span> <span class="RefGivenName" id="fe0f2642-297a-4aee-9f29-17dcbd8d00e8">B.</span></span> <span class="RefArticleTitle" id="6c68368c-3486-4054-9697-d8fd45527156">Vertebrate sex determination: Evolutionary plasticity of a fundamental switch</span>. <span class="RefJournalTitle" id="6ab363c3-5d2a-4b1c-91e9-f84058b9e0c5">Nat Rev Genet</span>. <span class="RefYear" id="ace092ab-4db2-4880-9415-d1aab847f684">2017</span>;<span class="RefVolume" id="79a8b1bf-a540-45f8-910d-cfcab4bf0a06">18</span>(<span class="RefIssue" id="29c89817-6036-4075-b988-d9a1031f5142">11</span>):<span class="RefFPage" id="1c92b9fb-91b7-401e-9186-4444576df158">675</span>–<span class="RefLPage" id="07fd21eb-2515-4c34-bc44-e8d9a243516d">89</span>. <span class="RefWebSite" id="6f542e38-3990-43b2-8d85-fe446c71ed95">http://dx.doi.org/10.1038/nrg.2017.60</span></p><p class="jrnlRefText" seqid="R5" data-id="R5" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-doi="10.1007/s10228-020-00754-6" data-class-name="RefText" id="R5" data-valid-id="R5" new-id="R5"><span class="RefSlNo">5.</span> <span class="RefAuthor" id="5b02c8d8-9cd3-424f-b36f-6d3f726e6c40"><span class="RefSurName" id="ebe9a8b7-b6ee-422e-a8f3-dec41b186cc1">Kuwamura</span> <span class="RefGivenName" id="391d9673-9bb6-466f-8eb4-34f618ba00fb">T</span></span>, <span class="RefAuthor" id="34a9e83f-74f6-49b3-a56c-395a69bb5036"><span class="RefSurName" id="b5a1e0c7-1faf-45d9-b33b-06042369c698">Sunobe</span> <span class="RefGivenName" id="5543046b-87bc-4ec0-af49-1c4286e8b24a">T</span></span>, <span class="RefAuthor" id="633316cd-46d8-4eae-bb7e-3246a17849cb"><span class="RefSurName" id="c48887fb-6ac2-41e2-a835-129b7ce5b35d">Sakai</span> <span class="RefGivenName" id="89800e12-d5c8-4071-9720-0019f7caafb1">Y</span></span>, <span class="RefAuthor" id="cd4743ae-96b6-480e-b404-8ad26767a2a2"><span class="RefSurName" id="cdfb494c-fca6-4e9b-a48b-f5a71c17c8ef">Kadota</span> <span class="RefGivenName" id="663b5915-f197-4ea5-ae97-2c207bf213b9">T</span></span>, <span class="RefAuthor" id="c4be530b-2f11-4328-8566-ad825661615e"><span class="RefSurName" id="4fbea541-0763-48a2-b1bc-544b7131c2dd">Sawada</span> <span class="RefGivenName" id="bffe5067-735b-4f94-b261-d8be4c132c9a">K.</span></span> <span class="RefArticleTitle" id="e2e7f047-0669-4fec-9557-da6a3d0ae849">Hermaphroditism in fishes: an annotated list of species, phylogeny, and mating system</span>. <span class="RefJournalTitle" id="d3d42a7f-1acc-4a3c-986a-68f75f13f04b">Ichthyol Res</span>. <span class="RefYear" id="8a406bcb-6c59-4f8d-9d82-fd8759b75f33">2020</span>;<span class="RefVolume" id="c9f905bd-c4d1-463e-be4b-07e168ab0aa0">67</span>(<span class="RefIssue" id="f53d1e85-e52b-41c0-a862-a4a02eb7675a">3</span>):<span class="RefFPage" id="259cc35b-2b7b-4b62-b159-f06863ebc98a">341</span>–<span class="RefLPage" id="ac558998-710e-484d-ad2f-af152998ca58">60</span>. <span class="RefWebSite" id="9eb3f1cd-c881-448e-b21d-7c9f89542419">https://doi.org/10.1007/s10228-020-00754-6</span></p><p class="jrnlRefText" seqid="R6" data-id="R6" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-class-name="RefText" id="R6" data-valid-id="R6" new-id="R6"><span class="RefSlNo">6.</span> <span class="RefAuthor" id="c2daa126-bb3d-461f-ae74-ea2f325bea7a"><span class="RefSurName" id="d83dc041-b8d8-4ee3-abd5-2a70b6c47d2d">Kobayashi</span> <span class="RefGivenName" id="3a0333c0-f178-4a7c-8f1b-44110788b6ae">Y</span></span>, <span class="RefAuthor" id="b33dd389-eaa4-4c15-8828-391a7f154a7b"><span class="RefSurName" id="05456fae-c9a9-4906-a7d1-e0f287626c84">Nagahama</span> <span class="RefGivenName" id="23dcb344-1e8a-4e38-bfa9-594b86e347dd">Y</span></span>, <span class="RefAuthor" id="b3ba70c3-ee5d-4c71-b34f-49786e8fc334"><span class="RefSurName" id="c53b89f3-2b85-4289-acc2-7e33a0b5b390">Nakamura</span> <span class="RefGivenName" id="287fbeb5-e990-49f7-a832-225971c310ed">M.</span></span> <span class="RefArticleTitle" id="05e5c218-7062-4d90-b92a-3a3a23cb58a2">Diversity and plasticity of sex determination and differentiation in fishes</span>. <span class="RefJournalTitle" id="f7d08d2d-38ff-4e79-ad01-cbc1afa0323b">Sex Dev</span>. <span class="RefYear" id="9c6a1407-f151-4880-a0fc-03ff5de41b2f">2013</span>;<span class="RefVolume" id="a3dfba94-b994-4af4-92ee-b45c1418ff3c">7</span>(<span class="RefIssue" id="d86b5ce3-a5db-4af2-b7b2-d2e49332ba96">1–3</span>):<span class="RefFPage" id="5b7c534d-cc26-4196-83f5-3354804fe281">115</span>–<span class="RefLPage" id="3d47d749-7b1b-44c8-be48-52dc8d7af08f">25</span>.</p><p id="R7" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="19124081" data-validated-online="true" title="Validated from PubMed" data-doi="10.1016/j.semcdb.2008.12.003" data-validated-log="true" data-validated="true" seqid="R7" data-id="R7" data-valid-id="R7" new-id="R7"><span class="RefSlNo" id="dedbc0ce-ca82-4d1c-b851-e060638f46f5">7.</span> <span class="RefAuthor" id="4c1c1c66-8e1c-40b7-bb94-66383f7368b9"><span class="RefSurName">Godwin</span> <span class="RefGivenName">J</span></span>. <span class="RefYear" id="5813d434-f14e-4515-8726-97db0799ea34">2009</span> <span class="RefArticleTitle" id="23724d10-43ba-4b6d-b258-28a4bedff65b">Social determination of sex in reef fishes</span>. <span class="RefJournalTitle" id="9fc2cee5-a889-4014-bb60-d6ac9a6d8da4"><i>Semin Cell Dev Biol</i></span>. <span class="RefVolume" id="8000ae36-4cd9-4452-8736-761b06bf88c8"><b>20</b></span>, <span class="RefFPage" id="1ca80385-08d2-441a-87b3-efc94f3b7deb">264</span>–<span class="RefLPage" id="20e77dcd-4c80-4246-afe4-2f570ee590e9">270</span>. (<span class="RefDOI" id="36f60ae5-8371-4518-a981-b2b6fcbaa963">doi: 10.1016/j.semcdb.2008.12.003</span>)</p><p id="R8" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1023/A:1007389519702" data-validated-log="true" data-validated="true" seqid="R8" data-id="R8" data-valid-id="R8" new-id="R8"><span class="RefSlNo" id="55861984-e6b1-4df3-b9a3-bfeef2b3f382">8.</span> <span class="RefAuthor" id="5a5c1987-5702-4106-8ea0-8c50fc0f1793"><span class="RefSurName">Kuwamura</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="4e99c069-08cb-4bc4-bf9a-a0be2e2d8fda"><span class="RefSurName">Nakashima</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear" id="70294e66-20be-4244-b13f-a5719727d5a0">1998</span> <span class="RefArticleTitle" id="793fb356-4ca3-4fdf-a8eb-d5fbce7b819f">New aspects of sex change among reef fishes: recent studies in Japan</span>. <span class="RefJournalTitle" id="f3758bec-969f-4e6c-8a56-2d79f9a52515"><i>Environmental Biology of Fishes</i></span>. <span class="RefVolume" id="2d65138e-7fe9-4119-b948-dba479fa419c"><b>52</b></span>, <span class="RefFPage" id="a88df89c-180c-4414-92fc-82d3bc287fe1">125</span>–<span class="RefLPage" id="c9e4c162-d18a-4ba2-83d8-33c96fae6c08">135</span>. (<span class="RefDOI" id="1256058b-b7e4-44b7-9d32-cde6e7d5a49f">doi: 10.1023/A:1007389519702</span>)</p><p id="R9" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1007/PL00012232" data-validated-log="true" data-validated="true" seqid="R9" data-id="R9" data-valid-id="R9" new-id="R9"><span class="RefSlNo" id="cb613812-f194-4951-b586-badc0d2e11c4">9.</span> <span class="RefAuthor" id="669d9855-8dd9-4a95-8702-06820743f117"><span class="RefSurName">Reavis</span> <span class="RefGivenName">RH</span></span>, <span class="RefAuthor" id="a3414b5e-0e04-49d7-902f-6ed0099d0138"><span class="RefSurName">Grober</span> <span class="RefGivenName">MS</span></span>. <span class="RefYear" id="b6b41e4f-f474-419b-9490-48bb4104ce94">1999</span> <span class="RefArticleTitle" id="a38ea845-8d3a-44f6-bef9-36c6bc907a08">An integrative approach to sex change: social, behavioural and neurochemical changes in Lythrypnus dalli (Pisces)</span>. <span class="RefJournalTitle" id="788cf089-36a5-45b4-b3ef-61206ae70abf"><i>Acta Ethol</i></span>. <span class="RefVolume" id="5116918b-7a3f-493c-940e-93d5808e8696"><b>2</b></span>, <span class="RefFPage" id="7fe629c4-85b9-4d1f-94ff-fe80303f9e5a">51</span>–<span class="RefLPage" id="4d968b34-710a-4161-8855-c3347ee2a300">60</span>. (<span class="RefDOI" id="ba6a66d7-a5ba-47a5-b5d8-7f7edad91f4c">doi: 10.1007/PL00012232</span>)</p><p id="R10" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="20176046" data-validated-online="true" title="Validated from PubMed" data-doi="10.1016/j.yfrne.2010.02.002" data-validated-log="true" data-validated="true" seqid="R10" data-id="R10" data-valid-id="R10" new-id="R10"><span class="RefSlNo" id="843ce561-8494-4481-9a95-ed7004907396">10.</span> <span class="RefAuthor" id="abb2d7e0-4752-41a7-b4da-d973e44d2260"><span class="RefSurName">Godwin</span> <span class="RefGivenName">J</span></span>. <span class="RefYear" id="59bf470f-b849-4f93-9a38-51f8c1f38607">2010</span> <span class="RefArticleTitle" id="352c483d-53c8-44b2-9dc2-8fcb261093b7">Neuroendocrinology of sexual plasticity in teleost fishes</span>. <span class="RefJournalTitle" id="330a575a-3dab-4d1a-8618-5ea1d2cc33c4"><i>Front Neuroendocrinol</i></span>. <span class="RefVolume" id="904ef4b2-bc31-4d4d-8761-686e18427674"><b>31</b></span>, <span class="RefFPage" id="6fbfd828-db90-4f4c-a544-a9fd7353a66b">203</span>–<span class="RefLPage" id="c3cb6051-f207-4b7a-91d4-f108a41528a2">216</span>. (<span class="RefDOI" id="17d098df-6d8a-479d-818f-f965f50f6476">doi: 10.1016/j.yfrne.2010.02.002</span>)<span class="RefWebSite" id="848c063d-469c-4f6f-97cf-71c785acf09d">http://dx.doi.org/10.1016/j.yfrne.2010.02.002</span></p><p id="R11" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="25980565" data-validated-online="true" title="Validated from PubMed" data-doi="10.1093/icb/icv041" data-validated-log="true" data-validated="true" seqid="R11" data-id="R11" data-valid-id="R11" new-id="R11"><span class="RefSlNo" id="efe2902c-7435-4fb6-b801-1ecfd46d72a4">11.</span> <span class="RefAuthor" id="8bfe8c10-33bd-4e40-9a42-f80014d1b481"><span class="RefSurName">Lamm</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor" id="11500d80-0557-4db7-b835-07445a2529cf"><span class="RefSurName">Liu</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="73b13077-866e-4b4f-b72f-93c6a105b58f"><span class="RefSurName">Gemmell</span> <span class="RefGivenName">NJ</span></span>, <span class="RefAuthor" id="8099b102-6df9-4740-9e6b-453ee3757eed"><span class="RefSurName">Godwin</span> <span class="RefGivenName">JR</span></span>. <span class="RefYear" id="aca773c9-51f4-48cd-8caa-542e6b27472a">2015</span> <span class="RefArticleTitle" id="4aaee7db-663c-4387-ad24-4b0088462417">The Need for Speed: Neuroendocrine Regulation of Socially-controlled Sex Change</span>. <span class="RefJournalTitle" id="a160d030-ebb0-400a-bfa5-45f5d0add77d"><i>Integr Comp Biol</i></span>. <span class="RefVolume" id="16286715-e5fe-4b18-8b41-0c30b94d28c9"><b>55</b></span>, <span class="RefFPage" id="5a96077d-365d-404f-a744-e19be80bf54a">307</span>–<span class="RefLPage" id="0defd4e7-59f8-4fbf-9610-9c66ccdd9281">322</span>. (<span class="RefDOI" id="d67d6c93-0ffa-42d8-8ceb-15c49a58eb6e">doi: 10.1093/icb/icv041</span>)</p><p id="R12" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1111/j.1439-0310.1993.tb00450.x" data-validated-log="true" data-validated="true" seqid="R12" data-id="R12" data-valid-id="R12" new-id="R12"><span class="RefSlNo" id="85612db9-cb3e-41b7-a673-bcf0f612cc1f">12.</span> <span class="RefAuthor" id="5b97a19d-2a73-4824-851c-7619f6c29f21"><span class="RefSurName">Sunobe</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="9d88d65b-bc3e-4231-a903-9b2f8cd88308"><span class="RefSurName">Nakazono</span> <span class="RefGivenName">A</span></span>. <span class="RefYear" id="61564d86-7f6c-40e7-be83-8b78b9112f8a">1993</span>         <span class="RefArticleTitle" id="88ce5bf1-e1c4-4190-b480-27b77dc39da7">Sex Change in Both Directions by Alteration of Social Dominance in        <i>Trimma okinawae</i>        (Pisces: Gobiidae)</span>       . <span class="RefJournalTitle" id="83ea2e86-0d65-4c51-8efd-1ce13d9d6cf5"><i>Ethology</i></span>. <span class="RefVolume" id="15431ac2-1243-4509-9e83-67f1c818a448"><b>94</b></span>, <span class="RefFPage" id="f60137c2-46fe-4621-bcc5-1f7732a54daf">339</span>–<span class="RefLPage" id="28514bb4-0d05-4709-8e8d-193e518431d2">345</span>. (<span class="RefDOI" id="32d7a6fc-428d-4bc2-a4ea-af74c2a57fb5">doi: 10.1111/j.1439-0310.1993.tb00450.x</span>)<span class="RefWebSite" id="965d3a7b-d18d-4558-b4fa-dcdf95491651">https://onlinelibrary.wiley.com/toc/14390310/94/4</span></p><p id="R13" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1111/j.1095-8649.2007.01338.x" data-validated-log="true" data-validated="true" seqid="R13" data-id="R13" data-valid-id="R13" new-id="R13"><span class="RefSlNo" id="9b35a62c-6f08-4295-b483-57fd85ef9c5e">13.</span> <span class="RefAuthor" id="77d32db1-9f7d-44da-ae7c-37d56cb36f92"><span class="RefSurName">Manabe</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="ccdb0645-a3ee-4fa8-a7b6-09185179de78"><span class="RefSurName">Ishimura</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="e5b6be09-90a4-440f-96fa-719d57907096"><span class="RefSurName">Shinomiya</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor" id="20e03867-1914-469f-99e0-82933b7692a2"><span class="RefSurName">Sunobe</span> <span class="RefGivenName">T</span></span>. <span class="RefYear" id="c03485f4-1ea4-4b2a-93bd-8c580662bddc">2007</span>         <span class="RefArticleTitle" id="05222d24-3252-4178-9d16-f486a2a488f2">Field evidence for bi‐directional sex change in the polygynous gobiid fish        <i>Trimma okinawae</i></span>       . <span class="RefJournalTitle" id="ecb39f25-eb8c-4dd5-a26b-eb3dcc73c904"><i>Journal of Fish Biology</i></span>. <span class="RefVolume" id="c23323c6-5589-40d1-83c9-341b70e75e72"><b>70</b></span>, <span class="RefFPage" id="492bfcae-a335-4922-9bef-449874b132a4">600</span>–<span class="RefLPage" id="58a1843e-26da-490a-8d47-5a34876b412d">609</span>. (<span class="RefDOI" id="e6fa66d8-c68d-4a92-ade6-193002c21756">doi: 10.1111/j.1095-8649.2007.01338.x</span>)<span class="RefWebSite" id="85d02dea-6f36-4dc1-a60a-f4b360db2fd6">https://onlinelibrary.wiley.com/toc/10958649/70/2</span></p><p id="R14" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="15916910" data-validated-online="true" title="Validated from PubMed" data-doi="10.1016/j.cbpb.2005.03.012" data-validated-log="true" data-validated="true" seqid="R14" data-id="R14" data-valid-id="R14" new-id="R14"><span class="RefSlNo" id="9032556a-434a-4a7e-a4d7-db90d928e772">14.</span> <span class="RefAuthor" id="cdf0aa71-a2b5-4976-964f-3f4936de622a"><span class="RefSurName">Sunobe</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="56b83f53-87f0-4cdd-9d19-de762e486780"><span class="RefSurName">Nakamura</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="0da0ab77-1361-4c6e-9af6-29f456e0b1c9"><span class="RefSurName">Kobayashi</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="2cab6481-7dfd-4242-a4dd-0f108466f6d7"><span class="RefSurName">Kobayashi</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="28d5fd54-44f8-4526-ae97-5879a89908e7"><span class="RefSurName">Nagahama</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear" id="6e3c99ed-7667-4b26-9e93-7e682fa9cb9f">2005</span> <span class="RefArticleTitle" id="017ed46b-4c14-46df-8478-3cd3df5df705">Aromatase immunoreactivity and the role of enzymes in steroid pathways for inducing sex change in the hermaphrodite gobiid fish Trimma okinawae</span>. <span class="RefJournalTitle" id="40e10dfd-ae09-4d61-afc7-fcb094a4216d"><span class="ins cts-1" data-cid="1702876681913" data-time="1702876681913" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:18:01 GMT+0000 (Coordinated Universal Time)" id="a9014a81-fd94-4244-9fb4-de339f42c3c3"><i>Comp Biochem Physiol A Mol Integr Physiol</i></span></span>. <span class="RefVolume" id="84a2b4ed-1c45-4bea-a635-d21f3f89b1f7"><b>141</b></span>, <span class="RefFPage" id="6f163afc-3add-4806-96f7-a3be530e1288">54</span>–<span class="RefLPage" id="f191e707-bcec-4679-8763-6507b2ed0fd6">59</span>. (<span class="RefDOI" id="e331c25a-4cc5-462e-9f1c-606bf5d27796">doi: 10.1016/j.cbpb.2005.03.012</span>)</p><p id="R15" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="22303852" data-validated-online="true" title="Validated from PubMed" data-doi="10.2108/zsj.29.121" data-validated-log="true" data-validated="true" seqid="R15" data-id="R15" data-valid-id="R15" new-id="R15"><span class="RefSlNo" id="d3a52c1d-b203-4ce7-99a8-1a1e6d8bb829">15.</span> <span class="RefAuthor" id="058139c5-01e4-4d8a-a178-bfc44fc3a9ea"><span class="RefSurName">Kobayashi</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="0ef28217-1359-490d-aa52-8c67cb5224eb"><span class="RefSurName">Usami</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="b2dbc77d-ec3a-4e26-aedd-953ddf838ff8"><span class="RefSurName">Sunobe</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="6a37e664-ba3e-4786-aea1-3c6892a967b8"><span class="RefSurName">Manabe</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="24e632d3-cdd7-418e-a0ce-11bd29dcc2bb"><span class="RefSurName">Nagahama</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="b01ba134-77a7-4956-bd66-9d3a308fa8fa"><span class="RefSurName">Nakamura</span> <span class="RefGivenName">M</span></span>. <span class="RefYear" id="8f46dc5a-2ac0-4829-a5b0-8db54d2e742e">2012</span> <span class="RefArticleTitle" id="bf2f01cf-d43f-47b0-83e1-721bd2706d0e">Histological observation of the urogenital papillae in the bi-directional sex-changing gobiid fish, Trimma okinawae</span>. <span class="RefJournalTitle" id="d5e6459c-2388-4516-8baa-5b1a30a2db0d"><span class="ins cts-1" data-cid="1702876681959" data-time="1702876681959" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:18:01 GMT+0000 (Coordinated Universal Time)" id="bb7ad156-ebb6-4cd5-b30a-2e8dd0394235"><i>Zoolog Sci</i></span></span>. <span class="RefVolume" id="e2796235-0ed9-4630-9921-5233af130732"><b>29</b></span>, <span class="RefFPage" id="cfbdd88b-9581-4698-9a0c-827e6a636d25">121</span>–<span class="RefLPage" id="2087c631-c3e6-446d-a1bf-f947efd6182d">126</span>. (<span class="RefDOI" id="9ac507c9-e9c6-43fe-b1f2-c3c8dfc66d5e">doi: 10.2108/zsj.29.121</span>)</p><p class="jrnlRefText" seqid="R16" data-id="R16" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="webpage" data-class-name="RefText" id="R16" data-valid-id="R16" new-id="R16"><span class="RefSlNo">16.</span> <span class="RefAuthor" id="2b967949-66bf-4081-ae27-b45cf15989ea"><span class="RefSurName" id="0f7fe460-6d9f-497b-9693-99c6a3534635">Andrews</span> <span class="RefGivenName" id="4881e91e-ab7f-4cce-8bdd-1e485268b8bb">S.</span></span> <span class="RefArticleTitle" id="03bc2c44-732f-40fc-841a-aa80ab4dca86">FastQC: A quality control tool for high throughput sequence data</span>. <span class="RefFPage" id="a9a14b04-4f85-4ab1-829e-d6de88054e20">2010;9–12.</span> <span class="RefWebSite" id="67c6bbd8-35b3-4536-8d6e-bf24ca4b3e41">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></p><p id="R17" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="24695404" data-validated-online="true" title="Validated from PubMed" data-doi="10.1093/bioinformatics/btu170" data-validated-log="true" data-validated="true" seqid="R17" data-id="R17" data-valid-id="R17" new-id="R17"><span class="RefSlNo" id="c93e249a-0026-445c-9785-058fefefdf6a">17.</span> <span class="RefAuthor" id="d3f5ed6a-eddb-49aa-a8a3-103efffffe32"><span class="RefSurName">Bolger</span> <span class="RefGivenName">AM</span></span>, <span class="RefAuthor" id="f6cd4b3b-16ff-40ab-a5b4-25e0e0810952"><span class="RefSurName">Lohse</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="fb9567d2-b37d-4891-aeff-a0e97b149afe"><span class="RefSurName">Usadel</span> <span class="RefGivenName">B</span></span>. <span class="RefYear" id="5a616c59-7763-4a68-9e2f-3dbb74e4d708">2014</span> <span class="RefArticleTitle" id="de4959ff-f7bb-40ee-a337-1c10b464b816">Trimmomatic: A flexible trimmer for Illumina sequence data</span>. <span class="RefJournalTitle" id="76745bd3-bafa-4f44-8134-934197f332f0"><i>Bioinformatics</i></span>. <span class="RefVolume" id="58ee19fd-45ef-4652-8671-fcc296379939"><b>30</b></span>, <span class="RefFPage" id="ce8090a1-337f-4b7a-b4ff-6002f6b5db4f">2114</span>–<span class="RefLPage" id="59d109f7-4372-44da-a836-7e04ec2167fd">2120</span>. (<span class="RefDOI" id="9811580d-8fff-454b-b889-33c2759db457">doi: 10.1093/bioinformatics/btu170</span>)</p><p id="R18" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="23845962" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/nprot.2013.084" data-validated-log="true" data-validated="true" seqid="R18" data-id="R18" data-valid-id="R18" new-id="R18"><span class="RefSlNo" id="300263be-6d41-4632-82a1-703973474d06">18.</span> <span class="RefAuthor" id="23fa8dcb-c3a4-4d8c-8e23-9ea7d59f77a9"><span class="RefSurName">Haas</span> <span class="RefGivenName">BJ</span></span> <span class="RefEtal" id="27ded78e-2652-4327-8f7d-aec7d9c4baa9">et al</span>. <span class="RefYear" id="6b22c687-d8fd-4d86-a742-41cec4fb55a4">2013</span> <span class="RefArticleTitle" id="dacde5bc-04cf-4262-80bc-bff64abe0ae8">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</span>. <span class="RefJournalTitle" id="c8156dd5-0c21-42c3-a882-828d4f43f91b"><i>Nat Protoc</i></span>. <span class="RefVolume" id="09901f22-d412-4011-91fd-0258b5f5106b"><b>8</b></span>, <span class="RefFPage" id="1d1a9132-c2fe-4e7f-ae65-eb9694491ddc">1494</span>–<span class="RefLPage" id="c8c7938f-9bf2-4d57-b034-9b65c274c72c">1512</span>. (<span class="RefDOI" id="1db8bda6-3d22-4e4e-bcaa-e1ebffa5dcfb">doi: 10.1038/nprot.2013.084</span>)</p><p id="R19" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="23060610" data-validated-online="true" title="Validated from PubMed" data-doi="10.1093/bioinformatics/bts565" data-validated-log="true" data-validated="true" seqid="R19" data-id="R19" data-valid-id="R19" new-id="R19"><span class="RefSlNo" id="48580819-4fff-4641-8762-e69b175299f1">19.</span> <span class="RefAuthor" id="b1721d9c-08e3-421f-aab6-cdd5bc22ef96"><span class="RefSurName">Fu</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor" id="4485f085-a592-452a-9d7a-57ba3a806561"><span class="RefSurName">Niu</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor" id="d4b98dd2-e43a-462e-aeb7-85f034ae0a90"><span class="RefSurName">Zhu</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor" id="fa03eec9-6f25-4655-bffa-e521762be897"><span class="RefSurName">Wu</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor" id="7f18b025-126d-48a6-8c27-a71be1c86c20"><span class="RefSurName">Li</span> <span class="RefGivenName">W</span></span>. <span class="RefYear" id="c7ca72c3-7bf3-4c7c-8f16-691819cb0979">2012</span> <span class="RefArticleTitle" id="ec5a21ee-15c9-48f8-a058-7f453de0e30a">CD-HIT: accelerated for clustering the next-generation sequencing data</span>. <span class="RefJournalTitle" id="642de02e-13c8-47e1-83a7-15c639f9c9d9"><i>Bioinformatics</i></span>. <span class="RefVolume" id="5a783b90-889b-4032-b761-218e985e23d0"><b>28</b></span>, <span class="RefFPage" id="1dc2e8e3-beab-450d-8914-0442f2ea8acb">3150</span>–<span class="RefLPage" id="4371ab49-5247-430f-a52c-eb50c5e734b8">3152</span>. (<span class="RefDOI" id="ffce4faf-8823-413d-a47f-2ef8e22d422e">doi: 10.1093/bioinformatics/bts565</span>)</p><p id="R20" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="29220515" data-validated-online="true" title="Validated from PubMed" data-doi="10.1093/molbev/msx319" data-validated-log="true" data-validated="true" seqid="R20" data-id="R20" data-valid-id="R20" new-id="R20"><span class="RefSlNo" id="efd5a064-e01e-4923-bb72-195d401a9a02">20.</span> <span class="RefAuthor" id="45f3140b-5506-4015-ae6a-a2f5057fefe1"><span class="RefSurName">Waterhouse</span> <span class="RefGivenName">RM</span></span>, <span class="RefAuthor" id="6fa0f6df-07ab-4f5b-a6d2-3df89fdb8186"><span class="RefSurName">Seppey</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="85b96a43-536c-4f8b-862c-f067e69c647b"><span class="RefSurName">Simão</span> <span class="RefGivenName">FA</span></span>, <span class="RefAuthor" id="2df7a563-531e-4f31-ab0a-b8e65abd2d07"><span class="RefSurName">Manni</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="85816c0f-5caf-433f-967a-2567bf52db41"><span class="RefSurName">Ioannidis</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor" id="4ee01884-4761-4a04-8976-75cd7d652719"><span class="RefSurName">Klioutchnikov</span> <span class="RefGivenName">G</span></span>, <span class="RefAuthor" id="18792483-7408-4775-b5ea-67178e414fff"><span class="RefSurName">Kriventseva</span> <span class="RefGivenName">EV</span></span>, <span class="RefAuthor" id="618d18f9-b659-4250-8372-84041d6f1c3d"><span class="RefSurName">Zdobnov</span> <span class="RefGivenName">EM</span></span>. <span class="RefYear" id="e742e240-d4b7-4cda-bc0d-055889665e8d">2018</span> <span class="RefArticleTitle" id="e767e1f5-16c5-468a-a4e5-8f1d817aa548">BUSCO Applications from Quality Assessments to Gene Prediction and Phylogenomics</span>. <span class="RefJournalTitle" id="388ad59c-1bc9-4f95-b999-92b6c18eef5a"><i>Mol Biol Evol</i></span>. <span class="RefVolume" id="1da5e346-a8c3-42bd-a005-7508d02009e5"><b>35</b></span>, <span class="RefFPage" id="34bab0cc-3d0c-47aa-8e64-3a6483082cf0">543</span>–<span class="RefLPage" id="d5f48d9d-e3c4-4504-9a6f-a632ea76a4a3">548</span>. (<span class="RefDOI" id="a90eb344-6135-4d12-88b1-ad51518d7afb">doi: 10.1093/molbev/msx319</span>)</p><p id="R21" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="26059717" data-validated-online="true" title="Validated from PubMed" data-doi="10.1093/bioinformatics/btv351" data-validated-log="true" data-validated="true" seqid="R21" data-id="R21" data-valid-id="R21" new-id="R21"><span class="RefSlNo" id="16576b16-171f-49e5-abc6-7767d544ff01">21.</span> <span class="RefAuthor" id="8fd100a8-d624-460e-b8c0-d841b14dee8f"><span class="RefSurName">Simão</span> <span class="RefGivenName">FA</span></span>, <span class="RefAuthor" id="c5628e70-1c05-4e56-97ae-c63260cc442c"><span class="RefSurName">Waterhouse</span> <span class="RefGivenName">RM</span></span>, <span class="RefAuthor" id="7e38da1e-952f-4c1d-9427-6723bd2bc641"><span class="RefSurName">Ioannidis</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor" id="cf9c8a9c-ff5e-4d7f-948e-dd44bbc00bf6"><span class="RefSurName">Kriventseva</span> <span class="RefGivenName">EV</span></span>, <span class="RefAuthor" id="3d9c58d8-f3ea-472d-abde-345ffbdb7808"><span class="RefSurName">Zdobnov</span> <span class="RefGivenName">EM</span></span>. <span class="RefYear" id="39c8d298-8b84-48e3-9821-50af394b1b2c">2015</span> <span class="RefArticleTitle" id="e42eef69-482f-4b69-8fd1-aa70a22bd00d">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</span>. <span class="RefJournalTitle" id="91197f50-11f6-43c1-b11d-4fb9e3829a7d"><i>Bioinformatics</i></span>. <span class="RefVolume" id="d3bf3293-969f-407b-8b2b-6c1a4642903a"><b>31</b></span>, <span class="RefFPage" id="0a9fb4e3-d1df-48f2-a55e-1e1258a25370">3210</span>–<span class="RefLPage" id="0cfa917b-305a-4b99-a864-7ebd6dbd3703">3212</span>. (<span class="RefDOI" id="94d51f7d-d8e4-4c8f-b641-391f5fe268c1">doi: 10.1093/bioinformatics/btv351</span>)</p><p class="jrnlRefText" seqid="R22" data-id="R22" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-class-name="RefText" id="R22" data-valid-id="R22" new-id="R22"><span class="RefSlNo">22.</span> <span class="RefAuthor" id="19e40d37-9c56-4c76-9f47-d8b4a17fc8dc"><span class="RefSurName" id="c466486f-da10-4d47-8329-e536ab27dbf9">Cantalapiedra</span> <span class="RefGivenName" id="855e2725-89c9-4a01-aec9-d0b0e8265ae2">CP</span></span>, <span class="RefAuthor" id="f729e5cc-ce58-412e-bc75-24892cc81063"><span class="RefSurName" id="1ab2f92a-7430-41ad-b1fe-cac76567c8f2">Hern̗andez-Plaza</span> <span class="RefGivenName" id="68a3ceea-d818-4d82-9652-c3d44c68c2ef">A</span></span>, <span class="RefAuthor" id="97e38f33-cbed-4261-b4e6-08e56a9c6a64"><span class="RefSurName" id="cf693240-08d5-4ec9-9b91-dca3c22d2aa2">Letunic</span> <span class="RefGivenName" id="2c6e1fbe-d63e-4c9c-a6f2-82a9050522f8">I</span></span>, <span class="RefAuthor" id="80d66c2d-870d-41bc-965e-65eed0fdd831"><span class="RefSurName" id="55944cfc-95a4-4441-846e-b02fa8ac3ed1">Bork</span> <span class="RefGivenName" id="801f62dd-4324-4380-8add-5bf692434279">P</span></span>, <span class="RefAuthor" id="cbe1d58d-f7c7-4c75-9329-8b521cff0408"><span class="RefSurName" id="35ee3446-fb3d-434d-8620-8f41d5cf6df0">Huerta-Cepas</span> <span class="RefGivenName" id="01c65da8-3cfd-480d-8b10-c3275194c0d0">J.</span></span> <span class="RefArticleTitle" id="b7f03248-7261-4a11-8816-3a94c15b93ac">eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale</span>. <span class="RefJournalTitle" id="48fd5be6-f3f0-44be-b13e-f7c485fd33ed">Mol Biol Evol</span>. <span class="RefYear" id="2ef75e43-ec59-4ae8-ba50-ac3640685118">2021</span>;<span class="RefVolume" id="99de9707-35c6-46e4-abd7-eeb9858f9c74">38</span>(<span class="RefIssue" id="0658157b-5c09-4f14-bd68-fdfa4612bd38">12</span>):<span class="RefFPage" id="7ad60600-d64e-4b51-be31-63990b5281d3">5825</span>–<span class="RefLPage" id="f42da23b-8032-4e73-8054-53ea8a3fc32f">9</span>.</p><p class="jrnlRefText" seqid="R23" data-id="R23" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-class-name="RefText" id="R23" data-valid-id="R23" new-id="R23"><span class="RefSlNo">23.</span> <span class="RefAuthor" id="1a508d10-b278-44db-a257-70060721d46e"><span class="RefSurName" id="04b5ce13-b3ed-43c8-b3bc-d7b0fbc2b0ca">Li</span> <span class="RefGivenName" id="28d88916-a396-497e-978b-a1e84bf59430">B</span></span>, <span class="RefAuthor" id="5fdfbfdf-f7d2-4e26-abd7-011c367f5d31"><span class="RefSurName" id="33d053b6-a57e-44c3-bbbf-28ac13b0e690">Dewey</span> <span class="RefGivenName" id="1bdcd334-1e50-4db2-8d6b-411d91416421">CN.</span></span> <span class="RefArticleTitle" id="e050fa70-1909-4396-97d4-393d308052cb">RSEM: Accurate transcript quantification from RNA-Seq data with or without a reference genome</span>. <span class="RefJournalTitle" id="fec562d0-b76f-45eb-a84b-c54adc52b1ab">BMC Bioinformatics</span>. <span class="RefYear" id="a7c3679c-8b8f-456d-8ccc-1a3393f82975">2011</span>;<span class="RefVolume" id="c7c38662-3d36-41eb-a94f-f3828bd6e93b">12</span>:<span class="RefFPage" id="3093188d-67f6-4545-8e7b-785a3621f044">41</span>–<span class="RefLPage" id="be9fcc56-cc39-4873-9edc-e28fb754a5fc">74</span>.</p><p class="jrnlRefText" seqid="R24" data-id="R24" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-class-name="RefText" id="R24" data-valid-id="R24" new-id="R24"><span class="RefSlNo">24.</span> <span class="RefAuthor" id="c36241c0-37f0-4c5f-aec6-7c0e2e77244f"><span class="RefSurName" id="cad3d868-215f-4705-893c-9777df8b6f3d">Langmead</span> <span class="RefGivenName" id="ec15fe93-b4a2-450c-ab86-e834d9072e9e">B</span></span>, <span class="RefAuthor" id="ebfb86f1-f0a0-4afb-90ce-1c8ce62e32f3"><span class="RefSurName" id="dff72398-e0ac-45b5-86aa-60f53f26b300">Salzberg</span> <span class="RefGivenName" id="135841f6-d16d-4769-ae47-dae2f5f78e86">SL.</span></span> <span class="RefArticleTitle" id="4625b039-1007-4821-b20e-7c7efce80ae4">Fast gapped-read alignment with Bowtie 2</span>. <span class="RefJournalTitle" id="d84cc3b3-7a53-4317-b3f8-86e814ab7295">Nat Methods</span>. <span class="RefYear" id="a38500c2-63a5-476d-b5e7-b5401a49417d">2012</span>;<span class="RefVolume" id="9da84a73-3e54-4a6b-8838-0e633d352483">9</span>(<span class="RefIssue" id="b0cb430d-0e0c-46cc-9e73-1c93008c2e36">4</span>):<span class="RefFPage" id="8cdfd435-9270-4d58-ba4c-c9b58a121ee0">357</span>–<span class="RefLPage" id="b8bfb877-a039-4a77-bfad-861305ce959c">9</span>.</p><p id="R25" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="25516281" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/s13059-014-0550-8" data-validated-log="true" data-validated="true" seqid="R25" data-id="R25" data-valid-id="R25" new-id="R25"><span class="RefSlNo" id="392081cd-471b-47df-b739-5f37397470ca">25.</span> <span class="RefAuthor" id="64444fa7-0380-46fe-aa39-34cc35bc4550"><span class="RefSurName">Love</span> <span class="RefGivenName">MI</span></span>, <span class="RefAuthor" id="a5dddb0c-b2cb-461b-ba23-ff333a027c15"><span class="RefSurName">Huber</span> <span class="RefGivenName">W</span></span>, <span class="RefAuthor" id="b0eb2ab5-1f32-401c-b78b-83574330bae6"><span class="RefSurName">Anders</span> <span class="RefGivenName">S</span></span>. <span class="RefYear" id="dcd954ed-9491-4cb5-940b-a5ebcb789a5c">2014</span> <span class="RefArticleTitle" id="4b1a5b2d-252d-4064-944b-92c1e55e8781">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <span class="RefJournalTitle" id="0c6730f0-fede-4189-b6ab-5192149f00a7"><i>Genome Biol</i></span>. <span class="RefVolume" id="74c2aaf6-d53d-418a-9737-1e65791c0d0f"><b>15</b></span><span class="RefFPage" id="c37eeacf-6c4b-4d99-a404-43f27aff7d4c">550</span>,  (<span class="RefDOI" id="d03240da-34e3-488b-899d-03854941c4c2">doi: 10.1186/s13059-014-0550-8</span>)</p><p id="R26" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="20132535" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/gb-2010-11-2-r14" data-validated-log="true" data-validated="true" seqid="R26" data-id="R26" data-valid-id="R26" new-id="R26"><span class="RefSlNo" id="b8fe35c2-a8ff-4887-ae9c-222c7c78e087">26.</span> <span class="RefAuthor" id="9081a102-c95f-4b60-923b-b221d66944b1"><span class="RefSurName">Young</span> <span class="RefGivenName">MD</span></span>, <span class="RefAuthor" id="e471d7fc-ba60-4a73-bf8e-9f89cd869341"><span class="RefSurName">Wakefield</span> <span class="RefGivenName">MJ</span></span>, <span class="RefAuthor" id="bf64e3f7-adf2-4dfb-bc17-479327cbbf91"><span class="RefSurName">Smyth</span> <span class="RefGivenName">GK</span></span>, <span class="RefAuthor" id="3816e024-7026-443d-88a3-21b2a4506a5f"><span class="RefSurName">Oshlack</span> <span class="RefGivenName">A</span></span>. <span class="RefYear" id="5b27dbbb-990c-4ee6-a6c3-b40f55b608f2">2010</span> <span class="RefArticleTitle" id="300fa3e8-728d-479b-bd28-f31a10cae3d6">Gene ontology analysis for RNA-seq: accounting for selection bias</span>. <span class="RefJournalTitle" id="3c492b97-5ba6-4272-b2b1-6cbfde34f04d"><i>Genome Biol</i></span>. <span class="RefVolume" id="13c35969-e9d9-4219-b740-a38befa3f1df"><b>11</b></span>, <span class="RefELocation" id="a780c447-6788-4d56-a9b1-84f029fc76de"><span class="ins cts-1" data-cid="1702876681556" data-time="1702876681556" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:18:01 GMT+0000 (Coordinated Universal Time)" id="cf55cc08-d228-40c5-8a4d-78a7ff3a4f23">R14</span></span>. (<span class="RefDOI" id="ef7ba9ae-396b-4631-adc0-e5844ab7105f">doi: 10.1186/gb-2010-11-2-r14</span>)</p><p id="R27" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1111/j.2517-6161.1995.tb02031.x" data-validated-log="true" data-validated="true" seqid="R27" data-id="R27" data-valid-id="R27" new-id="R27"><span class="RefSlNo" id="b5b6a6a3-2ef4-4521-a9e2-4d21f9da3fec">27.</span> <span class="RefAuthor" id="159f9177-746c-4928-a3b1-a97a77bbf179"><span class="RefSurName">Benjamini</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="6f56f031-3f25-4f07-a771-b2a8d0809d0b"><span class="RefSurName">Hochberg</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear" id="41d4014c-63e9-4611-8265-2ea2c3a64c4e">1995</span> <span class="RefArticleTitle" id="67e59127-b5d4-4ce7-9fc3-42b024eac3eb">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>. <span class="RefJournalTitle" id="078229e3-8d0d-41df-927b-02df7e7a4b54"><span class="ins cts-1" data-cid="1702876681610" data-time="1702876681610" data-userid="CrossRef" data-username="CrossRef" title="Inserted by CrossRef - Mon Dec 18 2023 05:18:01 GMT+0000 (Coordinated Universal Time)" id="3d1481ac-4987-4e31-9bcc-5ae2492eb5a9"><i>Journal of the Royal Statistical Society: Series B (Methodological</i>)</span></span>. <span class="RefVolume" id="a6cc5a35-966d-4cc3-a526-2182f97e1905"><b>57</b></span>, <span class="RefFPage" id="a3e6de6f-224b-49ce-b394-2326f173584c">289</span>–<span class="RefLPage" id="af439ded-a918-4826-8da6-a27a0b8b65ee">300</span>. (<span class="RefDOI" id="56d8a80b-3110-463d-b097-4ac85139e463">doi: 10.1111/j.2517-6161.1995.tb02031.x</span>)<span class="RefWebSite" id="c111fa88-43bd-4487-ad0c-b5d2d7a7ac1a">https://rss.onlinelibrary.wiley.com/toc/25176161/57/1</span></p><p id="R28" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="27748421" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/srep35461" data-validated-log="true" data-validated="true" seqid="R28" data-id="R28" data-valid-id="R28" new-id="R28"><span class="RefSlNo" id="34e18099-aacd-4cae-90ff-65cf545e0859">28.</span> <span class="RefAuthor" id="5521a485-f8a2-47fc-b412-1cd5d173d091"><span class="RefSurName">Casas</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor" id="fa25050b-8443-456f-97c6-d5af6d2ef8ca"><span class="RefSurName">Saborido-Rey</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor" id="bd6ead40-905c-4408-bf8d-4d43ea17a8d0"><span class="RefSurName">Ryu</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="9434fda8-6231-40e7-b252-9bd24d5e89ff"><span class="RefSurName">Michell</span> <span class="RefGivenName">C</span></span>, <span class="RefAuthor" id="7961c189-2ac8-497a-b6ee-897a4ae162f6"><span class="RefSurName">Ravasi</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="74e6fc7f-4e37-4bce-a63d-ca7cb6f06226"><span class="RefSurName">Irigoien</span> <span class="RefGivenName">X</span></span>. <span class="RefYear" id="079462e3-4282-41c5-ab83-eb1a236ce773">2016</span> <span class="RefArticleTitle" id="0b8f64fd-f7a3-4d23-8e2d-032ca9d18bee">Sex Change in Clownfish: Molecular Insights from Transcriptome Analysis</span>. <span class="RefJournalTitle" id="ec494d13-a0c7-4d4a-bc40-b12a82d544be"><i>Sci Rep</i></span>. <span class="RefVolume" id="f90a7947-f2af-4749-98ca-4ae3ca5eef91"><b>6</b></span><span class="RefFPage" id="c0c314b2-c3b5-4bd7-b898-a73403b3a476">35461</span>,  (<span class="RefDOI" id="35c92b09-0082-400e-b530-864ce2e2f5f4">doi: 10.1038/srep35461</span>)</p><p id="R29" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1093/molbev/msx293" data-validated-log="true" data-validated="true" seqid="R29" data-id="R29" data-valid-id="R29" new-id="R29"><span class="RefSlNo" id="c1e85e0a-8c2d-465c-92c4-1280c9f8c192">29.</span> <span class="RefAuthor" id="1343e662-21f9-41c1-809e-0d193c2dec65"><span class="RefSurName">Todd</span> <span class="RefGivenName">EV</span></span>, <span class="RefAuthor" id="6b038289-d89c-434f-8aa4-81e7333cac5f"><span class="RefSurName">Liu</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="76dc0406-f102-4717-b848-3e203f55b035"><span class="RefSurName">Lamm</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor" id="605f3669-8353-4e1e-93b0-cb8c3d907999"><span class="RefSurName">Thomas</span> <span class="RefGivenName">JT</span></span>, <span class="RefAuthor" id="34a8c803-5561-43b8-b1c2-d2131f4b6246"><span class="RefSurName">Rutherford</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor" id="499b892b-310a-4bc7-b999-6aac1049f0c1"><span class="RefSurName">Thompson</span> <span class="RefGivenName">KC</span></span>, <span class="RefAuthor" id="faffb2c1-d5f9-4ff0-aabc-a0f07cda8b04"><span class="RefSurName">Godwin</span> <span class="RefGivenName">JR</span></span>, <span class="RefAuthor" id="a6663b84-1230-4660-a3c4-03c488494419"><span class="RefSurName">Gemmell</span> <span class="RefGivenName">NJ</span></span>. <span class="RefYear" id="5e3dbeed-954b-4213-868c-49178bdce673">2018</span> <span class="RefArticleTitle" id="e29e7d56-69f8-4d06-8601-6159dcbea2c6">Female Mimicry by Sneaker Males Has a Transcriptomic Signature in Both the Brain and the Gonad in a Sex-Changing Fish</span>. <span class="RefJournalTitle" id="7c13c1de-65d2-452c-8116-47d863fcd911"><i>Mol Biol Evol</i></span>. <span class="RefVolume" id="efa26f57-d30d-4a44-a7c8-cc7d5f12bf6d"><b>35</b></span>, <span class="RefFPage" id="25ad7b59-c301-4573-b4e8-7b6b35b8d12d">225</span>–<span class="RefLPage" id="82ca7963-fdb5-48a8-8dee-5e0bf69e39d1">241</span>. (<span class="RefDOI" id="b9105708-8087-41f8-b9e8-cf005e1a0ba5">doi: 10.1093/molbev/msx293</span>)</p><p id="R30" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1186/s13293-015-0044-8" data-validated-log="true" data-validated="true" seqid="R30" data-id="R30" data-valid-id="R30" new-id="R30"><span class="RefSlNo" id="19a10524-2d51-483e-aec8-b2726d4410b6">30.</span> <span class="RefAuthor" id="786824a5-6f3b-440d-afa9-1029c61209ac"><span class="RefSurName">Liu</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="3e9f43cf-1950-46f0-b434-18b1f4f833cf"><span class="RefSurName">Lamm</span> <span class="RefGivenName">MS</span></span>, <span class="RefAuthor" id="63cbed2b-2ebd-4a00-8663-979e3e1d6293"><span class="RefSurName">Rutherford</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor" id="a99347da-87f0-4a84-b7fe-73d57665bd1a"><span class="RefSurName">Black</span> <span class="RefGivenName">MA</span></span>, <span class="RefAuthor" id="67274a7f-07d3-4694-b82f-56d476e3614b"><span class="RefSurName">Godwin</span> <span class="RefGivenName">JR</span></span>, <span class="RefAuthor" id="573db113-a8a6-457d-869a-e5608aa9a7bb"><span class="RefSurName">Gemmell</span> <span class="RefGivenName">NJ</span></span>. <span class="RefYear" id="9045b9b0-49ed-4fda-a44a-6b7384864631">2015</span> <span class="RefArticleTitle" id="f99ca90c-ec68-4e13-b6eb-13f52cd913bf">Large-scale transcriptome sequencing reveals novel expression patterns for key sex-related genes in a sex-changing fish</span>. <span class="RefJournalTitle" id="45230e7e-8f96-4ba3-9d0a-0518106d1e08"><i>Biol Sex Differ</i></span>. <span class="RefVolume" id="24a7a61e-d4cf-49f4-bb83-c130ab9b2839"><b>6</b></span>, <span class="RefFPage" id="72f0d045-d5a0-40af-89e1-94501f4d8797">1</span>–<span class="RefLPage" id="38853110-8792-4783-bced-f41b0057d838">20</span>. (<span class="RefDOI" id="6fd1eab3-4a99-4ae9-84f7-7d669c51dc67">doi: 10.1186/s13293-015-0044-8</span>)<span class="RefWebSite" id="3f16f34a-4356-4cbf-a183-2e23f8babc67">http://dx.doi.org/10.1186/s13293-015-0044-8</span></p><p id="R31" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="31309157" data-validated-online="true" title="Validated from PubMed" data-doi="10.1126/sciadv.aaw7006" data-validated-log="true" data-validated="true" seqid="R31" data-id="R31" data-valid-id="R31" new-id="R31"><span class="RefSlNo" id="403644e2-46d8-4fe5-afbe-54cfb1c75634">31.</span> <span class="RefAuthor" id="5cc29a95-fb49-46b6-b99a-6e760fdafdcc"><span class="RefSurName">Todd</span> <span class="RefGivenName">EV</span></span> <span class="RefEtal" id="78693cea-362c-4899-ac51-e7adfbf89c81">et al</span>. <span class="RefYear" id="b93c8af0-0c93-4521-af3a-6122f2ca292c">2019</span> <span class="RefArticleTitle" id="01edaf09-fd29-41e2-9f06-8f9c2ccd7024">Stress, novel sex genes, and epigenetic reprogramming orchestrate socially controlled sex change</span>. <span class="RefJournalTitle" id="c3668d11-fd44-4a2d-a569-2699358efc7e"><i>Sci Adv</i></span>. <span class="RefVolume" id="aef56418-fdd1-46f3-9a73-b2a764fcfc51"><b>5</b></span><span class="RefELocation" id="9ea0eb84-8f71-4f94-95fe-bd43c40769d0"><span class="ins cts-1" data-cid="1702876678193" data-time="1702876678193" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:17:58 GMT+0000 (Coordinated Universal Time)" id="4141365a-5988-4e91-9dc7-15ef2c745ff2">eaaw7006</span></span>,  (<span class="RefDOI" id="3ff9bb53-719a-4be4-93a1-ead5ea833d18">doi: 10.1126/sciadv.aaw7006</span>)</p><p id="R32" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="28388875" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/s12864-017-3652-3" data-validated-log="true" data-validated="true" seqid="R32" data-id="R32" data-valid-id="R32" new-id="R32"><span class="RefSlNo" id="7742aa87-f1d1-4043-9781-f4006913bc34">32.</span> <span class="RefAuthor" id="fba9dc69-e059-4675-870c-c4da12faad00"><span class="RefSurName">Cribbin</span> <span class="RefGivenName">KM</span></span>, <span class="RefAuthor" id="4effe779-9ab4-472d-96a5-84832114e85b"><span class="RefSurName">Quackenbush</span> <span class="RefGivenName">CR</span></span>, <span class="RefAuthor" id="4c9131ea-aab9-4029-8f1a-8ebd350e9462"><span class="RefSurName">Taylor</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor" id="452ad364-a710-45ab-84fa-5fead3a0a996"><span class="RefSurName">Arias-Rodriguez</span> <span class="RefGivenName">L</span></span>, <span class="RefAuthor" id="eab046da-f22c-4130-96d5-982dc1c7f9b3"><span class="RefSurName">Kelley</span> <span class="RefGivenName">JL</span></span>. <span class="RefYear" id="c393eedb-2289-4b18-8f2c-d23a86e35055">2017</span> <span class="RefArticleTitle" id="a0cf955d-3f02-4eed-b46b-9d7ffb33d7ed">Sex-specific differences in transcriptome profiles of brain and muscle tissue of the tropical gar</span>. <span class="RefJournalTitle" id="e6d75e53-b49e-48c8-8ed7-1d9f0ca62fdc"><i>BMC Genomics</i></span>. <span class="RefVolume" id="e3753cf1-f0e4-4b77-a75a-80cff57a389a"><b>18</b></span><span class="RefFPage" id="8f6d67ee-4718-4a5d-88e0-753d6a25ff1a">283</span>,  (<span class="RefDOI" id="9edb57ad-ebbc-49ad-a231-20068151895f">doi: 10.1186/s12864-017-3652-3</span>)</p><p id="R33" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="25326170" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/1471-2164-15-905" data-validated-log="true" data-validated="true" seqid="R33" data-id="R33" data-valid-id="R33" new-id="R33"><span class="RefSlNo" id="bc3d956a-3707-4b5b-ab85-ddc8e5013310">33.</span> <span class="RefAuthor" id="e9eaaf75-932d-46e4-89e5-4d1714684d09"><span class="RefSurName">Wong</span> <span class="RefGivenName">RY</span></span>, <span class="RefAuthor" id="bff08a0b-185c-43ed-a3e6-3f1fc6494e5f"><span class="RefSurName">McLeod</span> <span class="RefGivenName">MM</span></span>, <span class="RefAuthor" id="11105ae0-699c-4972-90b0-83f384292e0d"><span class="RefSurName">Godwin</span> <span class="RefGivenName">J</span></span>. <span class="RefYear" id="93703a4e-0066-47b7-a0cc-4b7a5d781062">2014</span> <span class="RefArticleTitle" id="282e5906-e5ff-4d27-9d3d-4a6282304758">Limited sex-biased neural gene expression patterns across strains in Zebrafish (Danio rerio)</span>. <span class="RefJournalTitle" id="fd90db04-22e5-4069-a54f-f72a4c598855"><i>BMC Genomics</i></span>. <span class="RefVolume" id="5d89bcd5-f0bc-44a0-8d8a-27499be716ea"><b>15</b></span><span class="RefFPage" id="2d31bd64-c681-4471-830d-e96cd4319c62">905</span>,  (<span class="RefDOI" id="68ac78d8-8dfd-4740-bc76-411d4c225e15">doi: 10.1186/1471-2164-15-905</span>)</p><p id="R34" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="25099474" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/1471-2164-15-655" data-validated-log="true" data-validated="true" seqid="R34" data-id="R34" data-valid-id="R34" new-id="R34"><span class="RefSlNo" id="46f5e5f0-4d11-46d1-a7f2-6998f1f909d2">34.</span> <span class="RefAuthor" id="9684d736-3134-4c62-9483-c9af80899de1"><span class="RefSurName">Manousaki</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="38e2c906-b6e2-45fd-a57b-76efb7a343f5"><span class="RefSurName">Tsakogiannis</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor" id="88e83c6a-53cf-4114-a7e0-e2546e994b66"><span class="RefSurName">Lagnel</span> <span class="RefGivenName">J</span></span>, <span class="RefAuthor" id="89ae9bc6-288e-42c7-8844-72268e8d7abf"><span class="RefSurName">Sarropoulou</span> <span class="RefGivenName">E</span></span>, <span class="RefAuthor" id="105ef7e6-fe11-4fea-bf6b-6ae576823af3"><span class="RefSurName">Xiang</span> <span class="RefGivenName">JZ</span></span>, <span class="RefAuthor" id="14cffe9e-0f2e-4ed9-bbf3-751a2741140f"><span class="RefSurName">Papandroulakis</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor" id="894a251d-2859-40f7-96a1-53b2fde3bd9d"><span class="RefSurName">Mylonas</span> <span class="RefGivenName">CC</span></span>, <span class="RefAuthor" id="17930e14-21f4-4d43-9eaa-6b12b3a67742"><span class="RefSurName">Tsigenopoulos</span> <span class="RefGivenName">CS</span></span>. <span class="RefYear" id="6f260c02-96eb-4a02-8de2-aa4943bc25a3">2014</span> <span class="RefArticleTitle" id="3b0e1e90-089e-4b24-8a9a-9480c7622ca6">The sex-specific transcriptome of the hermaphrodite sparid sharpsnout seabream (Diplodus puntazzo)</span>. <span class="RefJournalTitle" id="3ff345bd-3eb9-4f99-a4bb-176f60769d24"><i>BMC Genomics</i></span>. <span class="RefVolume" id="17e6cb6b-7537-44bf-991f-c3883cb4f162"><b>15</b></span><span class="RefFPage" id="aca1dc5d-2766-4f86-8f58-bcbcc4d0af98">655</span>,  (<span class="RefDOI" id="be3bc6ac-b1ff-4e67-a9ed-473ca6ff5aca">doi: 10.1186/1471-2164-15-655</span>)</p><p id="R35" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="29891564" data-validated-online="true" title="Validated from PubMed" data-doi="10.1242/dev.161356" data-validated-log="true" data-validated="true" seqid="R35" data-id="R35" data-valid-id="R35" new-id="R35"><span class="RefSlNo" id="72df4551-972b-4fdb-ab39-a6a13d0d9f24">35.</span> <span class="RefAuthor" id="fcc9e1a4-f701-4738-91e6-f2e0975a8d28"><span class="RefSurName">Neuman</span> <span class="RefGivenName">SD</span></span>, <span class="RefAuthor" id="62f02dcb-fd2a-4e50-ba3a-a6e9ec60250d"><span class="RefSurName">Bashirullah</span> <span class="RefGivenName">A</span></span>. <span class="RefYear" id="58ef2903-3e6d-492f-be3f-e49bdeefa177">2018</span> <span class="RefArticleTitle" id="cfe2f91e-37c0-4f45-b6d3-e1d3a15c2ad4">Hobbit regulates intracellular trafficking to drive insulin-dependent growth during <i>Drosophila</i> development</span>. <span class="RefJournalTitle" id="9b107ec6-c897-4121-a5f4-2b0c33b332fb"><i>Development</i></span>. <span class="RefVolume" id="66312edc-1500-424a-ad6b-77ebad849c6f"><span class="ins cts-1" data-cid="1702876678723" data-time="1702876678723" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:17:58 GMT+0000 (Coordinated Universal Time)" id="7ee93849-a691-443a-a4e7-a7e0804dda9e"><b>145</b></span></span><span class="RefFPage" id="2084a6b6-f8c1-46e1-98af-05f229810361">1</span>–<span class="RefLPage" id="15ee6527-d042-4313-88b1-0a22c364ae4b">11</span>, (<span class="RefDOI" id="88cd3ec7-458c-4eed-ab81-267bae87d42a">doi: 10.1242/dev.161356</span>)<span class="RefWebSite" id="37c3c162-cbea-4274-bc6d-bff46fb011d0">https://journals.biologists.com/dev/article/145/11/dev161356/48483/Hobbit-regulates-intracellular-trafficking-to</span></p><p id="R36" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="36461115" data-validated-online="true" title="Validated from PubMed" data-doi="10.1186/s40246-022-00439-3" data-validated-log="true" data-validated="true" seqid="R36" data-id="R36" data-valid-id="R36" new-id="R36"><span class="RefSlNo" id="ef34190e-8071-44ed-bfaf-89f04c31cbc9">36.</span> <span class="RefAuthor" id="6bd528ed-d0cf-499e-a451-5f3cc757b3a1"><span class="RefSurName">Braschi</span> <span class="RefGivenName">B</span></span>, <span class="RefAuthor" id="96e87499-4251-4305-9bcc-60919ea2887c"><span class="RefSurName">Bruford</span> <span class="RefGivenName">EA</span></span>, <span class="RefAuthor" id="17baae01-cd7c-4753-b7b7-d987cbcdd8d6"><span class="RefSurName">Cavanagh</span> <span class="RefGivenName">AT</span></span>, <span class="RefAuthor" id="e8c49377-3a3d-49e4-99de-ae3e74ca6388"><span class="RefSurName">Neuman</span> <span class="RefGivenName">SD</span></span>, <span class="RefAuthor" id="de91fb98-688a-4114-90eb-6c3f4b5cbfdd"><span class="RefSurName">Bashirullah</span> <span class="RefGivenName">A</span></span>. <span class="RefYear" id="c3ec58f7-d2f9-483f-8191-4634f7814687">2022</span> <span class="RefArticleTitle" id="1eb6fdad-49e5-4354-9a6d-bfe74b0f61b3">The bridge-like lipid transfer protein (BLTP) gene group: introducing new nomenclature based on structural homology indicating shared function</span>. <span class="RefJournalTitle" id="4a9abcbc-0a55-4ea2-ad57-65625abeabbd"><i>Hum Genomics</i></span>. <span class="RefVolume" id="37482e71-4110-4701-8851-74f1df5bb7dd"><b>16</b></span><span class="RefFPage" id="6ea150dd-0a0a-4b1f-8584-f0c1c51c6bf9">66</span>,  (<span class="RefDOI" id="68cb9736-d9e6-487b-98c1-56ca41b667fd">doi: 10.1186/s40246-022-00439-3</span>)<span class="RefWebSite" id="d2e894df-4c37-47c4-8dfa-0e4f6d9cb602">https://doi.org/10.1186/s40246-022-00439-3</span></p><p id="R37" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="28801581" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/s41598-017-08248-8" data-validated-log="true" data-validated="true" seqid="R37" data-id="R37" data-valid-id="R37" new-id="R37"><span class="RefSlNo" id="3077fdff-7644-4335-a267-6bc983d6687f">37.</span> <span class="RefAuthor" id="d171cf74-f691-4374-ba3d-a38a0ca4fd28"><span class="RefSurName">Moriya</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor" id="eaffaccf-e35e-43e8-bd82-641c9f2fe6e4"><span class="RefSurName">Tahsin</span> <span class="RefGivenName">N</span></span>, <span class="RefAuthor" id="e826a892-5889-44f0-8643-e32a3619f7ee"><span class="RefSurName">Parhar</span> <span class="RefGivenName">IS</span></span>. <span class="RefYear" id="32f4804a-0f25-43ac-bc0d-1052186bed34">2017</span> <span class="RefArticleTitle" id="0bcb61ec-bc44-4c13-8dcb-4eab04c2dbc5">Bpifcl modulates kiss2 expression under the influence of 11-ketotestosterone in female zebrafish</span>. <span class="RefJournalTitle" id="607aa83a-77d6-4e2e-811f-7998e0d8fb2c"><i>Sci Rep</i></span>. <span class="RefVolume" id="cec7c1f1-ba17-4524-812f-5510df2f2f94"><b>7</b></span><span class="RefFPage" id="2a5c5892-52f3-40bc-b862-f357df7ddca3">7926</span>,  (<span class="RefDOI" id="5fe07e71-8020-4e23-9c9a-86fb8e69ddeb">doi: 10.1038/s41598-017-08248-8</span>)<span class="RefWebSite" id="916732d5-265d-4a4c-8d4d-8c82d18a6710">http://dx.doi.org/10.1038/s41598-017-08248-8</span></p><p id="R38" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="23638144" data-validated-online="true" title="Validated from PubMed" data-doi="10.1371/journal.pone.0062776" data-validated-log="true" data-validated="true" seqid="R38" data-id="R38" data-valid-id="R38" new-id="R38"><span class="RefSlNo" id="7dd5f20e-b83e-43d3-80f1-c1b68e74da92">38.</span> <span class="RefAuthor" id="d65f2e1e-d344-4509-8f61-a33d057cce75"><span class="RefSurName">Kanda</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor" id="0f35b838-51b7-4e12-8ee5-4dfe6dfdeba5"><span class="RefSurName">Akazome</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="5d2de04b-690a-4af8-973a-55ae44421bfe"><span class="RefSurName">Mitani</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="7c55cc14-63b5-4cf1-92a6-98cf0b0fac38"><span class="RefSurName">Okubo</span> <span class="RefGivenName">K</span></span>, <span class="RefAuthor" id="af54454b-42d8-4457-aced-d0904cbcbe58"><span class="RefSurName">Oka</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear" id="364a379d-a1a5-4951-bdb6-69f0c7ca0f8d">2013</span> <span class="RefArticleTitle" id="321a7d21-05ab-4855-9229-d8ba4f898a49">Neuroanatomical evidence that kisspeptin directly regulates isotocin and vasotocin neurons</span>. <span class="RefJournalTitle" id="7359d7d2-4b1e-40b1-80b1-cd538e3500d6"><i>PLoS One</i></span>. <span class="RefVolume" id="1cef42ae-9372-403f-9083-b0c07f482dba"><b>8</b></span><span class="RefELocation" id="4f318785-276a-4daf-b741-5a98e520d194"><span class="ins cts-1" data-cid="1702876677764" data-time="1702876677764" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:17:57 GMT+0000 (Coordinated Universal Time)" id="e66e4b38-c289-4289-aecc-e1557c4a4ee8">e62776</span></span>,  (<span class="RefDOI" id="58e28bbd-0fb5-4fee-9c79-cd5a99d931f2">doi: 10.1371/journal.pone.0062776</span>)</p><p id="R39" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="29053844" data-validated-online="true" title="Validated from PubMed" data-doi="10.1210/en.2017-00808" data-validated-log="true" data-validated="true" seqid="R39" data-id="R39" data-valid-id="R39" new-id="R39"><span class="RefSlNo" id="d8b0bd43-c7d0-4849-aa7b-b1a247d6ca18">39.</span> <span class="RefAuthor" id="5c702f6e-624a-4e74-aeed-b72ebd849c38"><span class="RefSurName">Nakajo</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="ba8ae2b9-7b32-4c4b-9fef-7c70e5a25529"><span class="RefSurName">Kanda</span> <span class="RefGivenName">S</span></span>, <span class="RefAuthor" id="047263e8-3e2a-417d-a022-11b9d5ef7e45"><span class="RefSurName">Karigo</span> <span class="RefGivenName">T</span></span>, <span class="RefAuthor" id="440d0e77-3c2b-4f47-8f8f-d535904f30cb"><span class="RefSurName">Takahashi</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor" id="eec82c41-8bdd-475a-8793-c16fa530d443"><span class="RefSurName">Akazome</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="259a9441-c944-486a-92e0-a920101c33a6"><span class="RefSurName">Uenoyama</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="1b94e4b5-b323-404f-a9d0-c0d94a3113d7"><span class="RefSurName">Kobayashi</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="65f656dc-5f4b-4fd9-a244-215040188fe0"><span class="RefSurName">Oka</span> <span class="RefGivenName">Y</span></span>. <span class="RefYear" id="fa858068-659d-497d-8dd8-3f5e445ef29f">2018</span> <span class="RefArticleTitle" id="0adb16e5-f65d-444d-bcdf-d1ef3ad8a5ab">Evolutionally Conserved Function of Kisspeptin Neuronal System Is Nonreproductive Regulation as Revealed by Nonmammalian Study</span>. <span class="RefJournalTitle" id="e199e9c4-82a1-449d-adfb-85a140629c99"><i>Endocrinology</i></span>. <span class="RefVolume" id="e37558ae-d849-4269-8d6e-f647fa63e29d"><b>159</b></span>, <span class="RefFPage" id="68bf7dc7-fc5e-41f0-8e08-3e2c8aa9b1d0">163</span>–<span class="RefLPage" id="bb235f20-c723-4275-8af2-7c6c2af49427">183</span>. (<span class="RefDOI" id="590e2a2e-462c-43b3-b245-a7971bf5b6e4">doi: 10.1210/en.2017-00808</span>)</p><p id="R40" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="32710898" data-validated-online="true" title="Validated from PubMed" data-doi="10.1016/j.ygcen.2020.113568" data-validated-log="true" data-validated="true" seqid="R40" data-id="R40" data-valid-id="R40" new-id="R40"><span class="RefSlNo" id="b838f488-fe53-478e-ab8f-024228761b28">40.</span> <span class="RefAuthor" id="dc47558c-08a8-4abe-9bf1-95d247f5777e"><span class="RefSurName">Somoza</span> <span class="RefGivenName">GM</span></span>, <span class="RefAuthor" id="2d940f13-e63a-4309-99f7-5fe0ffbc1f6e"><span class="RefSurName">Mechaly</span> <span class="RefGivenName">AS</span></span>, <span class="RefAuthor" id="8865f73f-b885-4c92-9021-8734f9e1af89"><span class="RefSurName">Trudeau</span> <span class="RefGivenName">VL</span></span>. <span class="RefYear" id="3ac5accc-bc7b-4f4a-8318-3a65ddd36169">2020</span> <span class="RefArticleTitle" id="a713f024-b04b-4c4c-b139-a3b5e95fddd6">Kisspeptin and GnRH interactions in the reproductive brain of teleosts</span>. <span class="RefJournalTitle" id="d074db30-d407-444b-82c2-cd7ea6ed99e6"><i>Gen Comp Endocrinol</i></span>. <span class="RefVolume" id="7774a603-79c9-4dc0-9bbd-a524bb0e835a"><b>298</b></span><span class="RefFPage" id="24ea4090-d767-41b4-b482-0b59f2da6eca">113568</span>,  (<span class="RefDOI" id="3c2b38f9-61e7-4576-9e88-6775aa55f600">doi: 10.1016/j.ygcen.2020.113568</span>)<span class="RefWebSite" id="90454ba6-f646-4309-827c-5aaa689747ef">https://doi.org/10.1016/j.ygcen.2020.113568</span></p><p id="R41" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="25519132" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/nature13921" data-validated-log="true" data-validated="true" seqid="R41" data-id="R41" data-valid-id="R41" new-id="R41"><span class="RefSlNo" id="0b995699-7e68-4e07-a501-7ce67259d4ac">41.</span> <span class="RefAuthor" id="3caaa3b1-56bf-40d4-aea9-0744e883f957"><span class="RefSurName">Gao</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor" id="e8c5aa77-9da1-485a-98c1-a31f26ab4261"><span class="RefSurName">Lee</span> <span class="RefGivenName">P</span></span>, <span class="RefAuthor" id="0531f60a-c074-4a6e-9cda-546a561585e6"><span class="RefSurName">Stafford</span> <span class="RefGivenName">JM</span></span>, <span class="RefAuthor" id="d66ab85d-888b-4096-a682-faaac65efc15"><span class="RefSurName">von Schimmelmann</span> <span class="RefGivenName">M</span></span>, <span class="RefAuthor" id="e4be5f62-625a-4d56-b74c-0d6579775720"><span class="RefSurName">Schaefer</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor" id="6a76039d-f609-4356-afa0-ce4e7ff42c85"><span class="RefSurName">Reinberg</span> <span class="RefGivenName">D</span></span>. <span class="RefYear" id="730ca80e-b701-4050-9b91-b62e5eb30786">2014</span> <span class="RefArticleTitle" id="c21c7a62-8906-4b82-98cf-961156b2f4cd">An AUTS2-Polycomb complex activates gene expression in the CNS</span>. <span class="RefJournalTitle" id="3c091b4c-034c-4350-bfc0-a697ef269405"><i>Nature</i></span>. <span class="RefVolume" id="74542bad-de52-4535-86fd-8f30101b50a8"><b>516</b></span>, <span class="RefFPage" id="2653c2c7-afd9-44c1-9b58-cea2d3d9ce81">349</span>–<span class="RefLPage" id="1094d43a-b0bc-4f8b-af14-7981ba0b2764">354</span>. (<span class="RefDOI" id="834f791a-9edd-4d30-a8fe-c33249076625">doi: 10.1038/nature13921</span>)</p><p id="R42" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="32498016" data-validated-online="true" title="Validated from PubMed" data-doi="10.1016/j.isci.2020.101183" data-validated-log="true" data-validated="true" seqid="R42" data-id="R42" data-valid-id="R42" new-id="R42"><span class="RefSlNo" id="a55f0333-195a-4362-82b6-a49720cc84c8">42.</span> <span class="RefAuthor" id="27b4149d-421e-42bb-a837-631de2f214ca"><span class="RefSurName">Hori</span> <span class="RefGivenName">K</span></span> <span class="RefEtal" id="5a6158e8-565a-4f77-8e04-ca0e4c181917">et al</span>. <span class="RefYear" id="f50efa85-33f8-4120-ac0f-fc35a161308d">2020</span> <span class="RefArticleTitle" id="a0a39967-1b57-4095-b2da-4db2b83e95db">AUTS2 Regulation of Synapses for Proper Synaptic Inputs and Social Communication</span>. <span class="RefJournalTitle" id="94d55b2f-611d-49a0-87e8-eb3ec0e2962b"><i>iScience</i></span>. <span class="RefVolume" id="e692a7a4-552b-4367-a221-ec10b3ea9614"><b>23</b></span>, <span class="RefELocation" id="db732e79-a633-4355-a4df-f84ffc0faa21"><span class="ins cts-1" data-cid="1702876680016" data-time="1702876680016" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:18:00 GMT+0000 (Coordinated Universal Time)" id="482370ce-dd4b-4661-9883-65a2ab281639">101183</span></span>. (<span class="RefDOI" id="9bda5a7f-82da-401c-b8ae-32ab72343c62">doi: 10.1016/j.isci.2020.101183</span>)<span class="RefWebSite" id="282b1219-e341-407f-8ea7-82bef9fbc208">https://doi.org/10.1016/j.isci.2020.101183</span></p><p id="R43" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="30305620" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/s41467-018-06459-9" data-validated-log="true" data-validated="true" seqid="R43" data-id="R43" data-valid-id="R43" new-id="R43"><span class="RefSlNo" id="08db883d-2c11-4b5b-b575-b627b5019f2c">43.</span> <span class="RefAuthor" id="46a4ee71-606b-4a80-b579-c9b006031ba3"><span class="RefSurName">Vazquez</span> <span class="RefGivenName">MJ</span></span> <span class="RefEtal" id="785ed25f-1c3a-4f24-9716-2c6249f9d1ce">et al</span>. <span class="RefYear" id="be42f350-58e4-45f5-9877-e7ddb3141a3a">2018</span> <span class="RefArticleTitle" id="f4cd58f1-5f2e-4481-8f1f-a12c2651d468">SIRT1 mediates obesity- and nutrient-dependent perturbation of pubertal timing by epigenetically controlling Kiss1 expression</span>. <span class="RefJournalTitle" id="bb5a5f2d-a520-4dfa-a2fe-ae7bad9a35e0"><i>Nat Commun</i></span>. <span class="RefVolume" id="32a3091c-f4b0-4243-92f1-0c028ecefe51"><b>9</b></span><span class="RefFPage" id="5a4acdbc-ba6c-4d95-bcab-2f9b32409e19">4194</span>,  (<span class="RefDOI" id="5ae95251-7d42-46ec-935b-29af6b5f5bbf">doi: 10.1038/s41467-018-06459-9</span>)<span class="RefWebSite" id="4699e5e8-1127-4686-9e3d-918ea72f0abc">http://dx.doi.org/10.1038/s41467-018-06459-9</span></p><p id="R44" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="23727821" data-validated-online="true" title="Validated from PubMed" data-doi="10.1038/nn.3420" data-validated-log="true" data-validated="true" seqid="R44" data-id="R44" data-valid-id="R44" new-id="R44"><span class="RefSlNo" id="9322c774-d306-43cd-b360-ecd0f3849798">44.</span> <span class="RefAuthor" id="d9fa8df7-03a7-4b5e-b3af-541df0fb732c"><span class="RefSurName">Wang</span> <span class="RefGivenName">H</span></span>, <span class="RefAuthor" id="8aeb1cd9-39d0-4356-a38d-582fbed4ea61"><span class="RefSurName">Duclot</span> <span class="RefGivenName">F</span></span>, <span class="RefAuthor" id="48376e18-ef50-448b-a9c8-69e4fe5ef58e"><span class="RefSurName">Liu</span> <span class="RefGivenName">Y</span></span>, <span class="RefAuthor" id="aca7134d-7414-4524-87a6-f5a8cde764b6"><span class="RefSurName">Wang</span> <span class="RefGivenName">Z</span></span>, <span class="RefAuthor" id="e895831e-1898-4595-8823-dd7ed727f70c"><span class="RefSurName">Kabbaj</span> <span class="RefGivenName">M</span></span>. <span class="RefYear" id="5c863313-728b-4aa0-b5f9-74b5600be7db">2013</span> <span class="RefArticleTitle" id="234db113-13de-4897-9f0e-d2efd94886a1">Histone deacetylase inhibitors facilitate partner preference formation in female prairie voles</span>. <span class="RefJournalTitle" id="695e4f58-2174-4e8f-91df-ff5910c13d6b"><i>Nat Neurosci</i></span>. <span class="RefVolume" id="bb160004-01e5-4f10-9039-ee87c303801b"><b>16</b></span>, <span class="RefFPage" id="42c2f67f-0c2c-44e4-bb8a-259757c43071">919</span>–<span class="RefLPage" id="2f326e9b-b8d1-44c7-9625-699eca22dc05">924</span>. (<span class="RefDOI" id="cc93ef07-bcbc-44ea-bc3a-6db7d9257360">doi: 10.1038/nn.3420</span>)<span class="RefWebSite" id="f0134b2b-4bac-4db2-881c-2c1b226c7046">http://dx.doi.org/10.1038/nn.3420</span></p><p id="R45" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-pmid="26722000" data-validated-online="true" title="Validated from PubMed" data-doi="10.1126/science.aac6633" data-validated-log="true" data-validated="true" seqid="R45" data-id="R45" data-valid-id="R45" new-id="R45"><span class="RefSlNo" id="79655900-bacd-4e5c-80d1-1c9cd5d53f6e">45.</span> <span class="RefAuthor" id="c3143ea9-2771-4c27-812a-3d67b9e2c0c4"><span class="RefSurName">Simola</span> <span class="RefGivenName">DF</span></span> <span class="RefEtal" id="671f8ba9-317d-46fb-97f5-67328f42ce78">et al</span>. <span class="ins cts-1" data-cid="1702876680376" data-time="1702876680376" data-userid="PubMed" data-username="PubMed" title="Inserted by PubMed - Mon Dec 18 2023 05:18:00 GMT+0000 (Coordinated Universal Time)" id="88b95a16-afdc-4ff2-b52b-ac7788646f28"><span class="RefYear">2016</span> <span class="RefArticleTitle">Epigenetic (re)programming of caste-specific behavior in the ant Camponotus floridanus</span>. <span class="RefJournalTitle"><i>Science</i></span>. <span class="RefVolume"><b>351</b></span>, <span class="RefELocation">aac6633</span>. (<span class="RefDOI">doi: 10.1126/science.aac6633</span>)</span></p><p class="jrnlRefText" seqid="R46" data-id="R46" data-styled="RefText" data-validated="false" data-validated-log="false" data-csl="true" data-reftype="article-journal" data-class-name="RefText" id="R46" data-untag="true" data-refuntagged="true" data-valid-id="R46" new-id="R46"><span class="RefSlNo">46.</span> <span class="RefAuthor" id="88399571-febc-4168-8261-2d2f1a1acdc1"><span class="RefSurName" id="3761f91b-c7eb-4915-b64f-d4f9652fe91b">Ortega-Recalde</span> <span class="RefGivenName" id="3a93034d-b348-4788-a5bc-1f7951192ce4">O</span></span>, <span class="RefAuthor" id="e9e77e1a-9995-405a-a3e5-5be8e93725af"><span class="RefSurName" id="cac3f7e5-0933-4a05-93e8-4d5cb75636a2">Goikoetxea</span> <span class="RefGivenName" id="659ac2dc-2879-45c4-9563-797782f582c6">A</span></span>, <span class="RefAuthor" id="bceb0c87-3edd-4603-9fb0-8cea8f8ffff5"><span class="RefSurName" id="12560b90-7e15-4407-b67d-90b998e51cf6">Hore</span> <span class="RefGivenName" id="c4e311e5-e21a-4709-8f92-ee2ba12a0768">TA</span></span>, Todd E V., <span class="RefAuthor" id="1284c915-a22d-4be3-ae77-fa3976f479ef"><span class="RefSurName" id="a6fbbf4e-7069-430f-a035-3760f919fa77">Gemmell</span> <span class="RefGivenName" id="013fd2c1-5058-4f4e-a322-e59ab80a1c88">NJ.</span></span> <span class="RefArticleTitle" id="91c47abb-5189-49b4-868b-56696b973cb9">The Genetics and Epigenetics of Sex Change in Fish</span>. <span class="RefJournalTitle" id="67ed832a-8c94-4f06-8d3c-123a9c4ac39d">Annu Rev Anim Biosci</span>. <span class="RefYear" id="4cbe6fab-578a-41d1-a775-8b122017d0bc">2020</span>;<span class="RefVolume" id="f75a99e1-19da-4ad9-bee5-e4c101b35e99">8</span>:<span class="RefFPage" id="717f5dee-aa55-4d41-8ab0-0c95e3113e40">47</span>–<span class="RefLPage" id="af2e265d-1dbf-4d61-b028-f214f998aed1">69</span>.</p><p id="R47" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1016/j.ajhg.2012.12.011" data-validated-log="true" data-validated="true" seqid="R47" data-id="R47" data-valid-id="R47" new-id="R47"><span class="RefSlNo" id="5e689cdc-70cf-4500-9871-adbfeef45ab8">47.</span> <span class="RefAuthor" id="d9fa6fa5-c707-4e82-b755-137d6a14cf23"><span class="RefSurName">Beunders</span> <span class="RefGivenName">G</span></span> <span class="RefEtal" id="b026180a-0a80-4238-9e6b-8dd2ecb16879">et al</span>. <span class="RefYear" id="54fd2638-fb19-4042-bfbe-442f88e9036c">2013</span> <span class="RefArticleTitle" id="53fb6c87-8a55-4268-b8c7-29e3a7c7abfd">Exonic Deletions in AUTS2 Cause a Syndromic Form of Intellectual Disability and Suggest a Critical Role for the C Terminus</span>. <span class="RefJournalTitle" id="be40e767-3a30-4191-b3ff-0f6f07fd7128"><i>The American Journal of Human Genetics</i></span>. <span class="RefVolume" id="8a0cd88e-6f72-41a3-a056-c1001ee854a2"><b>92</b></span>, <span class="RefFPage" id="f431f008-23f1-4f91-b1a4-c8eff5123c2a">210</span>–<span class="RefLPage" id="d1dcb5a3-298c-4709-a4e1-8a24f6cc3e9c">220</span>. (<span class="RefDOI" id="29626cda-efa1-4069-a2a8-561a3d563f31">doi: 10.1016/j.ajhg.2012.12.011</span>)</p><p id="R48" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1126/science.1239864" data-validated-log="true" data-validated="true" seqid="R48" data-id="R48" data-valid-id="R48" new-id="R48"><span class="RefSlNo" id="47cdd157-242c-4631-84db-a557039d1548">48.</span> <span class="RefAuthor" id="a6cc578d-c682-443b-9d77-48fef35b20ed"><span class="RefSurName">Kuroki</span> <span class="RefGivenName">S</span></span> <span class="RefEtal" id="ffa7cb12-c37d-45ec-a510-2cd0f538c450">et al</span>. <span class="RefYear" id="19034cef-1191-4cd7-927f-adae699bf5ac">2013</span> <span class="RefArticleTitle" id="217d18bc-0426-4389-b80a-ce7f816e5cd5">Epigenetic Regulation of Mouse Sex Determination by the Histone Demethylase Jmjd1a</span>. <span class="RefJournalTitle" id="f5220f7c-be05-41b2-9fb8-f04854569a88"><i>Science</i></span>. <span class="RefVolume" id="0b70dc90-66fd-49a7-8930-d773f83fb371"><b>341</b></span>, <span class="RefFPage" id="57e339e3-66b6-45e3-840b-c89b16f98fb3">1106</span>–<span class="RefLPage" id="373b9fbe-5252-47c4-977a-65f8c54f2be8">1109</span>. (<span class="RefDOI" id="b6637294-d879-4676-9be3-251083f12364">doi: 10.1126/science.1239864</span>)</p><p id="R49" class="jrnlRefText" data-reftype="article-journal" data-reftype-name="Journal" data-csl="true" data-validated-online="true" title="Validated from CrossRef" data-doi="10.1016/j.gene.2018.12.057" data-validated-log="true" data-validated="true" seqid="R49" data-id="R49" data-valid-id="R49" new-id="R49"><span class="RefSlNo" id="fb43f3fd-2b17-4d9f-8e26-aa4406f1d26e">49.</span> <span class="RefAuthor" id="7508aa7e-710d-48d1-b7b6-b82f9a514501"><span class="RefSurName">Fellous</span> <span class="RefGivenName">A</span></span>, <span class="RefAuthor" id="0fb8ce6f-3e28-4ad0-aa2e-bd65f4ebd301"><span class="RefSurName">Earley</span> <span class="RefGivenName">RL</span></span>, <span class="RefAuthor" id="6adcf0e1-2592-4d47-bc53-ede1900d5437"><span class="RefSurName">Silvestre</span> <span class="RefGivenName">F</span></span>. <span class="RefYear" id="bc126b25-c810-4f4f-a0aa-851789e4598e">2019</span> <span class="RefArticleTitle" id="2a6d2c42-c1f7-480d-9fb7-c48ae7e54289">Identification and expression of mangrove rivulus (Kryptolebias marmoratus) histone deacetylase (HDAC) and lysine acetyltransferase (KAT) genes</span>. <span class="RefJournalTitle" id="7bf80686-f8d9-4f62-8d2c-4d6a03dc0632"><i>Gene</i></span>. <span class="RefVolume" id="646eae6e-3595-4117-b447-017975be25c1"><b>691</b></span>, <span class="RefFPage" id="e8d182d3-1f31-4f7e-b25d-3ef7cfdea149">56</span>–<span class="RefLPage" id="d55237f5-6be6-4f29-93f8-0a86614a95ee">69</span>. (<span class="RefDOI" id="beab6afa-ac98-40bf-9955-5e38f0505d7f">doi: 10.1016/j.gene.2018.12.057</span>)<span class="RefWebSite" id="c7bd6b3d-4c26-4f22-8511-1af88577d449">https://doi.org/10.1016/j.gene.2018.12.057</span></p></div><div class="jrnlAckGroup"><h1 class="jrnlAckHead" headweightage="45">Acknowledgments</h1><p class="jrnlAckPara">We thank Takuro Nakayama and Sinichiro Maruyama for advising data analysis, and Gen Kume and Midori Matsuoka for scuba sampling. Computations were partially performed on the NIG supercomputer at ROIS National Institute of Genetics.</p></div><h1 class="jrnlHead1" headweightage="45">Data availability</h1><h2 class="jrnlHead2" headweightage="45">Figure and table captions</h2><p class="jrnlTblCaption" id="T1" data-id="T1"><span class="label" title=" T1 " data-offset="0" data-match="Table 1">Table 1.</span>Results of <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="20" data-match="gene ontology" data-pattern="" data-tag-index="" data-abbr="GO">gene ontology</span> enrichment analysis for pairwise comparison and <span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="82" data-match="log-likelihood ratio test" data-pattern="" data-tag-index="" data-abbr="LRT">log-likelihood ratio test</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="109" data-match=" (LRT)" data-pattern="" data-tag-index="37_2">LRT</span>).</p><div class="jrnlSupplBlock" id="BLK_SF1" data-id="BLK_SF1" data-stream-name="a_SF1"><p class="jrnlSupplCaption" id="SF1" data-id="SF1"><span class="label" title=" SF1 " data-offset="0" data-match="Figure S1">Figure S1.</span>The number of protein-coding sequences classified into orthologs for each <span class="jrnlPatterns" data-tag-desc="LSQUOTE" title="LSQUOTE" data-tag-type="LSQUOTE" data-offset="85" data-match="‘" data-pattern="" data-tag-index="34_3">‘</span>max_annot_lvl<span class="jrnlPatterns" data-tag-desc="RSQUOTE" title="RSQUOTE" data-tag-type="RSQUOTE" data-offset="101" data-match="’" data-pattern="" data-tag-index="34_4">’</span> by eggNOG2-mapper, which represents maximum annotation levels of a taxon of the ortholog. The orange color indicates the orthologs of the teleost fish (Actinopterygii).</p></div><div class="jrnlSupplBlock" id="BLK_SF2" data-id="BLK_SF2" data-stream-name="a_SF2"><p class="jrnlSupplCaption" id="SF2" data-id="SF2"><span class="label" title=" SF2 " data-offset="0" data-match="Figure S2">Figure S2.</span><span class="jrnlPatterns" data-tag-desc="EXPN" title="EXPN" data-tag-type="EXPN" data-offset="11" data-match="Principal component analysis" data-pattern="" data-tag-index="" data-abbr="PCA">Principal component analysis</span> (<span class="jrnlPatterns" data-tag-desc="ABBR" title="ABBR" data-tag-type="ABBR" data-offset="41" data-match=" (PCA)" data-pattern="" data-tag-index="35_3">PCA</span>) using whole transcriptome expression profiles. Left, the colors indicate male and female, respectively. The solid and dashed ellipse represents stable and socially stimulated individuals, respectively. Right, a scree plot of the proportion of variance explained.</p></div><div class="jrnlSupplBlock" id="BLK_SF3" data-id="BLK_SF3" data-stream-name="a_SF3"><p class="jrnlSupplCaption" id="SF3" data-id="SF3"><span class="label" title=" SF3 " data-offset="0" data-match="Figure S3">Figure S3.</span>Gene expression profiles based on hierarchical clustering of the genes detected by each LRT associated with behavioral-sex, gonadal-sex, or social stimulation effects. The columns are individuals and each square at the top of the columns represents the comparison groups. The rows are genes detected in LRT comparing the full model vs. the reduced model. Colors represent z-scored expression for each gene and brighter color indicates higher expression in the individual.</p></div></div><div id="queryDivNode"><div class="query-div card author" id="1702876731006" data-type="action"><div class="query-holder"><div class="query-head"><span class="query-from">Exeter</span><span class="query-time">Dec 18 2023 (10:49)</span></div><div class="query-content" data-assigned-by="Exeter Pre-editing" data-assigned-by-role="preeditor" data-assigned-to-role="Author" data-date="Dec 18 2023 (10:49)">Abstract should be unstructured and only one paragraph is allowed</div></div></div></div></div>